---
title: "KRT13 is upregulated in pancreatic cancer stem-like cells and associated with radioresistance"
source: "https://academic.oup.com/jrr/article/64/2/284/6968637"
author:
  - "[[Takenaka]]"
  - "[[Wataru]]"
  - "[[Yokoyama]]"
  - "[[Yuhki]]"
  - "[[Ikehata]]"
  - "[[Katsuya]]"
  - "[[Kouda]]"
  - "[[Shihori]]"
  - "[[Hirose]]"
  - "[[Haruka]]"
published: 2023-01-04
created: 2025-06-06
tags:
---
[Skip to Main Content](https://academic.oup.com/jrr/article/64/2/284/#skipNav)

Journal Article

## Abstract

Pancreatic cancer is one of the most aggressive cancers and the seventh leading cause of cancer-associated death in the world. Radiation is performed as an adjuvant therapy as well as anti-cancer drugs. Because cancer stem-like cells (CSCs) are considered to be radioresistant and cause recurrence and metastasis, understanding their properties is required for the development of novel therapeutic strategies. To investigate the CSC properties of pancreatic cancer cells, we used a pancreatic CSC model, degron (++) cells, which have low proteasome activity. Degron (++) cells displayed radioresistance in comparison with control cells. Using Ribonucleic acid (RNA) sequencing, we successfully identified *KRT13* as a candidate gene responsible for radioresistance. Knockdown of *KRT13* sensitized the degron (++) cells to radiation. Furthermore, a database search revealed that *KRT13* is upregulated in pancreatic cancer cell lines and that high expression of *KRT13* is associated with poorer prognosis. These results indicate that a combination therapy of *KRT13* knockdown and radiation could hold therapeutic promise in pancreatic cancer.

## INTRODUCTION

Pancreatic cancer is one of the most aggressive cancers and its prognosis is extremely poor worldwide \[[1](https://academic.oup.com/jrr/article/64/2/284/)\]. Radiotherapy is an option for adjuvant therapy for pancreatic cancer \[[2](https://academic.oup.com/jrr/article/64/2/284/), [3](https://academic.oup.com/jrr/article/64/2/284/)\]. Although several irradiation methods with high accuracy, such as intensity-modulated radiation therapy, equipment and treatment planning systems have been developed, some patients still experience recurrence.

One possible reason for this problem is the existence of cancer stem-like cells (CSCs). Many studies have shown that CSCs constitute a small population existing in the tumor and have the capacity for self-renewal, high tumorigenicity and resistance to conventional cancer therapies such as radiotherapy and chemotherapy \[[4–8](https://academic.oup.com/jrr/article/64/2/284/)\]. The CSC population is therefore considered to be the source of recurrence and metastasis. Although a number of studies have been conducted to elucidate the properties of CSCs and establish a new treatment strategy for targeting them, as yet no therapeutic targeting CSC is available \[[9](https://academic.oup.com/jrr/article/64/2/284/)\].

One difficulty in analyzing the properties of CSCs is the isolation of the CSC population from tumors. To acquire the CSC population, cell surface markers are often used. In pancreatic cancer, several cell surface markers, such as CD133, CD24, CD44 and epithelial specific antigen, are reported to be CSC markers \[[10](https://academic.oup.com/jrr/article/64/2/284/)\]. In addition to the cell surface markers, low proteasome activity is reportedly a hallmark of CSCs. The proteasome is an enzymatic protein complex that degrades unnecessary or damaged proteins. Cancer cells have high proteasome activity; therefore, proteasome inhibitors such as bortezomib and carfilzomib are used clinically as anti-cancer drugs \[[11](https://academic.oup.com/jrr/article/64/2/284/)\]. On the other hand, it has been demonstrated that cancer cells with low proteasome activity have CSC properties, including radioresistance \[[12](https://academic.oup.com/jrr/article/64/2/284/), [13](https://academic.oup.com/jrr/article/64/2/284/)\]. CSCs can thus be purified on the basis of this difference in proteasome activity.

To assess their proteasome activity, cells are engineered to stably express green fluorescent protein (ZsGreen) fused to the degron sequence of ornithine decarboxylase (ODC), which is directly recognized by the proteasome. Because ZsGreen accumulates in cells such as CSCs that have low proteasome activity, this degron system enables real-time monitoring and cell sorting of CSCs without any treatment. In addition, this system is useful for multiple cancer types because proteasome activity is a common biologic property \[[14–21](https://academic.oup.com/jrr/article/64/2/284/)\]. Collectively, this method is better than cell surface marker methods, which depend on cancer type and require staining.

In this study, we generated PANC-1 cells stably expressing ZsGreen fused to the ODC degron to isolate the CSC population and identify the genes responsible for radioresistance. We found that *KRT13* is upregulated in pancreatic CSCs and associated with radioresistance. Database analysis revealed that *KRT13* is highly expressed in pancreatic cancer cell lines and that high expression of *KRT13* is associated with poorer prognosis. Taken together, these results suggest that *KRT13* knockdown in combination with radiotherapy could be a new strategy for treating pancreatic cancer.

## MATERIALS AND METHODS

### Generation of a pancreatic CSC model

Pancreatic CSC model was generated as described previously \[[22](https://academic.oup.com/jrr/article/64/2/284/), [23](https://academic.oup.com/jrr/article/64/2/284/)\]. Briefly, the retroviral expression vector pQCXIN-ZsGreen-cODC, which contains ZsGreen gene fused to the carboxyl-terminal degron sequence of ODC was transfected into Platinum retroviral packaging cells and the retrovirus collected from the supernatant was infected to human pancreatic cancer cell line PANC-1. pQCXIN-ZsGreen-cODC was kindly provided by Dr Frank Pajonk (Jonsson Comprehensive Cancer Center, UCLA, CA, USA). We defined this infected PANC-1 cell line as degron (−). Because degron (−) contains few ZsGreen-positive cells (<1%), ZsGreen-positive degron (−) cells could be repeatedly sorted out by using an SH800Z cell sorter (SONY, Tokyo, Japan) and cultured. This allowed us to obtain a concentrated cell population containing more than 40% ZsGreen-positive cells (degron (+)). For the *in vitro* experiment, the 50% of cells that were the most ZsGreen positive in this population were used as a CSC model, defined as degron (++). All assays (irradiation, small interfering RNA (siRNA) transfection) were performed 48 hours after cell sorting in order to reduce cell damage. Cells were cultured in Dulbecco’s Modified Eagle’s Medium (Sigma-Aldrich, St. Louis, MO, USA) with low glucose, 10% fetal bovine serum (Sigma-Aldrich) and G418 (Roche, Basel, Switzerland) for selection at 37°C in a humidified atmosphere of 5% CO <sub>2</sub>.

### Irradiation

Cells in a culture flask were irradiated with a cesium-137 gamma-ray irradiator, (Gammacell 40 Exactor) at our facility, at a dose rate of approximately 0.81 Gy/min.

### Colony formation assay

Immediately after irradiation, cells were trypsinized and seeded at 1000 or 5000 cells into 6-cm dishes. Fourteen days later, these cells were fixed with 4% paraformaldehyde and stained with a crystal violet solution. After staining, the colonies were counted and cell viability was calculated.

### Flow cytometry analysis

Apoptotic cells were detected by using a GFP-Certified Apoptosis/Necrosis Detection Kit (Enzo Life Sciences, East Farmingdale, NY, USA) according to the manufacturer’s instructions. Apoptotic cells were stained with annexin V–EnzoGold and/or 7-amino-actinomycin D (7AAD). For cell labeling to evaluate the cell surface expression of CD44 variant 9 (CD44v9) and CD133, primary antibody to CD44v9 (clone RV3, rat-IgG monoclonal, Cosmo Bio, Tokyo, Japan) and CD133 (clone AC133, mouse-IgG monoclonal, Miltenyi Biotec, Cologne, German) were used. As a secondary antibody, PE conjugated anti rat-IgG (clone RG7/1.30, BD Bioscience, NJ, USA) and APC conjugated anti mouse-IgG (clone X57, Miltenyi Biotec) were used. The analysis was performed by using an SH800Z Cell Sorter.

### Sphere formation assay

Cells were seeded in 96-Well Clear Ultra Low Attachment Microplates (Corning, Inc., Corning, NY, USA) at a density of 500 cells per well. For siRNA experiment, cells were seeded after 48 hours of siRNA treatment. The cells were cultured in DMEM serum-free medium (Thermo Fisher Scientific, Waltham, MA, USA) supplemented with 20 ng/ml epithelial growth factor, 10 ng/ml basic fibroblast growth factor-2 (PeproTech, Inc., Cranbury, NJ, USA), 100 U/ml penicillin and 100 μg/ml streptomycin, following our previous article \[[23](https://academic.oup.com/jrr/article/64/2/284/)\]. Cells were cultured in a humidified incubator at 37°C and 5% CO <sub>2</sub>. Images were captured by a bright field light microscope (CKX53; Olympus Corporation, Tokyo, Japan) with Visualix camera (Visualix, Kobe, Japan). The number of spheroids was counted larger than 100 μm in size manually on days 5 after seeding.

### Reverse transcription quantitative PCR analysis

Total RNA was purified from cultured cells using TRIzol Reagent (Thermo Fisher Scientific), and complementary DNA was generated from 2.0 μg total RNA using High Capacity cDNA Reverse Transcription kit (Thermo Fisher Scientific) following to the manufacturer’s instructions. Quantitative polymerase chain reaction (qPCR) was conducted by using THUNDERBIRD Next SYBR qPCR Mix (TOYOBO, Osaka, Japan). Relative expression was quantified using the 2 <sup>−ΔΔ</sup> Ct method. The expression level of the target gene was normalized by GAPDH mRNA expression.

Primers are shown below:

GAPDH F \[5′- CAACTACATGGTTTACATGTTC\], R \[5′- GCCAGTGGACTCCACGAC\].

KRT13 F \[5′- CCCCAGGCATTGACCTGAC\], R \[5′- GTGTTGGTAGACACCTCCTTG\].

### RNA sequence data

To identify candidate radioresistance genes, RNA sequence analysis was performed as previously described \[[24](https://academic.oup.com/jrr/article/64/2/284/)\]. Gene expression data were analyzed using fold change and fragments per kilobase of exon per million reads mapped (FPKM). Fold change means the ratio of gene expression between degron (−) and degron (++) cells. FPKM means the absolute value of gene expression used in RNA sequence analysis. The raw data were deposited in the NCBI Gene Expression Omnibus database under accession number GSE212883. For gene enrichment analysis, we used KEGG pathway database ([https://www.genome.jp/kegg/pathway.html](https://www.genome.jp/kegg/pathway.html)). In addition, we performed GSEA with PreRank option to input the whole list of genes. We used the gene sets from the Broad Institute’s Molecular Signature Database. False discovery rate (FDR) q-value <0.05 was considered as the threshold for significance.

### siRNA transfection

Cells were transfected with siRNA against *KRT13* (si *KRT13*) (FlexiTube siRNA, Qiagen, Hilden, Germany) and negative control siRNA (siNC) (GeneDesign, Osaka, Japan) by the use of Lipofectamine RNAiMAX (Thermo Fisher Scientific) according to the manufacturer’s instructions. Cells were irradiated 24 hours after transfection.

### Data analysis

The Cancer Cell Line Encyclopedia (CCLE) database ([https://portals.broadinstitute.org/ccle](https://portals.broadinstitute.org/ccle)) was used for the analysis of mRNA expression in the multiple cancer cell lines. The survival analysis was performed with the cancer genome atlas (TCGA) database and the Oncolnc tool ([http://www.oncolnc.org/](http://www.oncolnc.org/)) \[[25](https://academic.oup.com/jrr/article/64/2/284/)\].

### Statistics

Data are shown as mean ± SD. The data were compared by using the Student t-test. A value of *P*  < 0.05 was considered statistically significant. All statistical analyses were performed using Microsoft Excel.

## RESULTS

### Generation of the pancreatic CSC model

We generated PANC-1 cells stably expressing ZsGreen fused to the ODC degron to visualize CSCs as cells with low proteasome activity. In this cell line, fewer than 1% of the cells were ZsGreen positive; therefore, we defined this cell line as degron (−) cells ([Fig. 1A and C](https://academic.oup.com/jrr/article/64/2/284/)). This percentage was reasonable because the CSC population in the tumor is rare, but it might have been difficult to accurately analyze the properties of CSCs with this model. Therefore, we concentrated the ZsGreen-positive cell population by repeated FACS and culturing. We eventually obtained the degron (+) cell line, which contains more than 40% ZsGreen-positive cells ([Fig. 1B and C](https://academic.oup.com/jrr/article/64/2/284/) right). To further concentrate the CSC population, the 50% of cells that were the most ZsGreen positive (defined as degron (++)) were used for the subsequent experiments ([Fig. 1C](https://academic.oup.com/jrr/article/64/2/284/) blue color). We confirmed that degron (++) cells showed high expression of the cell surface CSC markers (CD44v9 and CD133) and increased the number of spheroids compared with degron (−) cells ([Fig. 1D and E](https://academic.oup.com/jrr/article/64/2/284/)).

![Generation of a pancreatic CSC model by use of the ODC-degron system. A. Photomicrograph of degron (−) cells. The percentage of ZsGreen-positive cells was less than 1%. B. Photomicrograph of degron (+) cells. The percentage of ZsGreen-positive cells was more than 40%. C. The expression levels of ZsGreen in the degron (−) and degron (+) cell populations. The 50% of cells with the highest ZsGreen expression (blue color) were defined as degron (++) cells and used for the subsequent experiments. D. The number of CD44v9 and CD133 double positive population in degron (−) and degron (++) cells. The number relative to the control cells (degron (−)) are shown. Each bar represents the mean ± standard deviation (*  P < 0.05). E. The number of spheroids in degron (−) and degron (++). Spheroids (>100 um) were counted on day 5 after seeding. Each bar represents the mean ± standard deviation (*  P < 0.05).](https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/jrr/64/2/10.1093_jrr_rrac091/1/m_rrac091f1.jpeg?Expires=1752211062&Signature=PUWJNhibvorluE8HSo1Oe0pMJ~wmOGU213xY2HjdhF9KAWUROYO4vwJwvgcd~LYB~P1IqHoQHxmKcbPyuZU12Ll9CX~MqSwQ17Q7EDeZKYSpTV~Cvae88CQZpVWRv4TEf1HhUcS8bUA-20MRWYGS7RBj0FRM8oatUJlEhmSHzXfV9gOR3-YX6Bs0NibVsn6uLnxRVkwsKU~7aPipnI9tNwx6ckj92BO3BRelT-bRgIuCnpKDInTCKPRgSkhEQ3R1~CePSWz0QnYWGDiWHOxHHTq9KRLvVdIZXPHd-j1aPEFPdtcL~NON4WFyRJb3A~zy6JItKPDvJHQEaJy66QinIw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA)

Fig. 1

Generation of a pancreatic CSC model by use of the ODC-degron system. A. Photomicrograph of degron (−) cells. The percentage of ZsGreen-positive cells was less than 1%. B. Photomicrograph of degron (+) cells. The percentage of ZsGreen-positive cells was more than 40%. C. The expression levels of ZsGreen in the degron (−) and degron (+) cell populations. The 50% of cells with the highest ZsGreen expression (blue color) were defined as degron (++) cells and used for the subsequent experiments. D. The number of CD44v9 and CD133 double positive population in degron (−) and degron (++) cells. The number relative to the control cells (degron (−)) are shown. Each bar represents the mean ± standard deviation (<sup>*</sup> *P*  < 0.05). E. The number of spheroids in degron (−) and degron (++). Spheroids (>100 um) were counted on day 5 after seeding. Each bar represents the mean ± standard deviation (<sup>*</sup> *P*  < 0.05).

[Open in new tab](https://academic.oup.com/view-large/figure/399452422/rrac091f1.tif) [Download slide](https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/jrr/64/2/10.1093_jrr_rrac091/1/rrac091f1.jpeg?Expires=1752211062&Signature=ijXqoy9UV3UZWpC3ty0dMZkiI-XDq93sqQDl5eyN3gd2hpzpPlpKU03RINTA9SyqaujKz9IgYIIyXd2psHfptsnn9a0IKfXtPfxfVuKlEHFqAiko~BYFcl4jCjjws52lJlAVahBqIBfFEUaqo4M3SgqbDWRGbM2ie6tjqzHTnn6sK7aGKcldZr7p4OgjhnaCpLd4c7Hu~DKWpAn88E7wetshCoP9p7q6abwXTitAmGfzqHVcjFDdamKYhKutPxzb8fRHw-RwCTynP8MFkOL6h8aQ4tsF3WO51pz12F92~xJS6hwES~V~iX~OgM3vToFyTTBiijtKbumG1gsxLVVFpA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=399452422&ar=6968637&xsltPath=~/UI/app/XSLT&imagename=&siteId=5319)

### Degron (++) cells displayed higher radioresistance

To investigate the radioresistance of degron (++) cells, we performed an annexin V assay and found that the number of apoptotic cells was lower in degron (++) cells than in degron (−) cells 72 hours after irradiation at 4 and 8 Gy ([Fig. 2A and B](https://academic.oup.com/jrr/article/64/2/284/)). We also performed a colony formation assay to assess long-term cell survival after irradiation. After irradiation, the viability of degron (++) cells was significantly higher than that of degron (−) cells ([Fig. 2C](https://academic.oup.com/jrr/article/64/2/284/)). These results indicate that the degron (++) cells were more radioresistant than the degron (−) cells.

![Radioresistance was higher in degron (++) cells than in degron (−) cells. A. Flow cytometry analysis of annexin V–positive (X axis) and 7-AAD–positive (Y axis) cells 72 hours after irradiation of degron (−) and degron (++) cells. B. Bar chart of the apoptotic cells (annexin V–positive cells) shown in Fig. 2A. Each bar represents the mean ± standard deviation (**  P < 0.01). C. Cell viability of degron (−) and degron (++) cells fourteen days after irradiation. The values relative to the control cells (no irradiation) is shown. Each bar represents the mean ± standard deviation (**  P < 0.01).](https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/jrr/64/2/10.1093_jrr_rrac091/1/m_rrac091f2.jpeg?Expires=1752211062&Signature=h3vX8K-DIJ6KgP49~JH2qX9VaIiiPl8-owhHnSPrj03bViGqO9u4y0Sa8W9YNulJS4f-ImytvFirQUhsesvvdOCq6J1IRnVO9ZsRil3vCfmw6TaK1Joo7hBt1PDZWWKcA4fGv1Vjnn9IxNtNSkrgOI0be4TuMTGeG30nE-J8SJET4Su~e89jcC1hZ5E8GWVoNTKr31WKMJnteej~bcqfnKsq6md6ODUO6EqtV18XM-SbrS8nKEzERdrSOk0dtTXMwL5HKhObgLrEG6Ji~~aGBTDs9JnBwfD6xEhT7t2ASp9JqHw2PEd9a-f6Y5JsZ4p1SfN0cK4jIOcNpSrnhnMsJg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA)

Fig. 2

Radioresistance was higher in degron (++) cells than in degron (−) cells. A. Flow cytometry analysis of annexin V–positive (X axis) and 7-AAD–positive (Y axis) cells 72 hours after irradiation of degron (−) and degron (++) cells. B. Bar chart of the apoptotic cells (annexin V–positive cells) shown in [Fig. 2A](https://academic.oup.com/jrr/article/64/2/284/). Each bar represents the mean ± standard deviation (<sup>**</sup> *P*  < 0.01). C. Cell viability of degron (−) and degron (++) cells fourteen days after irradiation. The values relative to the control cells (no irradiation) is shown. Each bar represents the mean ± standard deviation (<sup>**</sup> *P*  < 0.01).

[Open in new tab](https://academic.oup.com/view-large/figure/399452425/rrac091f2.tif) [Download slide](https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/jrr/64/2/10.1093_jrr_rrac091/1/rrac091f2.jpeg?Expires=1752211062&Signature=DuyKKEAakUYUUJ7GWWkQKD7bZw70ME0vseHUTW8tVMjOWpIrD-eez~0clHaeGgWD-Br9Yq1CUa9cUd3TRSlW9PP943i0C6qrd~Ldkcm4XwYI~p1sKijw6Y2qerq76cmJKr5RkgMEfhQdvpU9N3alfDL1j6mFu40keVC93DSIGG-sqabgp190W1773NLma9qie7HFJzvXYFcnBFqe8H-GCs7fbk1FfdtaxbOuyZQ845Lvb93GZa6hNVOpZOs5-JStOORhzEkJbk64oP-wuqt8JxJs7lKaMXXT6bScVsNH~4VhQsxN3DHSieTogWfofzHg1F8CWbRrq3dwSKp14A5s8w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=399452425&ar=6968637&xsltPath=~/UI/app/XSLT&imagename=&siteId=5319)

### KRT13 gene expression was upregulated in degron (++) cells

To identify candidate genes associated with radioresistance in degron (++) cells, RNA sequence analysis was performed to compare gene expression in degron (++) cells with that in degron (−) cells. Among over 20 000 genes analyzed, the top 10 most upregulated genes in degron (++) cells are shown in [Table 1](https://academic.oup.com/jrr/article/64/2/284/). We focused on *KRT13* as a candidate gene responsible for radioresistance in pancreatic CSC because an association of some KRT family genes, such as *KRT19*, with the stemness of normal and cancer cells has been reported \[[26–29](https://academic.oup.com/jrr/article/64/2/284/)\].

Table 1

[Open in new tab](https://academic.oup.com/view-large/399452428)

Gene list that is upregulated in degron (++) cells

<table><thead><tr><th><strong>Gene</strong></th><th><strong>Fold change</strong></th><th colspan="2"><strong>FPKM</strong></th><th><strong>Description</strong></th></tr><tr><th></th><th></th><th><strong>Degron (++)</strong></th><th><strong>Degron (−)</strong></th><th></th></tr></thead><tbody><tr><td>MMP12</td><td>36.02</td><td>39.69</td><td>1.17</td><td>matrix metallopeptidase 12 (macrophage elastase)</td></tr><tr><td>COL4A4</td><td>32.64</td><td>11.03</td><td>0.34</td><td>collagen, type IV, alpha 4</td></tr><tr><td>TMPRSS3</td><td>29.61</td><td>8.26</td><td>0.33</td><td>transmembrane protease, serine 3</td></tr><tr><td>TMEM125</td><td>28.35</td><td>8.65</td><td>0.37</td><td>transmembrane protein 125</td></tr><tr><td>STEAP4</td><td>23.52</td><td>9.08</td><td>0.39</td><td>STEAP family member 4</td></tr><tr><td><em>KRT13</em></td><td>22.24</td><td>83.22</td><td>3.89</td><td>keratin 13</td></tr><tr><td>TJP3</td><td>22.09</td><td>8.12</td><td>0.38</td><td>tight junction protein 3</td></tr><tr><td>TNFRSF11B</td><td>20.53</td><td>91.00</td><td>4.48</td><td>tumor necrosis factor receptor superfamily, member 11b</td></tr><tr><td>PSG5</td><td>20.21</td><td>7.46</td><td>0.40</td><td>pregnancy specific beta-1-glycoprotein 5</td></tr><tr><td>MUC16</td><td>19.94</td><td>6.88</td><td>0.34</td><td>mucin 16, cell surface associated</td></tr></tbody></table>

### KRT13 knockdown decreased radioresistance and stem-like cell property in degron (++) cells

To investigate whether *KRT13* is related to the radioresistance in degron (++) cells, we performed the knockdown experiment. We verified that si *KRT13* treatment decreased the mRNA expression of *KRT13* in degron (++) cells ([Supplementary Fig. 1](https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/jrr/64/2/10.1093_jrr_rrac091/1/supplementary_fig_1_rrac091.jpeg?Expires=1752211062&Signature=1~unGzTnGI~vmlYfYlU7w63KWjamsjKSDMnJn988Vd-gElrKRwj4Kdo~W1GGPHHY9306iOEbm9zXdjrsE~17TODjyx0MoAKKiVMXJw0WFW1Ox5yRsq4nAwCoTEJMxu2Fcvgg~RJJ2-ePML0UCN85jdulb8jAEuxjs~nbiW22Pl6uMVIBFl38zgwRr7O9hOaCt0OSXXbuZnIaPGTOgceV4MP-eRcXrxaUd5ADbphmbMJTq4dO7FbsD4i1uH0JLuG~v5VQ-u5cvS3o-fAu9lcRvc8UP04iMyoknLlUDwgA03MURKy-JIhbkmhOGyIDGF2nILVWNwNJNc0DDpcdrVrrCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA)). Knockdown of *KRT13* induced apoptosis in degron (++) cells even without irradiation, and this effect was further enhanced by irradiation at 4 Gy, but not enhanced at 8 Gy ([Fig. 3A and B](https://academic.oup.com/jrr/article/64/2/284/)). The colony formation assay showed that the viability of degron (++) cells after irradiation was significantly decreased in si *KRT13* -treated cells relative to that of cells treated with negative control siRNA ([Fig. 3C](https://academic.oup.com/jrr/article/64/2/284/)). Furthermore, we showed that si *KRT13* treatment decreased the cell surface CSC markers (CD44v9 and CD133) and the number of spheroids in degron (++) cells ([Fig. 3D and E](https://academic.oup.com/jrr/article/64/2/284/)). These results suggest that *KRT13* is related to the radioresistance and stemness of pancreatic cancer cells.

![KRT13 knockdown decreased radioresistance and stem-like cell property in degron (++) cells. A. Flow cytometry analysis of annexin V–positive (X axis) and 7-AAD–positive (Y axis) cells 72 hours after the irradiation of siNC- and siKRT13-treated degron (++) cells. B. Bar chart of the apoptotic cells (annexin V–positive cells) shown in Fig 3A. Each bar represents the mean ± standard deviation (*  P < 0.05). C. Viability of siNC- and siKRT13-treated degron (++) cells fourteen days after irradiation. Values relative to the control cells (no radiation) are shown. Each bar represents the mean ± standard deviation (**  P < 0.01). D. The number of CD44v9 and CD133 double positive population in siNC- and siKRT13-treated degron (++) cells. The number relative to the control cells (siNC-treated) are shown. Each bar represents the mean ± standard deviation (**  P < 0.01). E. The number of spheroids in siNC- and siKRT13-treated degron (++) cells. Spheroids (>100 um) were counted on day 5 after seeding. Each bar represents the mean ± standard deviation (**  P < 0.01).](https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/jrr/64/2/10.1093_jrr_rrac091/1/m_rrac091f3.jpeg?Expires=1752211062&Signature=hiQAPGEF1M2wQQB9k6E5DZAdw3gEG9TjArjiS-35-L7tIg164TE74t5icLiYeY58OK3CNTZYz8iuJl-mds65DFV7MkZ5MLh6NmLD3Kdswil9LmbiNt-KU6dnI5kkqK9NbIVrAAe3X6VNOEAWMgbtzWC-N4JpJ~EWMrVT9NL1lUUeuLKDt0lHIJ9BO8h9L5Skk6DaWk1kCObS2~x6JsvEUscwO5UeRaKbaJ52UlVTTCVGt4eCdE7G1Z~Nrv179YEuUnNZFaNqjSpyDX76Ti52JWpA2YiqvRzv~zkP1uziQaXai1Pea3Uzw9sNSyTuh539ZTxRcu0Bjx~pdg3acSdgtA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA)

Fig. 3

*KRT13* knockdown decreased radioresistance and stem-like cell property in degron (++) cells. A. Flow cytometry analysis of annexin V–positive (X axis) and 7-AAD–positive (Y axis) cells 72 hours after the irradiation of siNC- and si *KRT13* -treated degron (++) cells. B. Bar chart of the apoptotic cells (annexin V–positive cells) shown in [Fig 3A](https://academic.oup.com/jrr/article/64/2/284/). Each bar represents the mean ± standard deviation (<sup>*</sup> *P*  < 0.05). C. Viability of siNC- and si *KRT13* -treated degron (++) cells fourteen days after irradiation. Values relative to the control cells (no radiation) are shown. Each bar represents the mean ± standard deviation (<sup>**</sup> *P*  < 0.01). D. The number of CD44v9 and CD133 double positive population in siNC- and si *KRT13* -treated degron (++) cells. The number relative to the control cells (siNC-treated) are shown. Each bar represents the mean ± standard deviation (<sup>**</sup> *P*  < 0.01). E. The number of spheroids in siNC- and si *KRT13* -treated degron (++) cells. Spheroids (>100 um) were counted on day 5 after seeding. Each bar represents the mean ± standard deviation (<sup>**</sup> *P*  < 0.01).

[Open in new tab](https://academic.oup.com/view-large/figure/399452431/rrac091f3.tif) [Download slide](https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/jrr/64/2/10.1093_jrr_rrac091/1/rrac091f3.jpeg?Expires=1752211062&Signature=KXPUXAKPmvS9rBnFKBct5Leo3lH8rVi8PE2AYUeal41Ip9FwgDKt2YhjGuwXiSs1iad-GwEWSU96mx~nve1vvokqoV1C5F4du-LXAPO-PT574K1DwK3i2RMuRxg40RGdMznSUyx8fRWMseoDfw0r04UnwpuQNvek~nHP8k9azQL7~ork58ZdbHrckI6~DnTPDnt3m3cTQOjrpDB1haoow6ubir45I42qVuoceZaYPZq5iyX8ydcbdjqiTiPj5awCkw4a5lyCPvFl73QUE37MiWRnQd0zhzqkVwuySL8pxSnTs8xypSo3Lucj7OwnlV5oahauZRTEAybe8hpq1TxZzQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=399452431&ar=6968637&xsltPath=~/UI/app/XSLT&imagename=&siteId=5319)

### KRT13 is upregulated in pancreatic cancer and associated with poor prognosis

To investigate the clinical significance of *KRT13* in pancreatic cancer, we analyzed a publicly available database and found that *KRT13* expression is upregulated in pancreatic cancer cell lines ([Fig. 4A](https://academic.oup.com/jrr/article/64/2/284/); rank no. 4 among the multiple cancer types). Furthermore, TCGA survival data showed that high *KRT13* expression is associated with poor prognosis in patients with pancreatic cancer ([Fig. 4B](https://academic.oup.com/jrr/article/64/2/284/); *P*  < 0.0001).

![KRT13 is upregulated in pancreatic cancer cell lines, and high KRT13 expression is associated with poorer prognosis in pancreatic cancer. A. KRT13 expression levels in multiple cancer cell lines registered in the CCLE database. B. Patient survival in KRT13low and KRT13high expression groups in pancreatic cancer. The analysis was performed by using the OncoLnc web tool. P < 0.0001.](https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/jrr/64/2/10.1093_jrr_rrac091/1/m_rrac091f4.jpeg?Expires=1752211062&Signature=0WKJTAybdnyCcGm8t7Lg7dpMB-EwJKt7wRp4DOLuSU53lUtbQ~W75UL2Z4Hpvvk3Wen8J5K3u1GSLzp55CUMfd1NrLs9sUjq-9hqkkLo9Y1w0C3anROjXU3GJ3RNU41ewSVybDut0inXvfsARfLSSJvncrS2esgvx73UiM470KfPFdJeGRcJUZf1sq0Qu1uVv1dfHYeZKA-HbFiHAWOfzcvSPAWFjsR69nXeAvddqiwIBp2apQMKSj2KQfEXvfGc3CIZYpse3RwcOk~eZVbi5wpK6eP6K060BazYMOyD4G3gGxegohu-xpdk6NGAmigdzmwJwP-NwfvODryRffwPJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA)

Fig. 4

*KRT13* is upregulated in pancreatic cancer cell lines, and high *KRT13* expression is associated with poorer prognosis in pancreatic cancer. A. *KRT13* expression levels in multiple cancer cell lines registered in the CCLE database. B. Patient survival in *KRT13* <sup>low</sup> and *KRT13* <sup>high</sup> expression groups in pancreatic cancer. The analysis was performed by using the OncoLnc web tool. *P*  < 0.0001.

[Open in new tab](https://academic.oup.com/view-large/figure/399452434/rrac091f4.tif) [Download slide](https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/jrr/64/2/10.1093_jrr_rrac091/1/rrac091f4.jpeg?Expires=1752211062&Signature=S4oDU2neySputyyZumkJW8U-ac5uS2Hwv0UBBf7mbz6J5M0pds0GwU5nD9QiGi9P1Zd6zsGlbOx~I9wagXUyekbvA-E6RVsYeYDY8~ecfJL01Jxky4UaGGS1ThH6rWgr06VvxTZMnVf68ws9wFoI2dVuX8Yfqah9UKQN~zagFYgPp8istPrl--ODEXaWgLORpr-EY-KtOS~Dw8xICH9iIYZR1PrQpDJ-9p3MMqduWaAzg882q~6sO5xA2qa0vZHCWdcxjbsr25cTIAVF1YNoDKK1-su-wRzCsIza7zc6cdqcknn4iYrdDMbGnWovxgXifuy7m4AdftDE4RZu8rl5rg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=399452434&ar=6968637&xsltPath=~/UI/app/XSLT&imagename=&siteId=5319)

## DISCUSSION

In this study, we aimed to discover the gene responsible for radioresistance in pancreatic CSCs. To achieve this purpose, we used a ZsGreen-ODC degron fusion reporter system \[[12](https://academic.oup.com/jrr/article/64/2/284/)\]. This system enables the visualization of cells with low proteasome activity, which has been considered a property of CSCs, so that the CSC population can be sorted and concentrated by FACS. Adikrisna *et al.* revealed that ZsGreen highly positive PANC-1 cell population showed the increased sphere formation, chemoresistance, asymmetric cell division and *in vivo* tumorigenesis, which represent the stem-like characteristics, compared with the control cells \[[22](https://academic.oup.com/jrr/article/64/2/284/)\]. In addition, we previously showed that ZsGreen highly positive PANC-1 cell population were resistant to several microRNA treatments compared with control cells \[[23](https://academic.oup.com/jrr/article/64/2/284/)\]. Using this approach, we acquired the degron (+) cell line and confirmed that the further concentrated degron (++) cells displayed greater radioresistance and increased expression of the cell surface CSC markers (CD44v9 and CD133) and the number of spheroids compared with the non-concentrated degron (−) cells (<1% ZsGreen-positive cells). This indicates that we successfully generated a CSC model for pancreatic cancer.

We performed RNA sequence analysis to identify the gene responsible for radioresistance in pancreatic CSCs. In addition to identify the gene responsible for radioresistance, we performed gene enrichment analysis by using public database, Kyoto Encyclopedia of Genes and Genomes (KEGG) and Gene Set Enrichment Analysis (GSEA). Gene enrichment analysis using KEGG pathway database showed that 421 upregulated genes (Fold change >5 and *P*  < 0.01) in degron (++) cells compared with degron (−) cells were enriched in Axon guidance, ECM-receptor interaction and PI3K/AKT signaling pathway ([Supplementary Table 1](https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/jrr/64/2/10.1093_jrr_rrac091/1/supplementary_table1_rrac091.xlsx?Expires=1752211062&Signature=Ea9xWh2A1rTIVDoAT9a79Ich4IxIe91HUMDPZIBNj3vs8erXi1cMgCfpxErJ5T~HP9GEdxRrUz8zT5QnE~sj~LotyMa0F5ouD9c681gZPCi9Y8rNWKY59ft5b5lMXv90TrPCTAWZizIfL6dYn7EUpHKzqdDdYHdqe-6EvQG0zos~kzvH6-iGhzaDitykJfzOhKtRhl8Bg5CNDnSo~fuThwxofUv4egjPhEVG8EqQzQ0HMqAZeWit61l5IMyWodgPy9RL2U79QkXta0bLrD0LgIDPqS761alGrUDA6LE3eDvkAB19LlDiewwAbiZ1oEAtSCxD3vnyd-hKb6lbRg80tg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA)). On the other hand, we found the significant enrichment with 44 gene sets including KRAS dependency signature in degron (++) cells by GSEA ([Supplementary Table 2](https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/jrr/64/2/10.1093_jrr_rrac091/1/supplementary_table2_rrac091.xlsx?Expires=1752211062&Signature=AKRoe4vzhgkwGl5HwNa6S9jMKrGmOFPy5tfq9bHeO53ir7gOUU~avh4oUXroTeBqWVc~rQLEpVCyD0xD-szxN5Q6fZHqTFTWQlPUUF4ZEkjC4lJawSSLG6ixa0j1A0FWL8XgHTYLVrNYM4qJySR-TzcyprZHx0JJvDOw44eaAgLh9Y~4x7KsYSurO4P2KTjSBFJWqi2unnotxax2RjQdNE-qvMymQbAAu5PwoTQEVYPk2YJgF20JPnRA2LJubZMaTt3gOeUarx0vbDuhtRhyDx6rXeSblq7HphsCtbJtwwEkLkOXZ~jijpPA~aaSmW8qgmK2iz~3wfdamLoKSu-oEA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA), [Supplementary Fig. 2](https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/jrr/64/2/10.1093_jrr_rrac091/1/supplementary_fig_2_rrac091.jpeg?Expires=1752211062&Signature=JFc-I7zWfficN9vz6A3xigYTVFF1f89p5lV2HthQTmPbcL0FXEJbt5yMg16g6~eJGN3rjNGjmKhv3Nt5xP6yvRikFUReCUoOs2gDGXo5otvk--DHqFINNX0TlRajrNZKsnYs1yKQReG6PGbBd04XZeTKU6RiHWKHm-Dnk5f9wORnrbIzxD2idl7WIpjZGdJ2AnkYLLaL~RRE55Qflvh~-8BVRk3Gl1z3JOCr4JtpMG77MpmpoY1942X0mgv-YgZ-4oq~53osr3kJLzTbjtsmbh9hZ6kYr65HvLKnxKzwjZskW-cIXpc-8Zb60Zaa5xFtF~QXnV3V6GsWppQ9qf~okw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA)). Further investigation is needed to clarify whether these pathways are related to the radioresistance and stemness in pancreatic cancer cells.

As a result of RNA sequence analysis, several genes were identified whose expression in degron (++) cells was much higher than in degron (−) cells. These included matrix metalloproteinase-12 (*MMP12*), Collagen Type IV Alpha 4 Chain (*COL4A4*) and Transmembrane protease serine 3 (*TMPRSS3*). It is reported that MMP12 is upregulated by ultra violet (UV) irradiation in human skin \[[30](https://academic.oup.com/jrr/article/64/2/284/)\]. COL4A4 is one of the components of type IV collagen. There are no reports of a direct relationship between radiation and this gene. However, it is known that radiation induced TGF-β activation \[[31](https://academic.oup.com/jrr/article/64/2/284/)\]. And there is a positive correlation between TGF-β expression and enhanced collagen production. TMPRSS3 is a membrane-bound serine protease overexpressed in pancreatic cancer. This proteinase is of importance for processes involved in tumor invasion \[[32](https://academic.oup.com/jrr/article/64/2/284/)\]. It also reported in another article that it is related with radioresistance \[[33](https://academic.oup.com/jrr/article/64/2/284/)\]. In the present study, we focused on *KRT13* as a candidate radioresistance gene because certain keratins (KRTs) are reportedly shown to have cancer stem-like properties, including *KRT19* in hepatocellular carcinoma and colorectal cancer, *KRT6* in lung cancer and *KRT17* in cervical cancer \[[26–28](https://academic.oup.com/jrr/article/64/2/284/), [34](https://academic.oup.com/jrr/article/64/2/284/)\]. Accordingly, we took notice of *KRT13* among several candidates listed in [Table 1](https://academic.oup.com/jrr/article/64/2/284/). During preparation of this manuscript, other investigators reported that *KRT13* promotes the stemness of breast cancer cells and *KRT13* is associated with radioresistance and stemness in squamous cell carcinoma cells, which further emphasized the importance of *KRT13* in cancer stem cells \[[35](https://academic.oup.com/jrr/article/64/2/284/), [36](https://academic.oup.com/jrr/article/64/2/284/)\].

The KRTs are a family of intermediate filament proteins that constitute the cytoskeleton of epithelial cells. KRTs are often used in histopathology to determine cancer cell origins and aid the prognosis of several malignancies \[[37](https://academic.oup.com/jrr/article/64/2/284/), [38](https://academic.oup.com/jrr/article/64/2/284/)\]. Although the maintenance of cellular structure is well-known function of KRTs, recent studies have suggested that KRTs are involved in functions such as apoptosis, cell growth and cell motility in cancer. For example, abnormal expression of *KRT8* and *KRT18* is related to tumor progression and invasion in squamous carcinomas \[[39](https://academic.oup.com/jrr/article/64/2/284/)\]. *KRT17* has been identified as an oncogene in cervical cancer \[[40](https://academic.oup.com/jrr/article/64/2/284/)\]. Regarding CSCs, high *KRT19* expression is correlated with cancer stemness and radioresistance in hepatocellular carcinoma and colorectal cancer \[[27](https://academic.oup.com/jrr/article/64/2/284/), [28](https://academic.oup.com/jrr/article/64/2/284/)\]. In contrast, cells expressing lower levels of *KRT19* displayed increased level of stem cell markers, colony-formation activity and drug resistance in breast cancer \[[41](https://academic.oup.com/jrr/article/64/2/284/), [42](https://academic.oup.com/jrr/article/64/2/284/)\]. In our study, *KRT19* expression was not increased in degron (++) cells (data not shown).

*KRT13* is a type I keratin expressed in the suprabasal layers of noncornified squamous epithelia such as those of the oral cavity, tonsils and esophagus \[[43](https://academic.oup.com/jrr/article/64/2/284/)\]. In cancer, diverse clinical relevance and functions of *KRT13* have been reported. *KRT13* expression is decreased in oral and cervical cancer tissue \[[44](https://academic.oup.com/jrr/article/64/2/284/), [45](https://academic.oup.com/jrr/article/64/2/284/)\], and the gene is epigenetically silenced in an oral squamous cell carcinoma cell line and in invasive bladder cancer tissue \[[46](https://academic.oup.com/jrr/article/64/2/284/), [47](https://academic.oup.com/jrr/article/64/2/284/)\]. On the other hand, *KRT13* -overexpressing prostate cancer cell lines are reported to be highly migratory, with high expression of genes related to the epithelial-mesenchymal transition and CSCs \[[48](https://academic.oup.com/jrr/article/64/2/284/)\]. Another study showed that *KRT13* expression is increased in the tumor zone with a higher degree of stemness in prostate cancer \[[49](https://academic.oup.com/jrr/article/64/2/284/)\]. Notably, a recent study showed that enhanced *KRT13* expression is associated with radioresistance in squamous carcinoma cells, consistent with our current finding in pancreatic cancer \[[36](https://academic.oup.com/jrr/article/64/2/284/)\]. We showed that si *KRT13* treatment significantly enhanced the radiosensitivity of degron (++) cells after irradiation at 4 Gy, but not enhanced at 8 Gy. One possible reason for this result is that 8 Gy irradiation is lethal to cancer cells, even CSCs. To our knowledge, this is the first study to demonstrate the function and clinical relevance of *KRT13* in pancreatic cancer.

Our results indicate that a combination therapy of *KRT13* knockdown and radiation could be an effective strategy in pancreatic cancer. Nucleic acid–based medicines such as antisense oligonucleotides, siRNAs and miRNAs are anticipated as next-generation therapeutics. However, there are several clinical challenges. For example, nucleic acids are unstable in the blood stream and accumulate in the liver and kidney. Effective tumor targeting thus requires a drug delivery system. We developed the super carbonate apatite (sCA) nanoparticle, a pH-sensitive drug delivery system, and showed that siRNA and miRNA can be stably and efficiently transferred to tumors by sCA without any toxicity \[[50–52](https://academic.oup.com/jrr/article/64/2/284/)\]. Although further investigation is required, the combination therapy of sCA-siRNA for *KRT13* and radiation may be a promising therapeutic approach in pancreatic cancer.

## CONFLICT OF INTEREST

The authors declare no potential conflicts of interest.

## FUNDING

This work was supported by a grant from Kagoshima Shinsangyo Sousei Investment Limited Partnership (its general partner is Kagoshima Development Co., Ltd).

## References

1.

Sung

H

,

Ferlay

J

,

Siegel

RL

et al. .

CA Cancer J Clin

2021

;

71

:

209

–

49

.

2.

Hall

WA

,

Goodman

KA

. .

Radiat Oncol

2019

;

14

:

19

–

23

.

3.

Boyle

J

,

Czito

B

,

Willett

C

,

Palta

M

. .

J Gastrointest Oncol

2015

;

6

:

436

–

44

.

4.

Khan

AQ

,

Rashid

K

,

AlAmodi

AA

et al. .

Curr Opin Pharmacol

2020

;

54

:

130

–

41

.

5.

Baek

SJ

,

Ishii

H

,

Tamari

K

et al. .

Oncol Rep

2015

;

34

:

2233

–

7

.

6.

Zhang

X

,

Lin

SH

,

Fang

B

et al. .

J Thorac Oncol

2013

;

8

:

1484

–

91

.

7.

Cui

X

,

Oonishi

K

,

Tsujii

H

et al. .

Cancer Res

2011

;

71

:

3676

–

87

.

8.

Diehn

M

,

Cho

RW

,

Lobo

NA

et al. .

Nature

2009

;

458

:

780

–

3

.

9.

Yang

L

,

Shi

P

,

Zhao

G

et al. .

Signal Transduct Target Ther

2020

;

5

:

8

.

10.

Gzil

A

,

Zarębska

I

,

Bursiewicz

W

et al. .

Mol Biol Rep

2019

;

46

:

6629

–

45

.

11.

Manasanch

EE

,

Orlowski

RZ

. .

Nat Rev Clin Oncol

2017

;

14

:

417

–

33

.

12.

Vlashi

E

,

Kim

K

,

Lagadec

C

et al. .

J Natl Cancer Inst

2009

;

101

:

350

–

9

.

13.

Smith

L

,

Qutob

O

,

Watson

MB

et al.

Neoplasia

2009

;

11

:

1194

–

207

.

14.

Munakata

K

,

Uemura

M

,

Tanaka

S

et al. .

Clin Cancer Res

2016

;

22

:

5277

–

86

.

15.

Tamari

K

,

Hayashi

K

,

Ishii

H

et al. .

Int J Oncol

2014

;

45

:

2349

–

54

.

16.

Hayashi

K

,

Tamari

K

,

Ishii

H

et al. .

Int J Oncol

2014

;

45

:

2468

–

74

.

17.

Lagadec

C

,

Vlashi

E

,

Bhuta

S

et al. .

BMC Cancer

2014

;

14

:

152

.

18.

Vlashi

E

,

Lagadec

C

,

Chan

M

et al. .

Breast Cancer Res Treat

2013

;

141

:

197

–

203

.

19.

Adikrisna

R

,

Tanaka

S

,

Muramatsu

S

et al. .

Gastroenterology

2012

;

143

:

234

–

245.e7

.

20.

Della

DL

,

Lagadec

C

,

Pajonk

F

. .

Prostate

2012

;

72

:

868

–

74

.

21.

Pan

J

,

Zhang

Q

,

Wang

Y

,

You

M

. .

PLoS One

2010

;

5

:

e13298

.

22.

Adikrisna

R

,

Tanaka

S

,

Muramatsu

S

et al. .

Gastroenterology

2012

;

143

:

234

–

245.e7

.

23.

Wang

JQ

,

Yokoyama

Y

,

Hirose

H

et al. .

Int J Oncol

2022

;

60

:

13

.

24.

Morimoto

Y

,

Mizushima

T

,

Wu

X

et al. .

Br J Cancer

2020

;

122

:

1037

–

49

.

25.

Anaya

J

. .

PeerJ Comput Sci

2016

;

2

:

e67

.

26.

Yang

B

,

Zhang

W

,

Zhang

M

et al. .

Technol Cancer Res Treat

2020

;

19

:

1533033820921248

.

27.

Kawai

T

,

Yasuchika

K

,

Ishii

T

et al. .

Clin Cancer Res

2015

;

21

:

3081

–

91

.

28.

Asfaha

S

,

Hayakawa

Y

,

Rustgi

AK

et al. .

Cell Stem Cell

2015

;

16

:

627

–

38

.

29.

Arumugam

A

,

Weng

Z

,

Chaudhary

SC

et al. .

Biochem Biophys Res Commun

2014

;

451

:

394

–

401

.

30.

Tewari

A

,

Grys

K

,

Kollet

J

et al. .

J Invest Dermatol

2014

;

134

:

2598

–

609

.

31.

Barcellos-Hoff

MH

,

Derynck

R

,

Tsang

ML

,

Weatherbee

JA

. .

J Clin Invest

1994

;

93

:

892

–

9

.

32.

Wallrapp

C

,

Hähnel

S

,

Müller-Pillasch

F

et al. .

Cancer Res

2000

;

60

:

2602

–

6

.

33.

Wen

JY

,

Fang

YY

,

Chen

G

et al. .

Biomark Med

2022

;

16

:

693

–

715

.

34.

Wu

L

,

Han

L

,

Zhou

C

et al. .

FEBS J

2017

;

284

:

3000

–

17

.

35.

Yin

L

,

Li

Q

,

Mrdenovic

S

et al. .

Breast Cancer Res

2022

;

24

:

7

.

36.

Nguyen

TQ

,

Hamada

A

,

Yamada

K

et al. .

In Vitro Cell Dev Biol Anim

2021

;

57

:

300

–

14

.

37.

Moll

R

,

Divo

M

,

Langbein

L

. .

Histochem Cell Biol

2008

;

129

:

705

–

33

.

38.

Karantza

V

. .

Oncogene

2011

;

30

:

127

–

38

.

39.

Alam

H

,

Kundu

ST

,

Dalal

SN

et al. .

J Cell Sci

2011

;

124

:

2096

–

106

.

40.

Escobar-hoyos

LF

,

Shah

R

,

Vanner

EA

et al. .

Cancer Res

2015

;

75

:

3650

–

62

.

41.

Saha

SK

,

Yin

Y

,

Chae

H-S

,

Cho

SG

. .

Cancers (Basel)

2019

;

11

:

99

.

42.

Saha

SK

,

Kim

K

,

Yang

GM

et al. .

Int J Mol Sci

2018

;

19

:

1423

.

43.

Malecha

MJ

,

Miettinen

M

. .

Virchows Arch A Pathol Anat Histopathol

1991

;

418

:

249

–

54

.

44.

Carrilho

C

,

Alberto

M

,

Buane

L

,

David

L

. .

Hum Pathol

2004

;

35

:

546

–

51

.

45.

Ida-yonemochi

H

,

Maruyama

S

,

Kobayashi

T

. .

Mod Pathol

2012

;

25

:

784

–

94

.

46.

Marsit

CJ

,

Houseman

EA

,

Christensen

BC

et al. .

PLoS One

2010

;

5

:

e12334

.

47.

Naganuma

K

,

Hatta

M

,

Ikebe

T

,

Yamazaki

J

. .

BMC Cancer

2014

;

14

:

988

.

48.

Li

Q

,

Yin

L

,

Jones

LW

et al. .

Oncotarget

2016

;

7

:

84645

–

57

.

49.

Liu

S

,

Cadaneanu

RM

,

Zhang

B

et al. .

PLoS One

2016

;

11

:

e0163232

.

50.

Wu

X

,

Yamamoto

H

,

Nakanishi

H

et al. .

PLoS One

2015

;

10

:

e0116022

.

51.

Hiraki

M

,

Nishimura

J

,

Takahashi

H

et al. .

Mol Ther Nucleic Acids

2015

;

4

:

e231

.

52.

Inoue

A

,

Mizushima

T

,

Wu

X

et al. .

Mol Cancer Ther

2018

;

17

:

977

–

87

.

## Supplementary data

[Supplementary\_Fig\_1\_rrac091](https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/jrr/64/2/10.1093_jrr_rrac091/1/supplementary_fig_1_rrac091.jpeg?Expires=1752211062&Signature=1~unGzTnGI~vmlYfYlU7w63KWjamsjKSDMnJn988Vd-gElrKRwj4Kdo~W1GGPHHY9306iOEbm9zXdjrsE~17TODjyx0MoAKKiVMXJw0WFW1Ox5yRsq4nAwCoTEJMxu2Fcvgg~RJJ2-ePML0UCN85jdulb8jAEuxjs~nbiW22Pl6uMVIBFl38zgwRr7O9hOaCt0OSXXbuZnIaPGTOgceV4MP-eRcXrxaUd5ADbphmbMJTq4dO7FbsD4i1uH0JLuG~v5VQ-u5cvS3o-fAu9lcRvc8UP04iMyoknLlUDwgA03MURKy-JIhbkmhOGyIDGF2nILVWNwNJNc0DDpcdrVrrCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA) - jpeg file

[Supplementary\_Fig\_2\_rrac091](https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/jrr/64/2/10.1093_jrr_rrac091/1/supplementary_fig_2_rrac091.jpeg?Expires=1752211062&Signature=JFc-I7zWfficN9vz6A3xigYTVFF1f89p5lV2HthQTmPbcL0FXEJbt5yMg16g6~eJGN3rjNGjmKhv3Nt5xP6yvRikFUReCUoOs2gDGXo5otvk--DHqFINNX0TlRajrNZKsnYs1yKQReG6PGbBd04XZeTKU6RiHWKHm-Dnk5f9wORnrbIzxD2idl7WIpjZGdJ2AnkYLLaL~RRE55Qflvh~-8BVRk3Gl1z3JOCr4JtpMG77MpmpoY1942X0mgv-YgZ-4oq~53osr3kJLzTbjtsmbh9hZ6kYr65HvLKnxKzwjZskW-cIXpc-8Zb60Zaa5xFtF~QXnV3V6GsWppQ9qf~okw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA) - jpeg file

[Supplementary\_Table1\_rrac091](https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/jrr/64/2/10.1093_jrr_rrac091/1/supplementary_table1_rrac091.xlsx?Expires=1752211062&Signature=Ea9xWh2A1rTIVDoAT9a79Ich4IxIe91HUMDPZIBNj3vs8erXi1cMgCfpxErJ5T~HP9GEdxRrUz8zT5QnE~sj~LotyMa0F5ouD9c681gZPCi9Y8rNWKY59ft5b5lMXv90TrPCTAWZizIfL6dYn7EUpHKzqdDdYHdqe-6EvQG0zos~kzvH6-iGhzaDitykJfzOhKtRhl8Bg5CNDnSo~fuThwxofUv4egjPhEVG8EqQzQ0HMqAZeWit61l5IMyWodgPy9RL2U79QkXta0bLrD0LgIDPqS761alGrUDA6LE3eDvkAB19LlDiewwAbiZ1oEAtSCxD3vnyd-hKb6lbRg80tg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA) - xlsx file

[Supplementary\_Table2\_rrac091](https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/jrr/64/2/10.1093_jrr_rrac091/1/supplementary_table2_rrac091.xlsx?Expires=1752211062&Signature=AKRoe4vzhgkwGl5HwNa6S9jMKrGmOFPy5tfq9bHeO53ir7gOUU~avh4oUXroTeBqWVc~rQLEpVCyD0xD-szxN5Q6fZHqTFTWQlPUUF4ZEkjC4lJawSSLG6ixa0j1A0FWL8XgHTYLVrNYM4qJySR-TzcyprZHx0JJvDOw44eaAgLh9Y~4x7KsYSurO4P2KTjSBFJWqi2unnotxax2RjQdNE-qvMymQbAAu5PwoTQEVYPk2YJgF20JPnRA2LJubZMaTt3gOeUarx0vbDuhtRhyDx6rXeSblq7HphsCtbJtwwEkLkOXZ~jijpPA~aaSmW8qgmK2iz~3wfdamLoKSu-oEA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA) - xlsx file

[Supplementary\_Figures\_in\_MS\_word\_file\_rrac091](https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/jrr/64/2/10.1093_jrr_rrac091/1/supplementary_figures_in_ms_word_file_rrac091.docx?Expires=1752211062&Signature=OfMDw4M2-126Y2FuPWxHicdqOiwHD6pa8g0ef7Toq0iH054bBWeLc-CMNBvYo2xI5s5xvPC4Ei8YZy5fx3dpRDfNK-yFYEUAJM68R0WJGQ1RjtcltapRy0PmrR7Xl6vuWGxlao3gIuoepzAbJFpjP0K0YXSwLATOiW2Le2GBV1BZEpTDKxlG2ML6qkBpNdsp8Ij25G06e-r8eMm3u4ebeeNkF9OObY1CtadU2Q~oRQIC9JHgERmS7oKfoC9mQQAfUyf9CJSLFEsPjDUKHMih~s-ze6QfoCSUFthud2JyAHklPiOr0O690YG5Zm7M9fGvaixRiWi0OXddlrlyA5e6aw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA) - docx file

[PDF](https://scholar.google.com/scholar_url?url=https://academic.oup.com/jrr/article-pdf/64/2/284/49600545/rrac091.pdf&hl=ja&sa=T&oi=ucasa&ct=ufr&ei=-XlCaPCBIvCuieoP583H-QE&scisig=AAZF9b8CvgPVFu2PR4dY209bzEiL) [Help](https://scholar.google.com/scholar/help.html#access)