---
citation_key: yinL2022BreastCancerRes
citation_metadata:
  last_updated: '2025-06-16T08:16:07.781829'
  mapping_version: '2.0'
  references_bib_path: test_output/latest/workspace/Clippings/yinL2022BreastCancerRes/references.bib
  source_bibtex: references.bib
  total_citations: 51
citations:
  1:
    authors: D Acehan
    citation_key: dacehan2008
    doi: 10.1038/jid.2008.141
    journal: J Invest Dermatol
    number: 1
    title: Acehan D, Petzold C, Gumper I, Sabatini DD, Muller EJ, Cowin P, et al.
      Plakoglobin is required for e...
    year: '2008'
  2:
    authors: Z Aktary
    citation_key: zaktary2017
    doi: 10.18632/oncotarget.15650
    journal: Oncotarget
    number: 2
    title: 'Aktary Z, Alaee M, Pasdar M. Beyond cell-cell adhesion: Plakoglobin and
      the regulation of tumorigene...'
    year: '2017'
  3:
    authors: Z Aktary
    citation_key: zaktary2013
    doi: ''
    journal: J Cell Sci
    number: 3
    title: Aktary Z, Kulak S, Mackey J, Jahroudi N, Pasdar M. Plakoglobin interacts
      with the transcription fact...
    year: '2013'
  4:
    authors: Z Aktary
    citation_key: zaktary2012
    doi: 10.1155/2012/189521
    journal: Int J Cell Biol.
    number: 4
    title: 'Aktary Z, Pasdar M. Plakoglobin: role in tumorigenesis and metastasis.
      Int J Cell Biol. 2012;2012:18...'
    year: '2012'
  5:
    authors: M Alaee
    citation_key: malaee2018
    doi: 10.1111/cas.13612
    journal: Cancer Sci
    number: 5
    title: Alaee M, Nool K, Pasdar M. Plakoglobin restores tumor suppressor activity
      of p53(R175H) mutant by se...
    year: '2018'
  6:
    authors: I Baccelli
    citation_key: ibaccelli2013
    doi: 10.1038/nbt.2576
    journal: Nat Biotechnol
    number: 6
    title: Baccelli I, Schneeweiss A, Riethdorf S, Stenzinger A, Schillert A, Vogel
      V, et al. Identification of...
    year: '2013'
  7:
    authors: F Bertucci
    citation_key: fbertucci2007
    doi: 10.1056/NEJMc070393
    journal: N Engl J Med
    number: 7
    title: Bertucci F, Cervera N, Birnbaum D. A gene signature in breast cancer. N
      Engl J Med. 2007;356(18):188...
    year: '2007'
  8:
    authors: YJ Chen
    citation_key: yjchen2013
    doi: 10.1371/journal.pone.0064088
    journal: PLoS ONE
    number: 8
    title: Chen YJ, Lee LY, Chao YK, Chang JT, Lu YC, Li HF, et al. DSG3 facilitates
      cancer cell growth and inv...
    year: '2013'
  9:
    authors: KJ Cheung
    citation_key: kjcheung2016
    doi: 10.1126/science.aaf6546
    journal: Science
    number: 9
    title: 'Cheung KJ, Ewald AJ. A collective route to metastasis: Seeding by tumor
      cell clusters. Science. 2016...'
    year: '2016'
  10:
    authors: GC Chu
    citation_key: gcchu2014
    doi: 10.1007/s10555-013-9488-7
    journal: Cancer Metastasis Rev
    number: 10
    title: Chu GC, Chung LW. RANK-mediated signaling network and cancer metastasis.
      Cancer Metastasis Rev. 2014...
    year: '2014'
  11:
    authors: GC Chu
    citation_key: gcchu2014
    doi: 10.1530/ERC-13-0548
    journal: Endocr Relat Cancer
    number: 11
    title: Chu GC, Zhau HE, Wang R, Rogatko A, Feng X, Zayzafoon M, et al. RANK- and
      c-Met-mediated signal netw...
    year: '2014'
  12:
    authors: A de Bruin
    citation_key: adebruin2007
    doi: 10.1111/j.1600-0625.2007.00557.x
    journal: Exp Dermatol
    number: 12
    title: de Bruin A, Caldelari R, Williamson L, Suter MM, Hunziker T, Wyder M, et
      al. Plakoglobin-dependent d...
    year: '2007'
  13:
    authors: J Dittmer
    citation_key: jdittmer2018
    doi: 10.1016/j.semcancer.2018.07.007
    journal: Semin Cancer Biol
    number: 13
    title: 'Dittmer J. Breast cancer stem cells: features, key drivers and treatment
      options. Semin Cancer Biol....'
    year: '2018'
  14:
    authors: I Fabregat
    citation_key: ifabregat2016
    doi: 10.3390/jcm5030037
    journal: J Clin Med
    number: 14
    title: Fabregat I, Malfettone A, Soukupova J. New insights into the crossroads
      between EMT and stemness in ...
    year: '2016'
  15:
    authors: O Govaere
    citation_key: ogovaere2014
    doi: 10.1136/gutjnl-2012-304351
    journal: Gut
    number: 15
    title: 'Govaere O, Komuta M, Berkers J, Spee B, Janssen C, de Luca F, et al. Keratin
      19: a key role player i...'
    year: '2014'
  16:
    authors: MJ Hendrix
    citation_key: mjhendrix1996
    doi: 10.1007/BF00054016
    journal: Cancer Metastasis Rev
    number: 16
    title: Hendrix MJ, Seftor EA, Chu YW, Trevor KT, Seftor RE. Role of intermediate
      filaments in migration, in...
    year: '1996'
  17:
    authors: I Holen
    citation_key: iholen2012
    doi: 10.1186/bcr3201
    journal: Breast Cancer Res
    number: 17
    title: Holen I, Whitworth J, Nutter F, Evans A, Brown HK, Lefley DV, et al. Loss
      of plakoglobin promotes de...
    year: '2012'
  18:
    authors: P Hu
    citation_key: phu2011
    doi: 10.1371/journal.pone.0028670
    journal: PLoS ONE
    number: 18
    title: Hu P, Chu GC, Zhu G, Yang H, Luthringer D, Prins G, et al. Multiplexed
      quantum dot labeling of activ...
    year: '2011'
  19:
    authors: WY Hu
    citation_key: wyhu2017
    doi: 10.1016/j.scr.2017.06.009
    journal: Stem Cell Res
    number: 19
    title: Hu WY, Hu DP, Xie L, Li Y, Majumdar S, Nonn L, et al. Isolation and functional
      interrogation of adul...
    year: '2017'
  20:
    authors: T Kawai
    citation_key: tkawai2015
    doi: 10.1158/1078-0432.CCR-14-1936
    journal: Clin Cancer Res
    number: 20
    title: Kawai T, Yasuchika K, Ishii T, Katayama H, Yoshitoshi EY, Ogiso S, et al.
      Keratin 19, a cancer stem ...
    year: '2015'
  21:
    authors: S Kim
    citation_key: skim2006
    doi: 10.1038/nature04659
    journal: Nature
    number: 21
    title: Kim S, Wong P, Coulombe PA. A keratin cytoskeletal protein regulates protein
      synthesis and epithelia...
    year: '2006'
  22:
    authors: R Koyama-Nasu
    citation_key: rkoyamanasu2013
    doi: 10.1371/journal.pone.0053710
    journal: PLoS ONE
    number: 22
    title: Koyama-Nasu R, Takahashi R, Yanagida S, Nasu-Nishimura Y, Oyama M, Kozuka-Hata
      H, et al. The cancer ...
    year: '2013'
  23:
    authors: YH Lai
    citation_key: yhlai2011
    doi: 10.1186/1471-2121-12-50
    journal: BMC Cell Biol
    number: 23
    title: Lai YH, Cheng J, Cheng D, Feasel ME, Beste KD, Peng J, et al. SOX4 interacts
      with plakoglobin in a W...
    year: '2011'
  24:
    authors: Q Li
    citation_key: qli2016
    doi: 10.18632/oncotarget.13175
    journal: Oncotarget
    number: 24
    title: Li Q, Yin L, Jones LW, Chu GC, Wu JB, Huang JM, et al. Keratin 13 expression
      reprograms bone and bra...
    year: '2016'
  25:
    authors: X Li
    citation_key: xli2016
    doi: 10.18632/oncotarget.7331
    journal: Oncotarget
    number: 25
    title: Li X, Wu JB, Li Q, Shigemura K, Chung LW, Huang WC. SREBP-2 promotes stem
      cell-like properties and m...
    year: '2016'
  26:
    authors: S Liu
    citation_key: sliu2016
    doi: 10.1371/journal.pone.0163232
    journal: PLoS ONE
    number: 26
    title: Liu S, Cadaneanu RM, Zhang B, Huo L, Lai K, Li X, et al. Keratin 13 is
      enriched in prostate tubule-i...
    year: '2016'
  27:
    authors: PK Lo
    citation_key: pklo2012
    doi: 10.1038/onc.2011.439
    journal: Oncogene
    number: 27
    title: Lo PK, Kanojia D, Liu X, Singh UP, Berger FG, Wang Q, et al. CD49f and
      CD61 identify Her2/neu-induce...
    year: '2012'
  28:
    authors: J Madoz-Gurpide
    citation_key: jmadozgurpide2007
    doi: 10.1074/mcp.M700006-MCP200
    journal: Mol Cell Proteomics
    number: 28
    title: Madoz-Gurpide J, Canamero M, Sanchez L, Solano J, Alfonso P, Casal JI.
      A proteomics analysis of cell...
    year: '2007'
  29:
    authors: MJ Malecha
    citation_key: mjmalecha1991
    doi: 10.1007/BF01606063
    journal: Virchows Arch A Pathol Anat Histopathol
    number: 29
    title: Malecha MJ, Miettinen M. Expression of keratin 13 in human epithelial neoplasms.
      Virchows Arch A Pat...
    year: '1991'
  30:
    authors: R Moll
    citation_key: rmoll2008
    doi: 10.1007/s00418-008-0435-6
    journal: Histochem Cell Biol
    number: 30
    title: 'Moll R, Divo M, Langbein L. The human keratins: biology and pathology.
      Histochem Cell Biol. 2008;129...'
    year: '2008'
  31:
    authors: R Nair
    citation_key: rnair2014
    doi: 10.1038/onc.2013.368
    journal: Oncogene
    number: 31
    title: Nair R, Roden DL, Teo WS, McFarland A, Junankar S, Ye S, et al. c-Myc and
      Her2 cooperate to drive a ...
    year: '2014'
  32:
    authors: CM Perou
    citation_key: cmperou2000
    doi: 10.1038/35021093
    journal: Nature
    number: 32
    title: Perou CM, Sørlie T, Eisen MB, van de Rijn M, Jeffrey SS, Rees CA, et al.
      Molecular portraits of huma...
    year: '2000'
  33:
    authors: D Pradella
    citation_key: dpradella2017
    doi: 10.1186/s12943-016-0579-2
    journal: Mol Cancer
    number: 33
    title: 'Pradella D, Naro C, Sette C, Ghigna C. EMT and stemness: flexible processes
      tuned by alternative spl...'
    year: '2017'
  34:
    authors: E Provenzano
    citation_key: eprovenzano2018
    doi: 10.1016/j.cpet.2018.02.004
    journal: PET Clin
    number: 34
    title: Provenzano E, Ulaner GA, Chin SF. Molecular Classification of Breast Cancer.
      PET Clin. 2018;13(3):32...
    year: '2018'
  35:
    authors: S Ricardo
    citation_key: sricardo2011
    doi: 10.1136/jcp.2011.090456
    journal: J Clin Pathol
    number: 35
    title: Ricardo S, Vieira AF, Gerhard R, Leitao D, Pinto R, Cameselle-Teijeiro
      JF, et al. Breast cancer stem...
    year: '2011'
  36:
    authors: G Richard
    citation_key: grichard1995
    doi: 10.1038/ng1295-453
    journal: Nat Genet
    number: 36
    title: Richard G, De Laurenzi V, Didona B, Bale SJ, Compton JG. Keratin 13 point
      mutation underlies the her...
    year: '1995'
  37:
    authors: KM Rieger-Christ
    citation_key: kmriegerchrist2005
    doi: 10.1038/sj.bjc.6602651
    journal: Br J Cancer
    number: 37
    title: Rieger-Christ KM, Ng L, Hanley RS, Durrani O, Ma H, Yee AS, et al. Restoration
      of plakoglobin expres...
    year: '2005'
  38:
    authors: EC Robanus-Maandag
    citation_key: ecrobanusmaandag2003
    doi: 10.1002/path.1385
    journal: J Pathol
    number: 38
    title: Robanus-Maandag EC, Bosch CA, Kristel PM, Hart AA, Faneyte IF, Nederlof
      PM, et al. Association of C-...
    year: '2003'
  39:
    authors: SK Saha
    citation_key: sksaha2017
    doi: 10.1038/onc.2016.221
    journal: Oncogene
    number: 39
    title: Saha SK, Choi HY, Kim BW, Dayem AA, Yang GM, Kim KS, et al. KRT19 directly
      interacts with beta-caten...
    year: '2017'
  40:
    authors: S Sheng
    citation_key: ssheng2008
    doi: 10.1016/j.mce.2008.09.022
    journal: Mol Cell Endocrinol
    number: 40
    title: Sheng S, Barnett DH, Katzenellenbogen BS. Differential estradiol and selective
      estrogen receptor mod...
    year: '2008'
  41:
    authors: V Todorovic
    citation_key: vtodorovic2010
    doi: 10.1242/jcs.070391
    journal: J Cell Sci
    number: 41
    title: Todorovic V, Desai BV, Patterson MJ, Amargo EV, Dubash AD, Yin T, et al.
      Plakoglobin regulates cell ...
    year: '2010'
  42:
    authors: N Todorovic-Rakovic
    citation_key: ntodorovicrakovic2012
    doi: 10.1007/s10238-011-0169-y
    journal: Clin Exp Med
    number: 42
    title: Todorovic-Rakovic N, Neskovic-Konstantinovic Z, Nikolic-Vukosavljevic D.
      C-myc as a predictive marke...
    year: '2012'
  43:
    authors: GN van Muijen
    citation_key: gnvanmuijen1986
    doi: 10.1016/0014-4827(86)90429-5
    journal: Exp Cell Res
    number: 43
    title: van Muijen GN, Ruiter DJ, Franke WW, Achtstatter T, Haasnoot WH, Ponec
      M, et al. Cell type heterogen...
    year: '1986'
  44:
    authors: M Vita
    citation_key: mvita2006
    doi: 10.1016/j.semcancer.2006.07.015
    journal: Semin Cancer Biol
    number: 44
    title: Vita M, Henriksson M. The Myc oncoprotein as a therapeutic target for human
      cancer. Semin Cancer Bio...
    year: '2006'
  45:
    authors: A Waseem
    citation_key: awaseem1998
    doi: 10.1016/S0378-1119(98)00297-2
    journal: Gene
    number: 45
    title: Waseem A, Alam Y, Dogan B, White KN, Leigh IM, Waseem NH. Isolation, sequence
      and expression of the ...
    year: '1998'
  46:
    authors: L Williamson
    citation_key: lwilliamson2006
    doi: 10.1038/sj.emboj.7601224
    journal: Embo J
    number: 46
    title: Williamson L, Raess NA, Caldelari R, Zakher A, de Bruin A, Posthaus H,
      et al. Pemphigus vulgaris ide...
    year: '2006'
  47:
    authors: L Yang
    citation_key: lyang2012
    doi: 10.1093/carcin/bgs226
    journal: Carcinogenesis
    number: 47
    title: Yang L, Chen Y, Cui T, Knosel T, Zhang Q, Albring KF, et al. Desmoplakin
      acts as a tumor suppressor ...
    year: '2012'
  48:
    authors: Y Yang
    citation_key: yyang2019
    doi: 10.7717/peerj.6629
    journal: PeerJ
    number: 48
    title: Yang Y, Tang X, Song X, Tang L, Cao Y, Liu X, et al. Evidence for an oncogenic
      role of HOXC6 in huma...
    year: '2019'
  49:
    authors: T Yin
    citation_key: tyin2005
    doi: 10.1073/pnas.0501676102
    journal: Proc Natl Acad Sci U S A
    number: 49
    title: Yin T, Getsios S, Caldelari R, Kowalczyk AP, Muller EJ, Jones JC, et al.
      Plakoglobin suppresses kera...
    year: '2005'
  50:
    authors: X Zhao
    citation_key: xzhao2014
    doi: 10.1186/1471-2407-14-211
    journal: BMC Cancer
    number: 50
    title: Zhao X, Rødland EA, Sørlie T, Vollan HK, Russnes HG, Kristensen VN, et
      al. Systematic assessment of ...
    year: '2014'
  51:
    authors: HE Zhau
    citation_key: hezhau2008
    doi: 10.1007/s10585-008-9183-1
    journal: Clin Exp Metastasis
    number: 51
    title: Zhau HE, Odero-Marah V, Lue HW, Nomura T, Wang R, Chu G, et al. Epithelial
      to mesenchymal transition...
    year: '2008'
created: 2025-06-09
doi: 10.1186/s13058-022-01502-6
last_updated: '2025-06-16T08:16:12.705276'
paper_structure:
  parsed_at: '2025-06-16T08:16:07.614112'
  section_types_found:
  - abstract
  - introduction
  - methods
  - results
  - discussion
  - conclusion
  - unknown
  - references
  - acknowledgments
  sections:
  - end_line: 3
    level: 2
    section_type: abstract
    start_line: 2
    subsections:
    - end_line: 7
      level: 3
      start_line: 4
      title: Background
      word_count: 50
    - end_line: 11
      level: 3
      start_line: 8
      title: Methods
      word_count: 64
    - end_line: 15
      level: 3
      start_line: 12
      title: Results
      word_count: 114
    - end_line: 19
      level: 3
      start_line: 16
      title: Conclusions
      word_count: 34
    title: Abstract
    word_count: 2
  - end_line: 31
    level: 2
    section_type: introduction
    start_line: 20
    subsections: []
    title: Background
    word_count: 542
  - end_line: 33
    level: 2
    section_type: methods
    start_line: 32
    subsections:
    - end_line: 37
      level: 3
      start_line: 34
      title: Cell culture
      word_count: 89
    - end_line: 41
      level: 3
      start_line: 38
      title: Plasmids, siRNA transfection and viral transduction
      word_count: 140
    - end_line: 45
      level: 3
      start_line: 42
      title: Cell proliferation and behavioral assays
      word_count: 144
    - end_line: 49
      level: 3
      start_line: 46
      title: Cell fractionation and western blot analysis
      word_count: 104
    - end_line: 53
      level: 3
      start_line: 50
      title: Quantitative reverse transcription and polymerase chain reaction (qRT-PCR)
        and RNA sequencing (RNA-seq) analyses
      word_count: 160
    - end_line: 57
      level: 3
      start_line: 54
      title: Clinical specimens
      word_count: 58
    - end_line: 65
      level: 3
      start_line: 58
      title: Immunohistochemical analyses
      word_count: 153
    - end_line: 69
      level: 3
      start_line: 66
      title: Flow cytometry
      word_count: 70
    - end_line: 73
      level: 3
      start_line: 70
      title: Mammosphere assay
      word_count: 101
    - end_line: 77
      level: 3
      start_line: 74
      title: Co-immunoprecipitation (co-IP) analysis
      word_count: 106
    - end_line: 81
      level: 3
      start_line: 78
      title: In vivo experiments
      word_count: 142
    - end_line: 85
      level: 3
      start_line: 82
      title: Statistical analysis
      word_count: 75
    title: Methods
    word_count: 2
  - end_line: 87
    level: 2
    section_type: results
    start_line: 86
    subsections:
    - end_line: 101
      level: 3
      start_line: 88
      title: KRT13 overexpression increases breast cancer proliferation, migration
        and invasion in vitro, and promotes tumorigenesis and metastasis in vivo
      word_count: 347
    - end_line: 113
      level: 3
      start_line: 102
      title: KRT13 overexpression induces epithelial to mesenchymal transition (EMT)
        and stemness
      word_count: 323
    - end_line: 121
      level: 3
      start_line: 114
      title: KRT13 induces aggressiveness through activation of c-Myc
      word_count: 149
    - end_line: 131
      level: 3
      start_line: 122
      title: KRT13 interacts with PG and decreases its expression and nuclear translocation
      word_count: 293
    - end_line: 141
      level: 3
      start_line: 132
      title: KRT13 expression is correlated with c-Myc levels in clinical breast cancer
      word_count: 240
    title: Results
    word_count: 2
  - end_line: 157
    level: 2
    section_type: discussion
    start_line: 142
    subsections: []
    title: Discussion
    word_count: 960
  - end_line: 161
    level: 2
    section_type: conclusion
    start_line: 158
    subsections: []
    title: Conclusions
    word_count: 52
  - end_line: 165
    level: 2
    section_type: unknown
    start_line: 162
    subsections: []
    title: Availability of data and materials
    word_count: 11
  - end_line: 203
    level: 2
    section_type: unknown
    start_line: 166
    subsections: []
    title: Abbreviations
    word_count: 35
  - end_line: 205
    level: 2
    section_type: references
    start_line: 204
    subsections: []
    title: References
    word_count: 2
  - end_line: 209
    level: 2
    section_type: acknowledgments
    start_line: 206
    subsections: []
    title: Acknowledgements
    word_count: 29
  - end_line: 213
    level: 2
    section_type: unknown
    start_line: 210
    subsections: []
    title: Funding
    word_count: 29
  - end_line: 215
    level: 2
    section_type: unknown
    start_line: 214
    subsections:
    - end_line: 219
      level: 3
      start_line: 216
      title: Ethics approval and consent to participate
      word_count: 39
    - end_line: 223
      level: 3
      start_line: 220
      title: Consent for publication
      word_count: 12
    - end_line: 227
      level: 3
      start_line: 224
      title: Competing interests
      word_count: 12
    title: Ethics declarations
    word_count: 3
  - end_line: 229
    level: 2
    section_type: unknown
    start_line: 228
    subsections:
    - end_line: 233
      level: 3
      start_line: 230
      title: Publisher's Note
      word_count: 18
    title: Additional information
    word_count: 3
  - end_line: 235
    level: 2
    section_type: unknown
    start_line: 234
    subsections:
    - end_line: 239
      level: 3
      start_line: 236
      title: Additional file 1
      word_count: 22
    title: Supplementary Information
    word_count: 3
  - end_line: 344
    level: 2
    section_type: unknown
    start_line: 240
    subsections: []
    title: Rights and permissions
    word_count: 1600
  total_sections: 15
processing_status:
  ai_citation_support: completed
  fetch: completed
  organize: completed
  section_parsing: completed
  sync: completed
  tagger: completed
source: https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6
sync_metadata:
  auto_corrections_applied: 0
  checked_at: '2025-06-16T08:16:06.535326'
  consistency_status: validated
  corrections_applied: []
  issues_detected: 0
tag_quality:
  content_relevance: 0.7407407407407408
  evaluation_timestamp: '2025-06-16T08:16:12.665940'
  format_compliance: 1.0
  has_suggestions: true
  quality_score: 0.8296296296296297
  tag_count: 18
tags:
- breast_cancer
- keratin_proteins
- krt13_gene
- cancer_metastasis
- stem_cell_biology
- oncogenic_mechanisms
- plakoglobin_regulation
- c_myc_signaling
- epithelial_mesenchymal_transition
- tumor_progression
- cell_migration
- nuclear_translocation
- desmosomal_complex
- cancer_stem_cells
- gene_expression_regulation
- protein_interaction
- breast_cancer_markers
- molecular_oncology
title: KRT13 promotes stemness and drives metastasis in breast cancer through a plakoglobin/c-Myc
  signaling pathway
workflow_version: '3.2'
---

## Abstract

### Background

Keratins (KRTs) are intermediate filament proteins that interact with multiple regulatory proteins to initiate signaling cascades. Keratin 13 (KRT13) plays an important role in breast cancer progression and metastasis. The objective of this study is to elucidate the mechanism by which KRT13 promotes breast cancer growth and metastasis.

### Methods

The function and mechanisms of KRT13 in breast cancer progression and metastasis were assessed by overexpression and knockdown followed by examination of altered behaviors in breast cancer cells and in xenograft tumor formation in mouse mammary fat pad. Human breast cancer specimens were examined by immunohistochemistry and multiplexed quantum dot labeling analysis to correlate KRT13 expression to breast cancer progression and metastasis.

### Results

KRT13-overexpressing MCF7 cells displayed increased proliferation, invasion, migration and in vivo tumor growth and metastasis to bone and lung. Conversely, KRT13 knockdown inhibited the aggressive behaviors of HCC1954 cells. At the molecular level, KRT13 directly interacted with plakoglobin (PG, γ-catenin) to form complexes with desmoplakin (DSP). This complex interfered with PG expression and nuclear translocation and abrogated PG-mediated suppression of c-Myc expression, while the KRT13/PG/c-Myc signaling pathway increased epithelial to mesenchymal transition and stem cell-like phenotype. KRT13 expression in 58 human breast cancer tissues was up-regulated especially at the invasive front and in metastatic specimens (12/18) (*p*  < 0.05). KRT13 up-regulation in primary breast cancer was associated with decreased overall patient survival.

### Conclusions

This study reveals that KRT13 promotes breast cancer cell growth and metastasis via a plakoglobin/c-Myc pathway. Our findings reveal a potential novel pathway for therapeutic targeting of breast cancer progression and metastasis.

## Background

Breast cancer is the leading cause of cancer-related death among women worldwide. It is a heterogeneous disease with divergent molecular alterations and cellular changes. Distant metastasis is the final stage of breast cancer progression. The pathobiological features and the exact mechanism are far from being completely understood.

Keratins (KRTs) are considered structural proteins ranging in sizes from 40 to 76 kDa. They are the major component of intermediate filaments (IFs) in the intracytoplasmic cytoskeleton of epithelial and endothelial cells. IFs insert into the electron-dense desmosomal plaques and connect to other IFs to provide tensile strength to cells. Aberrant expression in various cancers makes KRTs useful as biomarkers for differential diagnoses and metastatic status. Recent studies suggest that KRTs in cancer cells are not only epithelial marker proteins but are also mediators capable of interacting with a range of proteins to regulate signaling networks associated with cell death, survival, proliferation, migration, invasion and metastasis \[[^1],[^2],[^3]\].

KRT13, a 54 kDa type I keratin that often pairs with type II keratin 4, is found in the suprabasal layers of non-cornified stratified squamous epithelia of the oral cavity, tonsils, larynx, esophagus, lower genital tract and transitional urothelium \[[^4], [^5]\], where KRT13 is a major component of the basal and intermediate cells and enriched in prostatic tubule-initiating cells \[[^6], [^7]\]. KRT13 expression can be regulated by factors including calcium, nuclear receptor ligands such as retinoids and 1α, 25-dihydroxyvitamin D3, and estrogens or selective estrogen receptor modulators \[[^8], [^9]\]. Point mutation of KRT13 was shown to be correlated with autosomal dominant disorder white sponge nevus \[[^10]\]. Dysregulated KRT13 expression was found in carcinomas of the tongue, head and neck, uterine cervix, mouth, ovary, breast, bladder cancer, esophageal cancer, and prostate cancer \[[^11], [^12]\].

KRT intermediate filaments are involved in the translation of environmental cues into modifications of gene expression \[[^13]\]. KRT is physically linked to nuclear membrane and desmosomes, where plakoglobin (PG) is a multifunctional protein. Also known as γ-catenin, PG is a paralog of β-catenin, both belonging to the Armadillo protein family. As a major component of the adherens junctions and desmosomes, PG coordinates with desmoplakin (DSP) to anchor intermediate filaments to desmosomes \[[^13]\]. PG also participates in cell signaling regulation, but with an antagonistic effect to β-catenin. While β-catenin has a well-defined oncogenic potential through the Wnt signaling pathway, PG exhibits tumor/metastasis suppressor activity \[[^14]\]. Recent evidence suggests that PG in the cytoplasm and nucleus can regulate tumor progression and metastasis \[[^15]\]. PG may implement its regulatory function by competing with β-catenin, interacting with intracellular proteins, or sequestering transcription factors \[[^16], [^17]\].

We previously reported that KRT13 promotes prostate cancer bone and brain metastases through RANKL-independent pathways \[[^18]\] by an undefined mechanism. In this study, we explored the mechanism by which KRT13 promotes breast cancer progression and metastasis. Overexpression of KRT13 led to increased breast cancer cell proliferation, migration and invasion in vitro, and tumor formation and metastasis in vivo, while knockdown of KRT13 resulted in alleviating effects in vitro and in vivo. Importantly, we report for the first time that KRT13 directly interacts with PG/DSP complexes and alters the expression and nuclear translocation of PG, thus modulating downstream c-Myc-signaling. Our findings provide a potential new therapeutic target in breast cancer progression and metastasis.

## Methods

### Cell culture

Human breast cancer cell lines MCF7 (indolent, low endogenous KRT13 expression) and HCC1954 cells (aggressive, high endogenous KRT13 expression) were provided by Dr. X. J. Cui at Cedars-Sinai Medical Center (CSMC). MCF7 cells were maintained in DMEM (ThermoFisher Scientific, Waltham, MA) and HCC1954 was maintained in RPMI1640 (ThermoFisher Scientific). Both culture media were supplemented with 10% fetal bovine serum (Atlanta Biologicals, Flowery Branch, GA), 100 IU/ml penicillin, and 100 μg/ml streptomycin. All cells were maintained at 37 °C in a humidified atmosphere with 5% CO <sub>2</sub>.

### Plasmids, siRNA transfection and viral transduction

For KRT13 overexpression, a cDNA containing the full-length open reading frame of human KRT13 (NM\_153490) was subcloned into pLVX-AcGFP1-N1 (pLV) (Clontech, CA) by introducing EcoR I and BamH I sites. The construct of pLVX-AcGFP1-N1-KRT13 (pLV-KRT13) was confirmed by DNA sequencing. The plasmid DNA was transfected to the 293 T cells to produce lentiviral particles, following the manufacturer’s instructions (System Biosciences, Mountain View, CA). MCF7 cells were transduced with the lentivirus and selected with puromycin (2 µg/ml for 2 weeks). For shRNA-mediated knockdown, non-targeting control or KRT13 shRNA lentiviral particles (Santa Cruz Biotechnology, Dallas, Texas) were used to infect the HCC1954 cells. The plasmid pcDNA3-c-Myc (Addgene, Cambridge, MA) was used to generate c-Myc-overexpressing cells. To silence c-Myc, cells were transfected with c-Myc siRNA (Santa Cruz Biotechnology) using Lipofectamine 2000 transfection reagent (Invitrogen, Carlsbad, CA).

### Cell proliferation and behavioral assays

To determine growth rates, breast cancer cells were seeded on 24-well plates for 5 days, and cell numbers from triplicate wells were counted daily with a TC20 automatic cell counter (Bio-Rad, Hercules, CA). Cancer cell migration was examined in triplicate transwells (8 μm pore size) coated with collagen I. For invasion assays, transwells were coated with collagen I, and each well was overlaid with growth factor reduced Matrigel (BD Biosciences, San Jose, CA) as previously described \[[^19]\]. After incubation at 37 °C for 24–48 h, cells on the upper surface of the filters were removed by cotton swab, and cells that had invaded and migrated to the lower surface were fixed with 10% formaldehyde and stained with 0.5% crystal violet. After washing, stain was eluted with Sorensen’s solution. The absorbance of each well was measured at 590 nm.

### Cell fractionation and western blot analysis

Cytoplasmic and nuclear extracts or whole cell lysates were prepared with NE-PER nuclear and cytoplasmic extraction reagents (ThermoFisher Scientific). Western blot analysis was performed as previously described \[[^19]\]. The primary antibodies used were to KRT13 (EPR3671, Abcam, Cambridge, UK or B-2, Santa Cruz Biotechnology), CD44 (156-3C11, Cell Signaling, Danvers, MA), c-Myc (D84C12, Cell Signaling), ALDH1A1 (B-5, Santa Cruz Biotechnology), Nanog (5A10, Santa Cruz Biotechnology), PG (A-6, Santa Cruz Biotechnology), DSP \[EPR4383(2), Abcam\], Actin (AC-15, Santa Cruz Biotechnology) and Lamin B (C-20, Santa Cruz Biotechnology). Immunoblots were subjected to morphometric analysis by Image Studio Software (LI-COR, Lincoln, NE).

### Quantitative reverse transcription and polymerase chain reaction (qRT-PCR) and RNA sequencing (RNA-seq) analyses

Total RNA was isolated using the RNeasy Mini Kit (Qiagen, Hilden, Germany). For RNA-seq analyses, duplicate samples were submitted to University of California Los Angeles (UCLA) Clinical Microarray Core for RNA-seq analysis. The RNA-seq data were first subjected to differentially gene expression with the DESeq2 program. Gene set enrichment analysis was then performed. For correlation study, RNA-seq gene expression pattern was compared to breast cancer subtypes of The Cancer Genome Atlas ([http://cancergenome.nih.gov](http://cancergenome.nih.gov/)). To assess correlation between breast cancer subtypes and KRT13 overexpression cells, we compared expression centroids of the five defined breast cancer subtypes \[[^20], [^21]\] with the KRT13 overexpressing MCF7 cells. Unsupervised clustering with Pearson’s correlation was performed to display the similarity in the heatmap with dendrogram. To analyze specific gene expression, qRT-PCR was conducted with experimental settings as we previously reported. Sequences of the primer pairs are listed in Additional file [1](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#MOESM1): Table S1.

### Clinical specimens

Formalin-fixed and paraffin-embedded tissue specimens including 41 primary, 11 bone and 10 brain metastatic breast cancer tissues were obtained from West China Hospital, Chengdu, China. Handling of tissue specimens conformed to the policies and practices of the institutions. The use of the specimens was approved by the Institutional Research Board of the CSMC (IRB: Pro00054328).

### Immunohistochemical analyses

Our immunohistochemical (IHC) staining protocol is previously published \[[^22]\]. Primary antibodies to KRT13 (ERP3671, Abcam), PG (A-6, Santa Cruz Biotechnology), and c-Myc (D84C12, Cell Signaling) were used. For this study, images of stained sections were scanned with a Keyence BZ-X710 microscope (Itasca, IL). The stains were scored with combined percentage of positive cells and intensity as reported \[[^23]\].

For immunofluorescence staining, sections were stained with Alexa Fluor 594‐conjugated goat anti‐rabbit IgG (H + L) or Alexa Fluor 488‐conjugated goat anti‐mouse IgG (H + L) secondary antibodies (ThermFisher Scientific). Sections were further counterstained with 4′,6-diamidino-2-phenylindole (DAPI) in mounting medium (Vector Laboratories, Burlingame, CA) and examined by Zeiss Axio Observer Z1 fluorescence microscope.

Multiplexed quantum dot labeling (mQDL) analysis was performed as we previously reported \[[^24]\]. Tissue sections were sequentially incubated with antibodies to KRT13, PG and c-Myc. Areas of interest were defined with manual segmentation by a pathologist (M. S. Lewis).

### Flow cytometry

Flow cytometric analyses were conducted as described previously \[[^23]\]. Cultured cells at 80% confluence were prepared in single cell suspension and fixed in 4% formaldehyde, washed in phosphate buffered saline (PBS), resuspended in PBS containing 1% BSA, and incubated with APC-conjugated anti-CD44 (IM7, Biolegend) and PE-conjugated anti-CD24 (ML5, Biolegend) antibodies at 4 °C for 30 min. Matched APC- or PE- conjugated isotypes were used as negative controls.

### Mammosphere assay

Cells were plated in ultra-low attachment 96-well plates (Corning Inc., Corning, NY) in serum-free DMEM/F12 medium (ThermoFisher Scientific) supplemented with 20 ng/ml epidermal growth factor (ThermoFisher Scientific), 10 ng/ml basic fibroblast growth factor (Pepro Tech, Rocky Hill, NJ), 5 µg/ml insulin (VWR, Radnor, PA), 1 × B-27 supplement (ThermoFisher Scientific) and 0.4% bovine serum albumin (Sigma, St. Louis, MO). For each experimental group, 500 cells in 100 µl medium were plated to each of 10 wells. After cultured for a week \[[^25]\], spheres were counted and documented with phase contrast microscopy. Each study was repeated for three times.

### Co-immunoprecipitation (co-IP) analysis

Cell lysates (50 µg) were precleared by incubating with 1 µg rabbit/mouse control IgG in 20 µl Protein A/G PLUS-Agarose suspension (Santa Cruz Biotechnology) at 4 °C for 1 h. After centrifugation, the supernatant was incubated with primary antibodies at 4 °C overnight, and then with 20 µl of Protein A/G PLUS-Agarose at 4 °C for 2 h. Immunoprecipitates were collected and rinsed four times with cell lysis buffer. Half of each sample was submitted to the Mass Spectrometry and Biomarker Discovery core at CSMC for interactome analysis, and the remaining half was used to verify the interactome results by western blot.

### In vivo experiments

All animal procedures were performed according to a protocol approved by the Institutional Animal Care and Use Committee (IACUC) of the CSMC as previously described \[[^18], [^19]\]. MCF7-KRT13 or HCC1954-shKRT13 cells were tagged for luciferase reporter expression and inoculated orthotopically to the fat pads of mammary glands (3 × 10 <sup>6</sup>  cells/100 µl PBS containing 50% Matrigel) or intracardially to the left ventricle (1 × 10 <sup>6</sup> cells/50 µl PBS) of 6- to 8-week-old female Fox Chase SCID Beige mice (Charles River, Wilmington, MA). Tumor volume and metastasis were monitored and assessed weekly by bioluminescence imaging (BLI) using an IVIS® Spectrum or Lumina XR Optical Imaging System. A Scanco viva CT 40 system was used to examine skeletal lesions. At the end of the studies, mice were euthanized, and tumor tissues and organs were harvested for histopathological analysis.

### Statistical analysis

All assays were done in triplicates for each of the independent experiments. Differences between groups were analyzed using Student’s *t* test (two groups) or one-way ANOVA (three or more groups). For Kaplan–Meier survival analysis, statistical significance was determined by the log-rank test. Other data were analyzed using GraphPad software (GraphPad Prism version 5.01 for Windows). Results are expressed as the mean ± SD. The *p* value < 0.05 was considered statistically significant.

## Results

### KRT13 overexpression increases breast cancer proliferation, migration and invasion in vitro, and promotes tumorigenesis and metastasis in vivo

We previously reported that KRT13 induced prostate cancer progression and metastasis \[[^18]\]. In this study, we investigated the potential role of KRT13 in breast cancer using cell lines that expressed low (MCF7) or high (HCC1954) endogenous KRT13. We established KRT13-overexpressing MCF7 cells (MCF7-KRT13) and knocked down KTR13 levels in HCC1954 cells (HCC1954-shKRT13) (Fig. [1](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig1) a). MCF7-KRT13 cells displayed increased proliferation (Fig. [1](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig1) b), migration (Fig. [1](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig1) c) and invasion (Fig. [1](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig1) d); while HCC1954-shKRT13 cells showed reduced aggressive behaviors. Moreover, MCF7-KRT13 cells inoculated orthotopically to mouse mammary glands showed accelerated tumor growth, forming hemorrhagic tumors quite different from the control MCF7-con tumors (Fig. [2](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig2) a), with significantly increased tumor weight (Fig. [2](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig2) b) and volume (Fig. [2](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig2) c). When inoculated intracardially, MCF7-KRT13 cells colonized multiple sites (Fig. [2](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig2) d) leading to tumor formation in lung, liver, adrenal gland, mesenteric lymph nodes, jaw, and other bones (Fig. [2](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig2) e and Additional file [1](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#MOESM1): Table S2), which were confirmed by IHC analysis (Fig. [2](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig2) f). Interestingly, when MCF7-KRT13 cells were cultured in vitro, high levels of KRT13 could be readily detected by western blotting (Fig. [1](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig1) a); whereas in xenograft tumors, KRT13 overexpression was often seen unevenly by IHC analysis, probably suggesting temporal or spatial modification of the protein in the tumor microenvironment. Bone µCT scans revealed severe osteolytic lesions (Fig. [2](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig2) g), and the mice had significant weight loss (Fig. [2](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig2) h) and shortened survival (F [i](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig2) g. [2](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig2) i).

![figure 1](https://media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13058-022-01502-6/MediaObjects/13058_2022_1502_Fig1_HTML.png?as=webp)

**Fig. 1**

![figure 2](https://media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13058-022-01502-6/MediaObjects/13058_2022_1502_Fig2_HTML.png?as=webp)

**Fig. 2**

Similar correlations between KRT13 levels and xenograft tumor formation were observed with HCC1954 cells. HCC1954-shcon xenograft tumors caused prevalent metastases to the liver, adrenal glands and lung as early as 8 weeks post-inoculation. Conversely, HCC1954-shKRT13 cells formed smaller tumors (Fig. [2](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig2) j) with lower incidence (6/10) (Fig. [2](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig2) k). KRT13 knockdown delayed HCC1954 tumor metastasis, which was only detected by week 19 (Fig. [2](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig2) l). These results indicate strongly that KRT13 is associated with elevated aggressive behaviors in breast cancer cells.

### KRT13 overexpression induces epithelial to mesenchymal transition (EMT) and stemness

We established single cell colonies using MCF7-KRT13 cells and detected KRT13 by immunostaining. Interestingly, the staining was stronger at the extending edge than at the center of the colony (Additional file [1](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#MOESM1): Fig. S1). The edge-positive phenomenon is in accord with our finding that KRT13 is associated with the tumor invasive front in breast cancer tissue specimens (see below).

To investigate the mechanism of KRT13-induced morphologic change and aggressive behavior, we conducted RNA-Seq gene expression profiling with the MCF7-KRT13 cells. Bioinformatic data analysis suggested that KRT13 overexpression had switched the cells from a typical epithelial type to more invasive basal cell-like and Her2-positive subtypes (Additional file [1](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#MOESM1): Fig. S2) \[[^26]\]. Gene set enrichment analysis (GSEA) suggested that KRT13-overexpression induces EMT (Fig. [3](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig3) a) and stemness (Fig. [3](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig3) b) properties. This explains the marked morphologic changes from connected cobble-stone epithelial MCF7 cells to a less cohesive mesenchymal-like arrangement of the MCF7-KRT13 cells (Fig. [3](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig3) c and Additional file [1](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#MOESM1): Fig. S1). MCF7-KRT13 cells were also found with decreased epithelial markers E-cadherin and claudin-7 but with increased mesenchymal markers vimentin and twist. Conversely, claudin-7 were up-regulated and twist was down-regulated in HCC1954-shKRT13 cells (Fig. [3](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig3) d).

![figure 3](https://media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13058-022-01502-6/MediaObjects/13058_2022_1502_Fig3_HTML.png?as=webp)

**Fig. 3**

We explored the relationship of KRT13 to stem cell properties. A panel of stemness-related marker and regulator proteins were determined in MCF7-KRT13 and HCC1954-shKRT13 cells. Overexpression of KRT13 increased the expression of CD44, c-Myc, ALDH1A1 and Nanog, whereas knockdown of KRT13 was seen with decreased expression of these proteins (Fig. [3](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig3) e). MCF7-KRT13 cells displayed a more robust self-renewal capability as reflected by accelerated mammosphere formation and increased mammosphere counts (Fig. [3](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig3) f). In addition, MCF7-KRT13 cells expressed significantly elevated CD44 <sup>+</sup> CD24 <sup>−</sup> stem cell features (Fig. [3](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig3) g), while HCC1954-shKRT13 cells showed slightly decreased CD44 <sup>+</sup> levels (Fig. [3](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig3) h). These results suggest that a high KRT13 level promotes stem cell properties.

### KRT13 induces aggressiveness through activation of c-Myc

We previously reported that KRT13 overexpression in prostate cancer cells was associated with stemness gene expression, including c-Myc \[[^18]\]. Here, we manipulated c-Myc expression in KRT13-overexpressing and KRT13-knockdown breast cancer cells. Specific siRNAs effectively repressed c-Myc expression in MCF7-KRT13 cells (Fig. [4](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig4) a). The c-Myc repression was accompanied by decreased expression of CD44 and Nanog (Fig. [4](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig4) a), reduced number and size of mammosphere formation (Fig. [4](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig4) b), cell growth (Fig. [4](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig4) c), invasion and migration (Fig. [4](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig4) d). In HCC1954-shKRT13 cells, overexpression of c-Myc induced the expression of CD44 and Nanog (Fig. [4](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig4) a) and promoted mammosphere formation (Fig. [4](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig4) b), cell growth (Fig. [4](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig4) e), invasion and migration (Fig. [4](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig4) f). Results from these analyses suggest that c-Myc is a key KRT13 downstream mediator promoting aggressive behavior and stem cell-like properties in breast cancer cells.

![figure 4](https://media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13058-022-01502-6/MediaObjects/13058_2022_1502_Fig4_HTML.png?as=webp)

**Fig. 4**

### KRT13 interacts with PG and decreases its expression and nuclear translocation

To explore how KRT13 elicits oncogenic signaling in cancer progression and metastasis, we carried out studies to identify intracellular proteins that were interactive to KRT13. In co-IP experiments, antibodies to KRT13 were used in MCF7-KRT13 cells to pull down protein complexes, which were characterized with liquid chromatography-tandem mass spectrometry (LC–MS/MS) and spectral counting methods. Bioinformatic analyses suggested 37 pulled-down proteins that were significantly higher in MCF7-KRT13 cells than MCF7-con cells (Additional file [1](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#MOESM1): Table S3). We found abundant DSP and PG co-precipitated with KRT13. In co-IP and western blotting analyses with the MCF7-KRT13 and HCC1954-shKRT13 cells, KRT13 interacted directly with both PG and DSP, because either anti-KRT13, anti-PG or anti-DSP antibody was able to pull down all three proteins (Fig. [5](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig5) a).

![figure 5](https://media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13058-022-01502-6/MediaObjects/13058_2022_1502_Fig5_HTML.png?as=webp)

**Fig. 5**

Throughout these studies, MCF7-KRT13 cells were found with decreased PG and DSP levels, while HCC1954-shKRT13 cells expressed increased PG and DSP (Fig. [5](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig5) a). When subcellular distribution was examined, MCF7-KRT13 cells showed significantly reduced cytoplasmic and nuclear PG. In reverse, HCC1954-shKRT13 cells showed significantly increased nuclear PG with a slight increment of PG in the cytoplasm (Fig. [5](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig5) b). Similar results were detected by immunofluorescence assay (Fig. [5](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig5) c). KRT13 overexpression seemed to suppress PG and DSP protein levels although the mechanism has yet to be elucidated. Among these cytoskeleton proteins, PG is known to be a potential signaling mediator, translocating to the cell nucleus to exert its tumor suppressor function \[[^27]\]. Lowered PG levels upon KRT13 overexpression may contribute to the aggressive behavior observed in MCF7-KRT13 cells. These results suggest that KRT13 is capable of binding with the DSP/PG complex, sequestering and lowering the expression and nuclear translocation of PG to promote oncogenic signaling.

### KRT13 expression is correlated with c-Myc levels in clinical breast cancer

We determined KRT13 expression in 41 primary and 21 metastatic breast cancer tissue specimens by IHC (Fig. [6](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig6) a). KRT13 expression in metastatic specimens (12/21) was significantly higher than in the primary breast cancer (13/41) (Fig. [6](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig6) b). In addition, the Oncomine gene expression database ([www.oncomine.org](http://www.oncomine.org/)) showed that KRT13 expression in primary breast cancer was associated with decreased patient overall survival (Fig. [6](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig6) c), similar to our report on prostate cancer \[[^18]\]. KRT13-positive cancer cells were detected at the invasive front of breast cancer (Fig. [6](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig6) d) as well as prostate cancer (Additional file [1](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#MOESM1): Fig. S3).

![figure 6](https://media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13058-022-01502-6/MediaObjects/13058_2022_1502_Fig6_HTML.png?as=webp)

**Fig. 6**

There was a positive correlation between KRT13 and c-Myc expression in this study and in the Oncomine database (Fig. [6](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig6) e–g). Using three pairs of clinical specimens, we further confirmed the association between KRT13, PG, and c-Myc by mQDL (Fig. [6](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig6) h), as both KRT13 and c-Myc levels were elevated, whereas both cytosolic and nuclear PG levels were lowered in metastatic tumors than in the primary tumor specimens. A high expression and positive nuclear PG was associated with low KRT13 and c-Myc expression, while low cytoplasmic PG expression was associated with moderate-high KRT13 and c-Myc expression (Fig. [6](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig6) h and Additional file [1](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#MOESM1): Fig. S4). These data suggest that KRT13 can downregulate PG expression and reduce its nuclear translocation, abrogate PG-mediated c-Myc suppression, and promote cancer progression.

## Discussion

Breast cancer metastasis is a complex multistep pathologic process. A deeper understanding of the process is critical for improving therapeutic outcomes. KRTs are intermediate filament proteins supporting structural integrity and functions of cells. Recent studies suggest that aberrant expression of KRTs is associated with cancer progression and metastasis. For instance, KRT19 was shown to be a cancer stem cell marker in hepatocellular carcinomas \[[^28]\], associated with poor overall survival \[[^29]\]. KRT14 was positive in leader tumor cell clusters which disseminate collectively in breast cancer metastasis \[[^30]\]. In this study, we identified KRT13 as a novel cancer marker and an oncogenic promoter. Overexpression of KRT13 markedly enhanced breast cancer cell growth, migration, and invasion in vitro (Fig. [1](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig1)) and promoted tumorigenesis and metastasis in athymic mice (Fig. [2](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig2)). Conversely, knockdown of KRT13 reduced proliferation and aggressive behaviors (Figs. [1](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig1) and [2](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig2)). These findings unveil for the first time the important role of KRT13 in promoting breast cancer progression and metastasis. The role is particularly relevant to breast cancer treatment, because KRT13 is a target gene of estrogen receptor α activation, while both estrogen and tamoxifen behave as antagonists of the KRT13 gene \[[^4],[^5],[^6]\].

Recent studies suggested that breast cancer is a stem cell disease, with cancer stem cells critical for cancer dissemination \[[^31], [^32]\]. KRT13 may be associated with stemness because its expression is characteristically localized in suprabasal layers and enriched in prostatic tubule-initiating cells \[[^4],[^5],[^6]\]. KRT13 plays essential roles in maintaining stem cell homeostasis and symmetric self-renewal in the prostate epithelial layer \[[^33]\]. In this study, KRT13 induced stem cell phenotypes in MCF7 breast cancer cells (Fig. [3](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig3)), and increased expression of stemness-related genes of CD44, c-Myc, ALDH1A1 and Nanog. KRT13 promoted the CD44 <sup>+</sup> CD24 <sup>−</sup> phenotype, a combination of breast cancer stem cell markers associated with invasion and poor prognosis \[[^34], [^35]\]. KRT13 also induced the formation of mammospheres and increased self-renewal capability. These studies thus suggested KRT13 as a stemness-related mediator contributing to progression and dissemination.

The proto-oncogene c-Myc plays a key role in tumor progression \[[^36], [^37]\]. Its oncogenic activity is associated with poor prognosis \[[^38]\] and is considered to drive a stem-like phenotype in breast cancer \[[^39]\]. In our study, KRT13-induced stemness, behavioral changes, and xenograft tumor growth and metastasis could be attributed to c-Myc dysregulation. To prove that KRT13 functions through c-Myc up-regulation, we conducted loss and gain of c-Myc function experiments in KRT13-overexpressing and KRT13-knockdown cells, in which siRNA-mediated c-Myc suppression reversed the stemness and growth advantage, while reduced self-renewal and aggressiveness were compensated by c-Myc overexpression (Fig. [4](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig4)). These results suggest that KRT13 can up-regulate c-Myc expression leading to stem cell features and aggressiveness.

Stemness and EMT in cancer cells are in tight crosstalk \[[^40], [^41]\]. KRT13-overexpressing MCF7 cells undertook a less cohesive mesenchymal-like shape markedly different from the epithelial-like parental MCF7 cells (Fig. [3](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig3)). KRT13 overexpression also increased the expression of mesenchymal stromal markers such as vimentin and twist, concomitantly with decreased epithelial markers of E-cadherin and claudin 7. These findings provide evidence that KRT13 activated EMT programs to increase migratory and invasive properties in MCF7 cells.

KRT13 complexed with both DSP and PG to regulate downstream signaling pathways. DSP, an obligate component of desmosomal plaques, connects the desmosomal cadherin/PG/plakophilin (PKP) complex to intermediate filaments. In human lung cancer, DSP was observed to enhance PG expression and act as a tumor suppressor \[[^27]\]. PG is also a major component of the adherens junctions and desmosomes mediating cell–cell adhesion. Loss of PG reduces cell–cell contact to promote cancer invasion and dissemination \[[^14], [^42],[^43],[^44]\]. In our study, KRT13-overexpression led to decreased DSP and PG, reducing tumor suppressor functions in MCF7-KRT13 cells.

Importantly, PG may be translocated to the nucleus. Both cytoplasmic and nuclear PGs could function to suppress tumor growth and metastasis \[[^14], [^45]\] by modulating the expression of genes involved in stemness, cell-cycle control and cell invasion \[[^16], [^17]\]. The expressions of some important regulatory proteins, such as P53 \[[^15]\], c-Myc \[[^46]\], SOX4 \[[^47]\] and CD133 \[[^48]\], are all under PG modulation. Nuclear PG is capable of interacting with Tcf/Lef transcription factors to inhibit downstream genes in the Wnt signaling pathway, including c-Myc \[[^49], [^50]\]. In our study, PG expression and nuclear translocation were reduced following KRT13-overexpression (Fig. [5](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig5)). The reduced PG level may relieve inhibition of c-Myc gene expression, resulting in up-regulated c-Myc driving cancer EMT, stemness and tumor metastasis (Fig. [6](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig6)). Conversely, KRT13 silencing increased PG expression and nuclear translocation, decreasing EMT, stemness, tumor growth and metastasis. This probability was further strengthened by single cell mQDL staining of clinical breast cancer tissues, which demonstrated that KRT13 expression was inversely correlated with PG, which was inversely correlated with c-Myc (Fig. [6](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#Fig6)). The molecular mechanisms by which PG regulates c-Myc levels in breast cancer cells need to be investigated further.

We reported previously that forced KRT13 expression drove prostate cancer metastasis through a RANKL-independent mechanism \[[^18]\], although RANKL-mediated signal network was shown to be associated with EMT, stemness, neuroendocrine or neuromimicry, osteomimicry and tumor cell bone colonization \[[^51]\]. In this study, we obtained results indicating that KRT13 did not affect RANKL, RANK, or OPG levels (Additional file [1](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#MOESM1): Fig. S5) in breast cancer cells, suggesting that KRT13-induced breast cancer progression and metastasis may be regulated by a RANKL-independent mechanism. On the other hand, it was surprising that KRT13-overexpression in prostate cancer cells elicited both bone and brain metastases, while no brain dissemination was detected in the breast cancer mouse model in this study, even following intracardiac inoculation. Co-incidentally, the expression of neuromimicry genes including CgA and NSE, elevated in prostate cancer KRT13-overexpresing cells, was not elevated in breast cancer MCF7-KRT13 cells (Additional file [1](https://breast-cancer-research.biomedcentral.com/articles/10.1186/s13058-022-01502-6#MOESM1): Fig. S5). A detailed comparative study of KRT13 function between breast and prostate cancer is warranted.

## Conclusions

In summary, this is the first study reporting that KRT13 interacts with PG/DSP complexes to downregulate the expression and nuclear translocation of PG, which suppresses c-Myc-dependent signaling pathways and consequently inhibits EMT, stem cell-like properties and metastasis. Our findings thus identify potential therapeutic targets for breast cancer progression and metastasis.

## Availability of data and materials

Available upon request to leland.chung@cshs.org.

## Abbreviations

IFs:

Intermediate filaments

qRT-PCR:

Quantitative real-time polymerase chain reaction

RNA-seq:

RNA sequencing

IHC:

Immunohistochemical

PBS:

Phosphate buffered saline

co-IP:

Co-immunoprecipitation

GSEA:

Gene set enrichment analysis

mQDL:

Multiplexed quantum dot labeling

EMT:

Mesenchymal transition

## References

## Acknowledgements

We thank Xiaojiang Cui and Bingchen Han (Cedars-Sinai Medical Center) for providing breast cancer cells and insightful discussion, and Jen-Ming Huang (Cedars-Sinai Medical Center) for technical help.

## Funding

This work was supported by NCI P01 CA098912 and the Board of Governors Chair of Cancer Research Fund from Cedars-Sinai Medical Center to Leland W. K. Chung.

## Ethics declarations

### Ethics approval and consent to participate

All animal procedures were performed under the institutional animal care and use committee of the Cedars-Sinai Medical Center with the recommendations in the Guide for the Care and Use of Laboratory Animals.

### Consent for publication

All authors have approved the manuscript to submission.

### Competing interests

The authors declare that they have no competing interests.

## Additional information

### Publisher's Note

Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

## Supplementary Information

### Additional file 1

. Additional file of KRT13 promotes stemness and drives metastasis in breast cancer through a plakoglobin/c-Myc signaling pathway.

## Rights and permissions

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/). The Creative Commons Public Domain Dedication waiver ([http://creativecommons.org/publicdomain/zero/1.0/](http://creativecommons.org/publicdomain/zero/1.0/)) applies to the data made available in this article, unless otherwise stated in a credit line to the data.

[^1]: Kim S, Wong P, Coulombe PA. A keratin cytoskeletal protein regulates protein synthesis and epithelial cell growth. Nature. 2006;441(7091):362–5.

[^2]: Saha SK, Choi HY, Kim BW, Dayem AA, Yang GM, Kim KS, et al. KRT19 directly interacts with beta-catenin/RAC1 complex to regulate NUMB-dependent NOTCH signaling pathway and breast cancer properties. Oncogene. 2017;36(3):332–49.

[^3]: Hendrix MJ, Seftor EA, Chu YW, Trevor KT, Seftor RE. Role of intermediate filaments in migration, invasion and metastasis. Cancer Metastasis Rev. 1996;15(4):507–25.

[^4]: Malecha MJ, Miettinen M. Expression of keratin 13 in human epithelial neoplasms. Virchows Arch A Pathol Anat Histopathol. 1991;418(3):249–54.

[^5]: van Muijen GN, Ruiter DJ, Franke WW, Achtstatter T, Haasnoot WH, Ponec M, et al. Cell type heterogeneity of cytokeratin expression in complex epithelia and carcinomas as demonstrated by monoclonal antibodies specific for cytokeratins nos. 4 and 13. Exp Cell Res. 1986;162(1):97–113.

[^6]: Liu S, Cadaneanu RM, Zhang B, Huo L, Lai K, Li X, et al. Keratin 13 is enriched in prostate tubule-initiating cells and may identify primary prostate tumors that metastasize to the bone. PLoS ONE. 2016;11(10):e0163232.

[^7]: Moll R, Divo M, Langbein L. The human keratins: biology and pathology. Histochem Cell Biol. 2008;129(6):705–33.

[^8]: Waseem A, Alam Y, Dogan B, White KN, Leigh IM, Waseem NH. Isolation, sequence and expression of the gene encoding human keratin 13. Gene. 1998;215(2):269–79.

[^9]: Sheng S, Barnett DH, Katzenellenbogen BS. Differential estradiol and selective estrogen receptor modulator (SERM) regulation of Keratin 13 gene expression and its underlying mechanism in breast cancer cells. Mol Cell Endocrinol. 2008;296(1–2):1–9.

[^10]: Richard G, De Laurenzi V, Didona B, Bale SJ, Compton JG. Keratin 13 point mutation underlies the hereditary mucosal epithelial disorder white sponge nevus. Nat Genet. 1995;11(4):453–5.

[^11]: Yang Y, Tang X, Song X, Tang L, Cao Y, Liu X, et al. Evidence for an oncogenic role of HOXC6 in human non-small cell lung cancer. PeerJ. 2019;7:e6629.

[^12]: Madoz-Gurpide J, Canamero M, Sanchez L, Solano J, Alfonso P, Casal JI. A proteomics analysis of cell signaling alterations in colorectal cancer. Mol Cell Proteomics. 2007;6(12):2150–64.

[^13]: Acehan D, Petzold C, Gumper I, Sabatini DD, Muller EJ, Cowin P, et al. Plakoglobin is required for effective intermediate filament anchorage to desmosomes. J Invest Dermatol. 2008;128(11):2665–75.

[^14]: Holen I, Whitworth J, Nutter F, Evans A, Brown HK, Lefley DV, et al. Loss of plakoglobin promotes decreased cell-cell contact, increased invasion, and breast cancer cell dissemination in vivo. Breast Cancer Res. 2012;14(3):R86.

[^15]: Aktary Z, Kulak S, Mackey J, Jahroudi N, Pasdar M. Plakoglobin interacts with the transcription factor p53 and regulates the expression of 14-3-3sigma. J Cell Sci. 2013;126(Pt 14):3031–42.

[^16]: Aktary Z, Alaee M, Pasdar M. Beyond cell-cell adhesion: Plakoglobin and the regulation of tumorigenesis and metastasis. Oncotarget. 2017;8(19):32270–91.

[^17]: Aktary Z, Pasdar M. Plakoglobin: role in tumorigenesis and metastasis. Int J Cell Biol. 2012;2012:189521.

[^18]: Li Q, Yin L, Jones LW, Chu GC, Wu JB, Huang JM, et al. Keratin 13 expression reprograms bone and brain metastases of human prostate cancer cells. Oncotarget. 2016;7(51):84645–57.

[^19]: Chu GC, Zhau HE, Wang R, Rogatko A, Feng X, Zayzafoon M, et al. RANK- and c-Met-mediated signal network promotes prostate cancer metastatic colonization. Endocr Relat Cancer. 2014;21(2):311–26.

[^20]: Zhao X, Rødland EA, Sørlie T, Vollan HK, Russnes HG, Kristensen VN, et al. Systematic assessment of prognostic gene signatures for breast cancer shows distinct influence of time and ER status. BMC Cancer. 2014;14:211.

[^21]: Perou CM, Sørlie T, Eisen MB, van de Rijn M, Jeffrey SS, Rees CA, et al. Molecular portraits of human breast tumours. Nature. 2000;406(6797):747–52.

[^22]: Zhau HE, Odero-Marah V, Lue HW, Nomura T, Wang R, Chu G, et al. Epithelial to mesenchymal transition (EMT) in human prostate cancer: lessons learned from ARCaP model. Clin Exp Metastasis. 2008;25(6):601–10.

[^23]: Li X, Wu JB, Li Q, Shigemura K, Chung LW, Huang WC. SREBP-2 promotes stem cell-like properties and metastasis by transcriptional activation of c-Myc in prostate cancer. Oncotarget. 2016;7(11):12869–84.

[^24]: Hu P, Chu GC, Zhu G, Yang H, Luthringer D, Prins G, et al. Multiplexed quantum dot labeling of activated c-Met signaling in castration-resistant human prostate cancer. PLoS ONE. 2011;6(12):e28670.

[^25]: Lo PK, Kanojia D, Liu X, Singh UP, Berger FG, Wang Q, et al. CD49f and CD61 identify Her2/neu-induced mammary tumor-initiating cells that are potentially derived from luminal progenitors and maintained by the integrin-TGFbeta signaling. Oncogene. 2012;31(21):2614–26.

[^26]: Provenzano E, Ulaner GA, Chin SF. Molecular Classification of Breast Cancer. PET Clin. 2018;13(3):325–38.

[^27]: Yang L, Chen Y, Cui T, Knosel T, Zhang Q, Albring KF, et al. Desmoplakin acts as a tumor suppressor by inhibition of the Wnt/beta-catenin signaling pathway in human lung cancer. Carcinogenesis. 2012;33(10):1863–70.

[^28]: Kawai T, Yasuchika K, Ishii T, Katayama H, Yoshitoshi EY, Ogiso S, et al. Keratin 19, a cancer stem cell marker in human hepatocellular carcinoma. Clin Cancer Res. 2015;21(13):3081–91.

[^29]: Govaere O, Komuta M, Berkers J, Spee B, Janssen C, de Luca F, et al. Keratin 19: a key role player in the invasion of human hepatocellular carcinomas. Gut. 2014;63(4):674–85.

[^30]: Cheung KJ, Ewald AJ. A collective route to metastasis: Seeding by tumor cell clusters. Science. 2016;352(6282):167–9.

[^31]: Dittmer J. Breast cancer stem cells: features, key drivers and treatment options. Semin Cancer Biol. 2018;53:59–74.

[^32]: Baccelli I, Schneeweiss A, Riethdorf S, Stenzinger A, Schillert A, Vogel V, et al. Identification of a population of blood circulating tumor cells from breast cancer patients that initiates metastasis in a xenograft assay. Nat Biotechnol. 2013;31(6):539–44.

[^33]: Hu WY, Hu DP, Xie L, Li Y, Majumdar S, Nonn L, et al. Isolation and functional interrogation of adult human prostate epithelial stem cells at single cell resolution. Stem Cell Res. 2017;23:1–12.

[^34]: Ricardo S, Vieira AF, Gerhard R, Leitao D, Pinto R, Cameselle-Teijeiro JF, et al. Breast cancer stem cell markers CD44, CD24 and ALDH1: expression distribution within intrinsic molecular subtype. J Clin Pathol. 2011;64(11):937–46.

[^35]: Bertucci F, Cervera N, Birnbaum D. A gene signature in breast cancer. N Engl J Med. 2007;356(18):1887–8 (**author reply -8**).

[^36]: Vita M, Henriksson M. The Myc oncoprotein as a therapeutic target for human cancer. Semin Cancer Biol. 2006;16(4):318–30.

[^37]: Todorovic-Rakovic N, Neskovic-Konstantinovic Z, Nikolic-Vukosavljevic D. C-myc as a predictive marker for chemotherapy in metastatic breast cancer. Clin Exp Med. 2012;12(4):217–23.

[^38]: Robanus-Maandag EC, Bosch CA, Kristel PM, Hart AA, Faneyte IF, Nederlof PM, et al. Association of C-MYC amplification with progression from the in situ to the invasive stage in C-MYC-amplified breast carcinomas. J Pathol. 2003;201(1):75–82.

[^39]: Nair R, Roden DL, Teo WS, McFarland A, Junankar S, Ye S, et al. c-Myc and Her2 cooperate to drive a stem-like phenotype with poor prognosis in breast cancer. Oncogene. 2014;33(30):3992–4002.

[^40]: Pradella D, Naro C, Sette C, Ghigna C. EMT and stemness: flexible processes tuned by alternative splicing in development and cancer progression. Mol Cancer. 2017;16(1):8.

[^41]: Fabregat I, Malfettone A, Soukupova J. New insights into the crossroads between EMT and stemness in the context of cancer. J Clin Med. 2016;5(3):37.

[^42]: Yin T, Getsios S, Caldelari R, Kowalczyk AP, Muller EJ, Jones JC, et al. Plakoglobin suppresses keratinocyte motility through both cell-cell adhesion-dependent and -independent mechanisms. Proc Natl Acad Sci U S A. 2005;102(15):5420–5.

[^43]: Todorovic V, Desai BV, Patterson MJ, Amargo EV, Dubash AD, Yin T, et al. Plakoglobin regulates cell motility through Rho- and fibronectin-dependent Src signaling. J Cell Sci. 2010;123(Pt 20):3576–86.

[^44]: Rieger-Christ KM, Ng L, Hanley RS, Durrani O, Ma H, Yee AS, et al. Restoration of plakoglobin expression in bladder carcinoma cell lines suppresses cell migration and tumorigenic potential. Br J Cancer. 2005;92(12):2153–9.

[^45]: Alaee M, Nool K, Pasdar M. Plakoglobin restores tumor suppressor activity of p53(R175H) mutant by sequestering the oncogenic potential of beta-catenin. Cancer Sci. 2018;109(6):1876–88.

[^46]: de Bruin A, Caldelari R, Williamson L, Suter MM, Hunziker T, Wyder M, et al. Plakoglobin-dependent disruption of the desmosomal plaque in pemphigus vulgaris. Exp Dermatol. 2007;16(6):468–75.

[^47]: Lai YH, Cheng J, Cheng D, Feasel ME, Beste KD, Peng J, et al. SOX4 interacts with plakoglobin in a Wnt3a-dependent manner in prostate cancer cells. BMC Cell Biol. 2011;12:50.

[^48]: Koyama-Nasu R, Takahashi R, Yanagida S, Nasu-Nishimura Y, Oyama M, Kozuka-Hata H, et al. The cancer stem cell marker CD133 interacts with plakoglobin and controls desmoglein-2 protein levels. PLoS ONE. 2013;8(1):e53710.

[^49]: Williamson L, Raess NA, Caldelari R, Zakher A, de Bruin A, Posthaus H, et al. Pemphigus vulgaris identifies plakoglobin as key suppressor of c-Myc in the skin. Embo J. 2006;25(14):3298–309.

[^50]: Chen YJ, Lee LY, Chao YK, Chang JT, Lu YC, Li HF, et al. DSG3 facilitates cancer cell growth and invasion through the DSG3-plakoglobin-TCF/LEF-Myc/cyclin D1/MMP signaling pathway. PLoS ONE. 2013;8(5):e64088.

[^51]: Chu GC, Chung LW. RANK-mediated signaling network and cancer metastasis. Cancer Metastasis Rev. 2014;33(2–3):497–509.