---
citation_key: keratin_profiling_2021
created: 2025-06-09
doi: 10.3390/ijms22158109
last_updated: '2025-06-14T13:08:54.376380'
processing_status:
  organize: completed
source: https://www.mdpi.com/1422-0067/22/15/8109
title: Keratin Profiling by Single-Cell RNA-Sequencing Identifies Human Prostate Stem
  Cell Lineage Hierarchy and Cancer Stem-Like Cells
workflow_version: '3.2'
---

Next Article in Journal  

[The Mechanism and Effect of Autophagy, Apoptosis, and Pyroptosis on the Progression of Silicosis](https://www.mdpi.com/1422-0067/22/15/8110)

Next Article in Special Issue  

[Dickkopf-1 Promotes Angiogenesis and is a Biomarker for Hepatic Stem Cell-like Hepatocellular Carcinoma](https://www.mdpi.com/1422-0067/23/5/2801)

Previous Article in Journal  

[Effects of Visfatin on Intracellular Mechanics and Catabolism in Human Primary Chondrocytes through Glycogen Synthase Kinase 3β Inactivation](https://www.mdpi.com/1422-0067/22/15/8107)

Previous Article in Special Issue  

[Genomic Analyses of Pediatric Acute Lymphoblastic Leukemia Ph+ and Ph-Like—Recent Progress in Treatment](https://www.mdpi.com/1422-0067/22/12/6411)

[![ijms-logo](https://pub.mdpi-res.com/img/journals/ijms-logo.png?41930e44070e4940 "International Journal of Molecular Sciences")](https://www.mdpi.com/journal/ijms)

[Submit to this Journal](https://susy.mdpi.com/user/manuscripts/upload?form%5Bjournal_id%5D%3D2) [Review for this Journal](https://susy.mdpi.com/volunteer/journals/review) [Propose a Special Issue](https://www.mdpi.com/journalproposal/sendproposalspecialissue/ijms)

- [Academic Editor](https://www.mdpi.com/1422-0067/22/15/#academic_editors)
- [Subscribe SciFeed](https://www.mdpi.com/1422-0067/22/15/8109/scifeed_display)
- [Related Info Links](https://www.mdpi.com/1422-0067/22/15/#related)[Article Views 4802](https://www.mdpi.com/1422-0067/22/15/#metrics)[Citations 10](https://www.mdpi.com/1422-0067/22/15/#metrics)

Open Access Article

<sup>1</sup>

Department of Urology, University of Illinois at Chicago, Chicago, IL 60612, USA

<sup>2</sup>

Department of Pathology, University of Illinois at Chicago, Chicago, IL 60612, USA

<sup>3</sup>

Massachusetts General Hospital Center for Cancer Research and Harvard Medical School, Charlestown, MA 02129, USA

<sup>*</sup>

Author to whom correspondence should be addressed.

*Int. J. Mol. Sci.***2021**, *22* (15), 8109; [https://doi.org/10.3390/ijms22158109](https://doi.org/10.3390/ijms22158109)

Submission received: 9 June 2021 / Revised: 23 July 2021 / Accepted: 26 July 2021 / Published: 28 July 2021

(This article belongs to the Special Issue [Cancer Stem Cells in Basic Science and in Translational Oncology](https://www.mdpi.com/journal/ijms/special_issues/cancer_stemcells))  

## Abstract

Single prostate stem cells can generate stem and progenitor cells to form prostaspheres in 3D culture. Using a prostasphere-based label retention assay, we recently identified keratin 13 (KRT13)-enriched prostate stem cells at single-cell resolution, distinguishing them from daughter progenitors. Herein, we characterized the epithelial cell lineage hierarchy in prostaspheres using single-cell RNA-seq analysis. Keratin profiling revealed three clusters of label-retaining prostate stem cells; cluster I represents quiescent stem cells (PSCA, CD36, SPINK1, and KRT13/23/80/78/4 enriched), while clusters II and III represent active stem and bipotent progenitor cells (KRT16/17/6 enriched). Gene set enrichment analysis revealed enrichment of stem and cancer-related pathways in cluster I. In non-label-retaining daughter progenitor cells, three clusters were identified; cluster IV represents basal progenitors (KRT5/14/6/16 enriched), while clusters V and VI represent early and late-stage luminal progenitors, respectively (KRT8/18/10 enriched). Furthermore, MetaCore analysis showed enrichment of the “cytoskeleton remodeling–keratin filaments” pathway in cancer stem-like cells from human prostate cancer specimens. Along with common keratins (KRT13/23/80/78/4) in normal stem cells, unique keratins (KRT10/19/6C/16) were enriched in cancer stem-like cells. Clarification of these keratin profiles in human prostate stem cell lineage hierarchy and cancer stem-like cells can facilitate the identification and therapeutic targeting of prostate cancer stem-like cells.

## 1\. Introduction

Resident stem cells are a rare cell population found in adult tissues that act as a repair system by maintaining normal regenerative turnover \[[1](https://www.mdpi.com/1422-0067/22/15/#B1-ijms-22-08109),[2](https://www.mdpi.com/1422-0067/22/15/#B2-ijms-22-08109)\]. Prostate epithelial stem cells have been identified in the human prostate gland and are thought to generate differentiated epithelial cells that maintain glandular homeostasis throughout life \[[3](https://www.mdpi.com/1422-0067/22/15/#B3-ijms-22-08109),[4](https://www.mdpi.com/1422-0067/22/15/#B4-ijms-22-08109)\]. Three types of prostate epithelial cells comprise the simple columnar epithelium: apical secretory luminal cells, underlying basal cells adjacent to the basement membrane, and a small number of neuroendocrine cells \[[5](https://www.mdpi.com/1422-0067/22/15/#B5-ijms-22-08109)\]. While the differentiation of prostate stem cells into basal, luminal, and neuroendocrine cells has been documented in both the rodent and human prostate \[[3](https://www.mdpi.com/1422-0067/22/15/#B3-ijms-22-08109),[4](https://www.mdpi.com/1422-0067/22/15/#B4-ijms-22-08109),[6](https://www.mdpi.com/1422-0067/22/15/#B6-ijms-22-08109),[7](https://www.mdpi.com/1422-0067/22/15/#B7-ijms-22-08109),[8](https://www.mdpi.com/1422-0067/22/15/#B8-ijms-22-08109),[9](https://www.mdpi.com/1422-0067/22/15/#B9-ijms-22-08109),[10](https://www.mdpi.com/1422-0067/22/15/#B10-ijms-22-08109)\], the lineage hierarchy for prostate stem cell self-renewal, progenitor cell proliferation, and differentiation remains poorly understood.

Several prostate epithelial cell lineage differentiation models have been proposed. Phenotypic intermediate-type cells, aka the transit-amplifying cells that co-express basal and luminal markers, have been observed in vitro and in vivo \[[8](https://www.mdpi.com/1422-0067/22/15/#B8-ijms-22-08109),[9](https://www.mdpi.com/1422-0067/22/15/#B9-ijms-22-08109),[10](https://www.mdpi.com/1422-0067/22/15/#B10-ijms-22-08109)\]. Therefore, the traditional linear hierarchy model proposes that self-renewing prostate stem cells undergo asymmetric cell division, giving rise to common daughter progenitor cells with high proliferative potential \[[11](https://www.mdpi.com/1422-0067/22/15/#B11-ijms-22-08109),[12](https://www.mdpi.com/1422-0067/22/15/#B12-ijms-22-08109),[13](https://www.mdpi.com/1422-0067/22/15/#B13-ijms-22-08109)\]. In response to signals from the stem cell niche, these cells enter early differentiation pathways to eventually form separate basal, luminal, and neuroendocrine cells \[[14](https://www.mdpi.com/1422-0067/22/15/#B14-ijms-22-08109),[15](https://www.mdpi.com/1422-0067/22/15/#B15-ijms-22-08109),[16](https://www.mdpi.com/1422-0067/22/15/#B16-ijms-22-08109)\]. This suggests that basal and luminal cells are hierarchically related through common progenitor cells. On the other hand, the bifurcated model suggests that basal and luminal progenitor cells represent separate epithelial lineages originating from a common stem cell \[[17](https://www.mdpi.com/1422-0067/22/15/#B17-ijms-22-08109),[18](https://www.mdpi.com/1422-0067/22/15/#B18-ijms-22-08109)\].

In a 3D culture system, a single prostate stem cell can undergo both symmetric and asymmetric cell division by self-renewal, generating relatively quiescent daughter stem cells and rapidly proliferating daughter progenitor cells to form spheroids, also known as prostaspheres \[[19](https://www.mdpi.com/1422-0067/22/15/#B19-ijms-22-08109),[20](https://www.mdpi.com/1422-0067/22/15/#B20-ijms-22-08109),[21](https://www.mdpi.com/1422-0067/22/15/#B21-ijms-22-08109),[22](https://www.mdpi.com/1422-0067/22/15/#B22-ijms-22-08109)\]. Thus, prostaspheres in 3D culture provide a useful system to interrogate the stem cell lineage hierarchy. Recently, our laboratory established a prostasphere-based label retention assay with the capability to identify and isolate long-term label-retaining prostate stem-like cells from their non-label-retaining daughter progenitor cells \[[23](https://www.mdpi.com/1422-0067/22/15/#B23-ijms-22-08109),[24](https://www.mdpi.com/1422-0067/22/15/#B24-ijms-22-08109)\]. This approach further enhanced the spheroid model for tracking stem and daughter progenitor cell populations and permitted the functional and transcriptomic analysis of these distinct populations using primary epithelial cells from disease-free human prostates. Of note, the highest differentially expressed gene for the stem cell population was keratin 13 (KRT13) \[[23](https://www.mdpi.com/1422-0067/22/15/#B23-ijms-22-08109),[24](https://www.mdpi.com/1422-0067/22/15/#B24-ijms-22-08109)\]. Keratins are a family of intermediate filament proteins that perform both mechanical and non-mechanical functions \[[25](https://www.mdpi.com/1422-0067/22/15/#B25-ijms-22-08109),[26](https://www.mdpi.com/1422-0067/22/15/#B26-ijms-22-08109),[27](https://www.mdpi.com/1422-0067/22/15/#B27-ijms-22-08109)\]. Several distinct keratins have been identified in stem cells and cancers where they may play unique roles \[[28](https://www.mdpi.com/1422-0067/22/15/#B28-ijms-22-08109),[29](https://www.mdpi.com/1422-0067/22/15/#B29-ijms-22-08109),[30](https://www.mdpi.com/1422-0067/22/15/#B30-ijms-22-08109),[31](https://www.mdpi.com/1422-0067/22/15/#B31-ijms-22-08109)\]. Previous studies had identified KRT13 as a marker for leading edge prostate cancer regional invasion in organ-confined disease \[[32](https://www.mdpi.com/1422-0067/22/15/#B32-ijms-22-08109),[33](https://www.mdpi.com/1422-0067/22/15/#B33-ijms-22-08109)\], and our findings confirmed it as a specific human prostate stem cell marker regulating its self-renewal \[[23](https://www.mdpi.com/1422-0067/22/15/#B23-ijms-22-08109),[24](https://www.mdpi.com/1422-0067/22/15/#B24-ijms-22-08109)\].

In the present study, we employed the label-retention spheroid assay combined with the Fluidigm C1 and 10X Genomics platforms for single-cell isolation and RNA-seq to identify distinct cell types in the heterogeneous label-retaining stem-like and non-label-retaining progenitor cell populations. Single cell gene profiling combined with bioinformatic tools permitted the clarification of the normal prostate epithelial stem and progenitor cell lineage hierarchy from disease-free human prostate specimens. Specifically, RNA-seq analysis revealed a select group of keratins enriched in label-retaining prostate stem cells, while a separate and distinct group of keratins were enriched in non-retaining basal and luminal progenitors. Clustering analysis, keratin profiling, combined with pseudo-temporal reconstruction and ordering of single cells \[[34](https://www.mdpi.com/1422-0067/22/15/#B34-ijms-22-08109),[35](https://www.mdpi.com/1422-0067/22/15/#B35-ijms-22-08109)\], revealed the lineage hierarchy of normal prostate epithelial cells: quiescent prostate stem cells (cluster I) become active stem cells (cluster II) to generate bipotent progenitor cells (cluster III), which give rise to unipotent basal (cluster IV) progenitors as well as early (cluster V) and late (cluster VI) stage luminal progenitors. Using a similar approach with a prostate cancer specimen from organ-confined disease, we identified gene profiles for prostate cancer stem-like cells (cSCs) \[[23](https://www.mdpi.com/1422-0067/22/15/#B23-ijms-22-08109),[24](https://www.mdpi.com/1422-0067/22/15/#B24-ijms-22-08109)\] with keratins common to benign stem cells as well as cancer stem-like cell unique keratins. Clarification of the stem cell hierarchy and keratin profiling of human prostate stem cells and cSCs has significant potential for translational research as it permits the identification of novel biomarkers and opens opportunities for developing novel therapeutic strategies that target the cSC population to limit tumor growth and progression.

## 2\. Results

#### 2.1. Differentially Expressed Keratin Genes in Prostate Stem-Like and Progenitor Cells by Fluidigm Single Cell RNA-Seq

Both structural and non-structural functions of keratins have been identified, and several keratins are associated with stem cells and cancers \[[25](https://www.mdpi.com/1422-0067/22/15/#B25-ijms-22-08109),[26](https://www.mdpi.com/1422-0067/22/15/#B26-ijms-22-08109),[27](https://www.mdpi.com/1422-0067/22/15/#B27-ijms-22-08109),[28](https://www.mdpi.com/1422-0067/22/15/#B28-ijms-22-08109),[29](https://www.mdpi.com/1422-0067/22/15/#B29-ijms-22-08109),[30](https://www.mdpi.com/1422-0067/22/15/#B30-ijms-22-08109),[31](https://www.mdpi.com/1422-0067/22/15/#B31-ijms-22-08109)\]. Furthermore, keratin profiling has been previously used to study prostate development and epithelial cell differentiation \[[28](https://www.mdpi.com/1422-0067/22/15/#B28-ijms-22-08109),[29](https://www.mdpi.com/1422-0067/22/15/#B29-ijms-22-08109),[30](https://www.mdpi.com/1422-0067/22/15/#B30-ijms-22-08109)\]. To investigate the keratin gene profiles in prostate stem and progenitor cells, we pre-labeled pooled primary prostate epithelial cells (PrECs) from three normal prostate tissues with BrdU or 5(6)-carboxyfluorescein N-hydroxysuccinimidyl ester (CFSE) followed by wash-out in 3D Matrigel culture for 5 days during spheroid formation. Prostaspheres were dispersed into single cells, and CFSE <sup>High</sup> prostate stem-like cells (1.01%) were separated from CFSE <sup>Low</sup> progenitor cells (97.52%) by FACS ([Figure 1](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f001) A) \[[23](https://www.mdpi.com/1422-0067/22/15/#B23-ijms-22-08109),[24](https://www.mdpi.com/1422-0067/22/15/#B24-ijms-22-08109)\]. The cells were captured using the Fluidigm C1 platform that although isolating and tagging fewer single cells than other systems, permits greater transcriptome resolution with superior coverage across transcripts including gene isoforms. Initial heatmap clustering of 37 keratin genes in the two sorted populations revealed enrichment of KRT13, 23, 80, 78, 86, and 4 in the CFSE <sup>High</sup> prostate stem-like cells, while KRT6, 17, 14, 5, 8, and 18 were enriched in CFSE <sup>Low</sup> progenitor cells ([Figure 1](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f001) B). Immunostaining confirmed the enrichment of keratins 13, 23, 4, 78, 80, and 19 at the protein level in BrdU label retaining or CFSE <sup>High</sup> prostate stem cells ([Figure 1](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f001) C). To better identify the cell subpopulations, we undertook heatmap clustering with selected known gene markers for stem, basal, luminal, and neuroendocrine cells ([Figure 1](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f001) D). Enrichment of the stem cell marker gene KRT13 was confirmed in CFSE <sup>High</sup> label-retaining prostate stem cells, while basal marker genes KRT5, KRT14, KRT19, TP63, SOX9, GSTP1, luminal genes KRT8/18, and neuroendocrine gene SYP were enriched in CFSE <sup>Low</sup> non-label-retaining prostate progenitor cells, suggesting a stem cell-driven epithelial cell lineage commitment during prostasphere formation. Of note, transcription factor SOX9 was enriched in both label-retaining stem cells and non-retaining progenitor cells.

![Ijms 22 08109 g001 550](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g001-550.jpg)

**Figure 1.** Two groups of differentially expressed keratin genes in CFSE <sup>High</sup> vs. CFSE <sup>Low</sup> day 5 prostasphere cells. (**A**) Normal primary prostate epithelial cells (PrEC) pre-labeled with CFSE were 3D cultured (5 days) to form prostaspheres (PS), followed by FACS to separate CFSE <sup>High</sup> stem-like cells (1.01%) from CFSE <sup>Low</sup> progenitor cells (97.52%). Scale bar = 50 µm. (**B**) Heatmap clustering of Fluidigm C1 captured single-cell RNA-seq of prostasphere cells revealed enrichment of KRT13, 23, 80, 78, 86, and 4 in CFSE <sup>High</sup> prostate stem-like cells (89 cells), while KRT6, 17, 14, 5, 8, and 18 were enriched in CFSE <sup>Low</sup> progenitor cells (170 cells). (**C**) Top, Immunostaining confirmed the colocalization of keratins 13, 80, and 78 in BrdU label retaining prostasphere cells (pink) and keratin 14 localization in non-labeled progenitor cells. Bottom, FACS isolated CFSE <sup>High</sup> prostate stem-like cells immunostained for keratin 23, 19, while CFSE <sup>Low</sup> progenitor cells did not. Scale bar = 50 µm. (**D**) Single-cell RNA-seq of CFSE labeled prostaspheres followed by heatmap clustering using selected genes for stem, basal, luminal, and neuroendocrine cells showed enrichment of stem cell marker gene KRT13 in CFSE <sup>High</sup> prostate stem cells, while basal marker genes KRT5, KRT14, TP63, GSTP1, and KRT19, luminal genes KRT8/18, and neuroendocrine gene SYP were enriched in CFSE <sup>Low</sup> prostate progenitor cells. SOX9 was uniquely enriched in CFSE <sup>High</sup> cells and a subpopulation of CFSE <sup>Low</sup> cells.

#### 2.2. Identification of Three Subpopulations in Prostasphere CFSEHigh Prostate Stem-Like Cells by Single-Cell Analysis

In response to extrinsic signals from the niches, stem cells residing in the quiescent state become activated and enter the cell cycle to progress to lineage-committed daughter progenitor cells \[[36](https://www.mdpi.com/1422-0067/22/15/#B36-ijms-22-08109),[37](https://www.mdpi.com/1422-0067/22/15/#B37-ijms-22-08109),[38](https://www.mdpi.com/1422-0067/22/15/#B38-ijms-22-08109)\]. As such, we hypothesized that CFSE <sup>High</sup> cells may represent a heterogeneous mix of stem cells at different quiescent and activated stages. Indeed, Fluidigm single-cell RNA-seq of 172 CFSE <sup>High</sup> cells followed by uniform manifold approximation and projection (UMAP) and Tool of Single Cell Analysis (TSCAN) identified three major clusters ([Figure 2](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f002) A,B), which were further confirmed by unsupervised heatmap clustering ([Figure 2](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f002) C). Cell hierarchical clustering (cluster I → II → III) was reconstructed by a minimum spanning tree (MST) incorporated with the traveling salesman problem algorithm to minimize the distance linking cell clusters. A pseudo-temporal ordering score was used to determine starting and ending points of the tree ([Figure 2](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f002) B), which closely mimics the true biological time \[[35](https://www.mdpi.com/1422-0067/22/15/#B35-ijms-22-08109)\]. Based on these analyses, cluster I, the originating cells, are predicted to be quiescent stem-like cells, yielding cluster II, likely active stem cells that lead to cluster III, which are proposed as bipotent progenitor cells.

![Ijms 22 08109 g002 550](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g002-550.jpg)

**Figure 2.** Single-cell RNA-seq of CFSE <sup>High</sup> cells revealed three subpopulations of cells. Data from Fluidigm C1 single-cell RNA-seq of CFSE <sup>High</sup> isolated prostasphere cells was analyzed by UMAP (**A**) and TSCAN (**B**) principal component analyses, which identified three major cell clusters that were further confirmed by unsupervised heatmap clustering (**C**). CFSE <sup>High</sup> cell hierarchical clustering (I (1) → II (2) → III (3)) was ordered by MST-based pseudo-time reconstruction to mimic true biological time, identifying cells in cluster I as originating cells which yield cluster II that then yield cluster III (labeled in B). (**D**) Known stemness genes PSCA, CD36, and SPINK1 are enriched in cluster I (1) as compared to clusters II (2) & III (3), which is demarcated by a circle size (top) that represents gene expression levels that are relatively quantified at the bottom.

Genes previously associated with prostate stemness, e.g., PSCA \[[39](https://www.mdpi.com/1422-0067/22/15/#B39-ijms-22-08109)\], SPINK1 \[[40](https://www.mdpi.com/1422-0067/22/15/#B40-ijms-22-08109)\], as well as stemness genes not previously reported in prostate stem cells, e.g., CD36 \[[41](https://www.mdpi.com/1422-0067/22/15/#B41-ijms-22-08109)\] were enriched in CFSE <sup>High</sup> cluster I cells. Temporal tracing of these genes by TSCAN positioned them in cluster I with decreasing expression through clusters II and III ([Figure 2](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f002) D), confirming the stem cell nature of cluster I cells and the power of this approach.

#### 2.3. Keratin Gene Profiling Distinguishes the Prostate Stem and Bipotent Progenitor Cell Clusters

Heatmap clustering of 21 keratin genes in 172 CFSE <sup>High</sup> sorted prostasphere cells was next used to further distinguish the three cell clusters found in the label-retaining stem-like cell population. Keratin gene profiles revealed the enrichment of KRT13, 23, 80, 78, and 4 in cluster I stem cells ([Figure 3](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f003) A, middle). In contrast, these keratins were sharply decreased in both clusters II (left) and III (right), permitting easy identification of the proposed quiescent stem cells. Clusters II and III, proposed as active stem cells and bipotent progenitors, respectively, were further differentiated by increased KRT16(16P2, 16P4, 16P5), 17(17P2, 17P3, 17P6), and 6 gene expressions in cluster III, which were not heightened in cluster II ([Figure 3](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f003) A). TSCAN was next employed to trace relative keratin gene expression levels in the three clusters in pseudo-time from cluster I → II → III, which is graphically illustrated by circle size for each denoted cell ([Figure 3](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f003) B). Enrichment of KRT13, 80, 78, and 23 was confirmed for cluster I quiescent stem cells, while KRT16P4 and 17P6 were enriched in clusters III bipotent progenitor cells. Active stem cells in cluster II express decreased levels of stemness keratins KRT13, 80, 78, and 23 enriched in cluster I with increased levels of KRT16 (16P2, 16P4, 16P5) and 17 (17P2, 17P3, 17P6) enriched in cluster III. Together, these sequential changes in keratin genes as the stem cells enter lineage commitment further supports the progression of cell hierarchy from cluster I → II → III. Furthermore, gene set enrichment analysis (GSEA) revealed the enrichment of stem-cell-related Notch, Toll-like receptor, autophagy, and lysosome pathways in cluster I vs. cluster III ([Figure 3](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f003) C). These are pathways previously associated with stemness in the prostate and other tissues \[[42](https://www.mdpi.com/1422-0067/22/15/#B42-ijms-22-08109),[43](https://www.mdpi.com/1422-0067/22/15/#B43-ijms-22-08109),[44](https://www.mdpi.com/1422-0067/22/15/#B44-ijms-22-08109)\]. Collectively, these data demonstrate that the keratin profiles rapidly change as the stem cells lineage commit from quiescence to active stem cells and bipotent progenitor cells.

![Ijms 22 08109 g003 550](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g003-550.jpg)

**Figure 3.** Differentially expressed keratin genes in CFSE <sup>High</sup> prostasphere cells are sufficient to distinguish cells in cluster I, cluster II, and cluster III. (**A**) In CFSE <sup>High</sup> cells, KRT13, 23, 80, 78, and 4 are enriched in cluster I quiescent stem cells and have low expression in clusters II and III. KRT16, 17, and 6 genes are enriched in cluster III, whereas cluster II has low expression levels of this keratin gene set. (**B**) TSCAN analysis documents enrichment of KRT13, 80, 78, and 23 in cluster I (1) quiescent stem cells and of KRT16P4 and 17P6 in cluster III (3) cells demarcated by the circle size (top) which is directionally relatively quantified at bottom. (**C**) GSEA analysis identifies enrichment of stem cell-related Notch, Toll-like receptor, autophagy, and lysosome pathways in cluster I (1) vs. cluster III (3).

#### 2.4. Heatmap Clustering Analysis Identifies Three Separate Clusters of Prostasphere Progenitor Cells with Differentially Expressed Keratin Genes

In the human prostate, epithelial stem cells replenish the tissue and maintain glandular homeostasis by generating lineage-committed progenitor cells that generate basal, luminal, and neuroendocrine cells \[[10](https://www.mdpi.com/1422-0067/22/15/#B10-ijms-22-08109),[11](https://www.mdpi.com/1422-0067/22/15/#B11-ijms-22-08109),[45](https://www.mdpi.com/1422-0067/22/15/#B45-ijms-22-08109)\]. As it is likely that the stem cell-derived, daughter bipotent progenitor cells in prostaspheres initiate and progress along this lineage commitment, we sought to classify the separate progenitor cell populations in the non-label retaining cell population. We FACS separated CFSE <sup>Low</sup> cells, captured 197 individual cells, and sequenced their transcripts using the Fluidigm scRNA-seq platform. Heatmap clustering analysis for the keratin genes revealed three distinct subpopulations of progenitor cells in the CFSE <sup>Low</sup> cell fraction ([Figure 4](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f004) A). Cluster IV cells (on right) were highly enriched in KRT5, 6, 14, and 1 6 and are predicted to be unipotent basal progenitor cells. In contrast, clusters V and VI (middle and left, [Figure 4](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f004) A) were highly enriched in KRT8, 18, and 10 with negligible KRT5, indicating a luminal progenitor cell lineage. As cluster V retains KRT16, a marker for the label-retaining bipotent progenitor (cluster III, [Figure 3](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f003) A), and since KRT8 and 18 are slightly lower in cluster V compared to cluster VI shown by TSCAN ([Figure 4](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f004) B), cluster V is likely an early-stage luminal progenitor, while cluster VI represents luminal progenitors further along the lineage commitment pathway. This is further supported by the lowest expressions of KRT14, 6A, and 6 C in cluster VI as compared to clusters IV and V. TSCAN analysis for individual keratin genes in CFSE <sup>Low</sup> cells were non-linear, representing two separated cell lineages (basal and luminal) at this stage, thus providing no consistent evidence for the derivation of luminal progenitors from basal progenitors ([Figure 4](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f004) B).

![Ijms 22 08109 g004 550](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g004-550.jpg)

**Figure 4.** Keratin profiles of differentially expressed keratin genes in non-label retaining prostasphere cells. (**A**) Single-cell RNA-seq of the sorted CFSE <sup>Low</sup> prostasphere cells with subsequent keratin gene heatmap clustering reveals three subpopulations of progenitor cells: cluster IV (right), cluster V (middle), and cluster VI (left). Cluster IV is enriched in KRT14, 5, 16, and 6, indicating a basal progenitor cell lineage. Clusters V and VI are enriched in KRT8, 18, and 10, indicating a luminal cell lineage. (**B**) TSCAN analysis of KRT8 and KRT18 in CFSE <sup>Low</sup> cells suggests a relationship between cluster V (2) and VI (1), but not cluster IV (3). Relative gene expression levels are noted numerically for each cluster.

#### 2.5. RNA-Seq Reveals Differentially Expressed Genes in Cancer Stem-Like Cells

Cancer stem cells often share similar gene marker signatures/signaling pathways with embryonic stem cells and normal adult stem cells \[[46](https://www.mdpi.com/1422-0067/22/15/#B46-ijms-22-08109),[47](https://www.mdpi.com/1422-0067/22/15/#B47-ijms-22-08109)\]. To identify common and unique stem cell genes in prostate cancer, primary prostate cancer (PCa) cells were cultured from punch biopsies of three patient specimens collected at prostatectomy with adjacent sections confirmed as 80–100% cancer cells by a board-certified pathologist. These were transferred to 3D culture to generate tumor spheroids using the sphere-based BrdU or CFSE label-retaining system. Cancer stem-like cells (cSCs, CFSE <sup>High</sup>) and non-stem cancer cells (non-cSCs; CFSE <sup>low</sup>) were separated by FACS ([Figure 5](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f005) A), and cDNA libraries were prepared using protocols developed for low cell numbers \[[48](https://www.mdpi.com/1422-0067/22/15/#B48-ijms-22-08109)\], deep RNA-seq was performed with Illumina NextSeq 500 at the Center for Cancer Research at Harvard Medical School. Unsupervised hierarchical clustering of all transcripts identified 1310 differentially expressed genes (Q value < 0.05); 916 enriched in cSCs, and 394 genes enriched in the non-cSC population ([Figure 5](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f005) B). Top enriched genes in cSCs included stem-cell-related genes PSCA, CD36, and SPINK1, among others not previously associated with prostate cancer stem-like cells ([Table 1](https://www.mdpi.com/1422-0067/22/15/#table_body_display_ijms-22-08109-t001)). Heatmap profiles of CD36, KRT13, SPINK1, and SCGB2A1 document their enrichment in cSCs vs. non-cSCs populations ([Figure 5](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f005) C). Furthermore, immunocytochemistry (ICC) of the PSCA, CD36, and SPINK1 proteins ([Figure 5](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f005) D) confirms their colocalization in the BrdU-labeled cSCs within the tumor spheroids.

![Ijms 22 08109 g005 550](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g005-550.jpg)

**Figure 5.** RNA-seq reveals differentially expressed genes in cancer stem-like cells and vs. non-stem cells in tumor spheroids of patient PCa specimens. (**A**) Cancer stem-like cells (cSC; (CFSE <sup>High</sup>) and non-cSC (CFSE <sup>low</sup>) from spheroids of three PCa specimens were separated by FACS. (**B**) Following RNA-seq of the sorted cells, unsupervised transcriptome profiling identified 1310 differentially expressed genes (Q value < 0.05); 916 enriched in cSCs and 394 enriched in non-cSCs. Sequencing of the non-cSC fractions from three patients were performed in duplicate, highlighting data consistency. For the cSC fraction, one patient sample was sequenced in duplicate, and the other two were sequenced in singlet. (**C**) Heatmap of CD36, KRT13, SPINK1, and SCGB2A1 expression shows enrichment in cSCs vs. non-cSCs. (**D**) ICC documents that PSCA, CD36, and SPINK1 proteins (red) colocalized with BrdU-labeled (green) cSC cells in the 3D cultured tumor spheroids. Similar to normal prostate stem cells, the cSCs exhibit decreased E-cadherin, increased LC3 and KRT13 (Hu et al., 2017), which together identifies unique features of PCa-derived cancer stem-like cells. (E) siRNA knockdown of KRT13 significantly decreased the percentage of CFSE <sup>High</sup> cells in tumor spheroids (left) and tumor spheroid formation (right). \* p < 0.05 vs. siControl, N = 3.

**Table 1.** Top genes enriched in prostate cancer stem-like cells.

![Table](https://www.mdpi.com/img/table.png)

Similar to normal prostate stem cells, the cSCs in tumor spheroids exhibited decreased E-cadherin, elevated LC3, an autophagy marker, and KRT13 ([Figure 5](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f005) D) \[[23](https://www.mdpi.com/1422-0067/22/15/#B23-ijms-22-08109),[24](https://www.mdpi.com/1422-0067/22/15/#B24-ijms-22-08109),[49](https://www.mdpi.com/1422-0067/22/15/#B49-ijms-22-08109)\]. Importantly, the knockdown of KRT13 by siRNA in primary cells from one patient specimen significantly decreased the tumor spheroid formation, likely by inhibiting cancer stem-like cell self-renewal ([Figure 5](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f005) E).

#### 2.6. Prostate Cancer Stem-Like Cells Contain Common and Unique Keratin Genes Compared to Normal Prostate Stem Cells

**Table 2.** Enriched pathways in cSC cells (**A**) and non-cSC cells (**B**) by MetaCore Enrichment analysis.

![Table](https://www.mdpi.com/img/table.png)

Heatmap clustering of 16 keratin genes in the CFSE <sup>High</sup> and CFSE <sup>Low</sup> cells from the tumor spheroids further confirmed the enrichment of specific keratin genes in the cSC fraction ([Figure 6](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f006)). In addition to enrichment for keratins found in normal spheroid stem cells (KRT13, 23, 80, 78, and 4), a group of other keratin genes that includes KRT10, 19, 6, 75, 16, 79, 3, and 82 were enriched in the cSCs relative to the non-cSC population of tumor spheroids. Thus, while some similarities to normal prostate stem cells are observed in cancer stem-like cells, several other genes and keratins not typical for normal prostate stem cells are expressed, indicating unique gene profiles in this cancer cell population. We propose that these unique cancer stemness keratin profiles likely reflect the cell-of-origin of the initial tumors.

![Ijms 22 08109 g006 550](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g006-550.jpg)

**Figure 6.** Heatmap clustering of keratin genes in PCa stem-like cells (cSCs) and the non-stem cells in tumor spheroids from PCa specimens. RNA-seq followed by heatmap clustering analysis shows that in addition to normal stem cell-enriched keratins (KRT13, 23, 80, 78, 4), KRT10, 19, 6C, 75, 16, 79, 3, and 82 also were enriched in cancer stem-like cells relative to expression levels in the non-cSC population. Q -value < 0.05.

It is also noteworthy that the cSC population was enriched for genes involved in ligand-independent androgen receptor signaling, IGF-1 receptor signaling, autophagy, and HIF-1 targets ([Table 2](https://www.mdpi.com/1422-0067/22/15/#table_body_display_ijms-22-08109-t002) A), thus identifying potential therapeutic targets for this unique cancer cell type.

#### 2.7. Single-Cell RNA-Seq Identifies Cancer Stem-Like Cells with their Differentially Expressed Keratin Genes

To further verify the keratin profile in cSC at the single cell levels, day-7 tumor spheroids were established from a PCa patient specimen and dispersed for single-cell capture using the 10XGenomics platform. Cancer cell subpopulations within the spheroid were interrogated by single-cell RNA-seq analysis using Nova-seq. UMAP clustering defined five cell clusters (cluster 0-4) of cancer cells in the tumor spheroids ([Figure 7](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f007) A), with cluster 2 distinctly separated from the other cell types and enriched in stem cell markers including newly identified stemness keratin gene KRT13 ([Figure 7](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f007) B, purple dots). Similar to bulk RNA-seq results, single cells RNA-seq followed by keratin gene profiling using heatmap ([Figure 7](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f007) C) and dot-plot ([Figure 7](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f007) D) analyses identified a group of keratin genes (KRT4, 13, 80, 78, 23, 10, 19, 16, 6A, 6B, 6C, 15, and 17) enriched in the cSC cluster. Among them, KRT4, 13, 80, 78, and 23 are common stemness keratins in normal stem cells, while KRT10, 18, 8, 19, 6C, and 16 are unique cSC associated keratins, mostly identified by bulk RNA-seq ([Figure 6](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f006)).

![Ijms 22 08109 g007 550](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g007-550.jpg)

**Figure 7.** Keratin profiling in tumor spheroids using single-cell RNA-seq identifies cancer stem-like cells. (**A**,**B**) Uniform manifold approximation and projection (UMAP) clustering defined five clusters in cancer cells, identifying cluster 2 as cancer stem-like cells enriched for stemness markers, including KRT13. (**C**) Keratin gene profiling by heatmap and (**D**) dot-plot analyses identified a group of keratin genes enriched in the cluster 2 cancer stem-like cells.

## 3\. Discussion

Overall, the present results show that prostate stem cells can undergo both symmetric and asymmetric cell division in a 3D culture system giving rise to daughter stem and progenitor cells that lineage commit to separate basal and luminal progenitors, primed for further differentiation. Prostate epithelial stem cells play critical roles during prostate development and glandular tissue replenishment. Prostate stem cells generate progenitor cells that transiently amplify and differentiate to mature epithelial lineages \[[11](https://www.mdpi.com/1422-0067/22/15/#B11-ijms-22-08109),[12](https://www.mdpi.com/1422-0067/22/15/#B12-ijms-22-08109),[13](https://www.mdpi.com/1422-0067/22/15/#B13-ijms-22-08109)\]. During prostate epithelial cell differentiation, evidence suggests that there may be a common precursor stem cell for all lineages \[[14](https://www.mdpi.com/1422-0067/22/15/#B14-ijms-22-08109),[15](https://www.mdpi.com/1422-0067/22/15/#B15-ijms-22-08109),[16](https://www.mdpi.com/1422-0067/22/15/#B16-ijms-22-08109)\], while other findings support distinct basal and luminal stem cell populations within the adult prostate \[[17](https://www.mdpi.com/1422-0067/22/15/#B17-ijms-22-08109),[18](https://www.mdpi.com/1422-0067/22/15/#B18-ijms-22-08109)\]. In a murine model, Shen et al. reported that single luminal epithelial progenitors can generate prostate organoids in culture \[[50](https://www.mdpi.com/1422-0067/22/15/#B50-ijms-22-08109)\]. Gao et al. have shown that murine prostate basal cells undergo both symmetric and asymmetric divisions, leading to different cell fates that contribute to the luminal population and tumorigenesis, while luminal cells only exhibit symmetric divisions \[[12](https://www.mdpi.com/1422-0067/22/15/#B12-ijms-22-08109)\]. Yet these scenarios are not mutually exclusive, as emerging data indicate the inherent plasticity and stage/context-specific utilization of stem and progenitor cell populations \[[11](https://www.mdpi.com/1422-0067/22/15/#B11-ijms-22-08109),[12](https://www.mdpi.com/1422-0067/22/15/#B12-ijms-22-08109),[51](https://www.mdpi.com/1422-0067/22/15/#B51-ijms-22-08109)\]. We and others have previously demonstrated the clonal origin of single human prostate stem cell-derived prostaspheres \[[19](https://www.mdpi.com/1422-0067/22/15/#B19-ijms-22-08109),[20](https://www.mdpi.com/1422-0067/22/15/#B20-ijms-22-08109)\]. More recently, using a prostasphere-based label retaining assay, we identified rare prostate stem cells in human specimen-derived prostaspheres and determined that the vast majority of cells are daughter progenitor cells or their progeny \[[23](https://www.mdpi.com/1422-0067/22/15/#B23-ijms-22-08109),[24](https://www.mdpi.com/1422-0067/22/15/#B24-ijms-22-08109)\]. These stem cell models provide an excellent opportunity to trace the lineage of prostate epithelial cells differentiation in human samples. Furthermore, single cell RNA-seq analysis now permits the identification of the stem and progenitor subpopulations \[[34](https://www.mdpi.com/1422-0067/22/15/#B34-ijms-22-08109),[35](https://www.mdpi.com/1422-0067/22/15/#B35-ijms-22-08109)\], while keratin profiling helps to delineate the hierarchical biological time of progenitor cell lineage commitment \[[29](https://www.mdpi.com/1422-0067/22/15/#B29-ijms-22-08109),[30](https://www.mdpi.com/1422-0067/22/15/#B30-ijms-22-08109)\]. The results of the present study show that there are distinct subpopulations of both stem and progenitor cells at different stages in the epithelial lineage hierarchy of the normal prostate and indicate that committed basal and luminal progenitor cells are derived from a common prostate stem cell.

Keratins are useful biological markers for determining the origin and differentiation status of specific epithelial cells in normal tissues as well as their malignant counterparts \[[28](https://www.mdpi.com/1422-0067/22/15/#B28-ijms-22-08109),[29](https://www.mdpi.com/1422-0067/22/15/#B29-ijms-22-08109),[30](https://www.mdpi.com/1422-0067/22/15/#B30-ijms-22-08109),[52](https://www.mdpi.com/1422-0067/22/15/#B52-ijms-22-08109),[53](https://www.mdpi.com/1422-0067/22/15/#B53-ijms-22-08109)\]. Keratins serve both mechanical and non-mechanical cellular functions and have been recognized as regulators of various cellular properties and functions, including apicobasal polarization, motility, cell size, protein synthesis, and membrane traffic and signaling \[[25](https://www.mdpi.com/1422-0067/22/15/#B25-ijms-22-08109),[26](https://www.mdpi.com/1422-0067/22/15/#B26-ijms-22-08109),[27](https://www.mdpi.com/1422-0067/22/15/#B27-ijms-22-08109)\]. Using a series of human prostate autoptic tissues of various gestational ages and immunostaining with a comprehensive panel of keratin antibodies, Trompetter et al. identified luminal-type keratins 7, 8, 18, 19, and 20 as well as basal/squamous-type keratins 5, 6, 13, 14, and 17 in the developing fetal human prostate. They proposed that prostate stem cells may contain only keratin 8 but not 5 or 14 \[[29](https://www.mdpi.com/1422-0067/22/15/#B29-ijms-22-08109)\].

In this study, to monitor stem cell differentiation, we evaluated the expression patterns of keratin genes along with other stem cell markers to classify each subpopulation in the prostate stem and progenitor cell pool. KRT13, 23, 80, 78, 86, and 4 were enriched in label-retaining prostate stem cells, while KRT6, 17, 14, 5, 8, 18, and P63 were enriched in non-label retaining progenitors. Importantly, single-cell sequencing permitted the identification of the dynamic process of lineage progression within the label-retaining stem-like cell population, as the expression of these and additional keratins rapidly shifted upon early cell division, allowing the temporal mapping of the prostate stem cell lineage hierarchy from quiescent to active stem cells and the bipotent progenitor cell stages. Similar keratin profile shifts were found in the non-labeled progenitor population, enabling the identification of the distinct basal and luminal cell progenitor populations. It is important to note that cells in day 5 spheroids do not express detectable androgen receptor, HOXB13, or PSA, indicating that the model system represents the early stages of lineage commitment and not full differentiation \[[19](https://www.mdpi.com/1422-0067/22/15/#B19-ijms-22-08109),[21](https://www.mdpi.com/1422-0067/22/15/#B21-ijms-22-08109),[23](https://www.mdpi.com/1422-0067/22/15/#B23-ijms-22-08109),[24](https://www.mdpi.com/1422-0067/22/15/#B24-ijms-22-08109)\]. Based on these new data, we propose a model of human prostate stem cell hierarchy in the adult gland ([Figure 8](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f008)). Triggered by extrinsic signals from the stem cell niche, quiescent prostate stem cells enriched in KRT13, 23, 80, 78, and 4 (cluster I) enter the cell cycle and upon division become active stem cells (cluster II) with a concomitant decrease in expression of these stemness keratins. With subsequent cell division that triggers increased KRT 16, 17, and 6 expression, active stem cells generate bipotent progenitor cells (cluster III). The transient amplification of bipotent progenitor cells gives rise to unipotent basal progenitors (cluster IV: enriched in KRT5, 14, 6, and 16), as well as early stage luminal progenitors (cluster V: enriched in KRT8, 18, and 10, with medium expression of KRT5, 14, 6, and 16) that further lineage progress to late-stage luminal progenitors (cluster VI: enriched in KRT8, 18, and 10, with the lowest levels of KRT5, 14, 6, and 16) upon continued division. Thus, common bipotent prostate progenitor cells give rise to committed unipotent basal and luminal progenitor cells, which further supports terminal differentiation toward both lineages ([Figure 8](https://www.mdpi.com/1422-0067/22/15/#fig_body_display_ijms-22-08109-f008)). It is important to point out here that these separated cell populations—with stemness activity—have not been previously interrogated in this manner, so this is the first report of putative quiescent and active stem cells as well as bipotent progenitor cells for the human prostate gland. As such, there are no known markers for these three populations to use for confirmatory studies. Enriched genes in these separate populations are currently being interrogated in depth in our lab and will be the subject of a future publication.

![Ijms 22 08109 g008 550](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g008-550.jpg)

**Figure 8.** Model of prostate epithelial stem cell hierarchy based upon keratin profiles. Quiescent prostate stem cells (cluster I: enriched for KRT13, 23, 80, 78, 4) become active stem cells (cluster II), with increased expression of KRT 16, 17, and 6 that generate bipotent progenitor cells (cluster III) and further give rise to committed unique potent basal progenitors (cluster IV: enriched for KRT5, 14, 6, 16), as well as early (cluster V) and late-stage (cluster VI) luminal progenitors (enriched for KRT8, 18, 10).

Keratins are also used extensively as diagnostic tumor markers, as epithelial malignancies largely maintain the specific keratin patterns associated with their respective cells of origin \[[28](https://www.mdpi.com/1422-0067/22/15/#B28-ijms-22-08109),[52](https://www.mdpi.com/1422-0067/22/15/#B52-ijms-22-08109),[53](https://www.mdpi.com/1422-0067/22/15/#B53-ijms-22-08109)\]. It is widely accepted that most cells in human prostatic adenocarcinomas have keratin phenotypes resembling those of luminal cells but not basal cells, and the lack of basal keratins is a key diagnostic marker for pathologists \[[30](https://www.mdpi.com/1422-0067/22/15/#B30-ijms-22-08109)\]. LNCaP, an androgen-dependent prostate cancer cell line, expresses KRT18 but not KRT5 \[[30](https://www.mdpi.com/1422-0067/22/15/#B30-ijms-22-08109)\]; however, androgen-independent lines DU145 and PC3 both express basal cell keratins KRT5. The expression of KRT5 in the absence of KRT14 identifies the existence of an intermediate cell population in prostate carcinomas, and the number of KRT5 -expressing cells increases significantly after androgen deprivation therapy \[[54](https://www.mdpi.com/1422-0067/22/15/#B54-ijms-22-08109)\]. Furthermore, aberrant cytoplasmic expression of basal cell marker p63 is associated with increased PCa stem cells as well as PCa mortality \[[55](https://www.mdpi.com/1422-0067/22/15/#B55-ijms-22-08109),[56](https://www.mdpi.com/1422-0067/22/15/#B56-ijms-22-08109)\]. As such, it is suggested that the re-expression of basal cell markers including basal keratins may represent the progression to an invasive stage of therapy-resistant PCa. Stem cells are closely associated with tumorigenesis in PCa \[[57](https://www.mdpi.com/1422-0067/22/15/#B57-ijms-22-08109),[58](https://www.mdpi.com/1422-0067/22/15/#B58-ijms-22-08109),[59](https://www.mdpi.com/1422-0067/22/15/#B59-ijms-22-08109),[60](https://www.mdpi.com/1422-0067/22/15/#B60-ijms-22-08109),[61](https://www.mdpi.com/1422-0067/22/15/#B61-ijms-22-08109)\], and cSCs are largely enhanced during PCa progression and after conventional therapy \[[62](https://www.mdpi.com/1422-0067/22/15/#B62-ijms-22-08109),[63](https://www.mdpi.com/1422-0067/22/15/#B63-ijms-22-08109),[64](https://www.mdpi.com/1422-0067/22/15/#B64-ijms-22-08109)\]. Thus, increased basal cell markers and keratin expression during PCa metastasis and after androgen deprivation therapy may be an indicator of cSCs expansion. This is supported by our current findings that cSCs express both stemness and basal keratin genes in PCa. We herein demonstrate the enrichment of the “cytoskeleton remodeling–keratin filaments” pathway in cSCs from PCa specimens, identifying both normal stemness genes (PSCA, CD36, and SPINK1) and stem cell keratins (KRT13, 23, 80, 78, and 4) as well as distinctive keratins (KRT10, 19, 6C, and 16) in the cSC population. Thus, this profiling allows a reliable gene expression fingerprint to identify the stem-like population in PCa specimens and assess unique and/or enriched genes that can be used for stem-cell isolation, as biomarkers, and for druggable targets to eliminate cancer stem-like cells that are classically resistant to hormone and chemotherapeutic therapeutic modalities.

Taken together, the detailed clarification of the prostate epithelial cell lineage hierarchy by single cell RNA-seq analysis and keratin profiling of human prostate stem cells and cSCs provides abundant opportunities for future translational studies that therapeutically target stem cell populations in both benign prostatic disease and PCa.

## 4\. Materials and Methods

#### 4.1. Prostate Epithelial Cells and Prostasphere Culture

Primary normal human PrECs were obtained from young (19–21 years of age) disease-free organ donors (Lonza, Walkersville, MD, USA) at passage 2–3 and cultured at 37 °C, 5% CO <sub>2</sub> in ProstaLife Epithelial Cell Growth Medium (PrEGM) (LifeLine Cell Technology, Carlsbad, CA, USA) on fibronectin-coated flasks. PrECs from four donors were used, and all exhibited similar growth and hormone responsive behaviors. Prostaspheres were cultured from PrECs as previously described and confirmed to be clonally derived spheroids of stem/progenitor cells \[[19](https://www.mdpi.com/1422-0067/22/15/#B19-ijms-22-08109),[21](https://www.mdpi.com/1422-0067/22/15/#B21-ijms-22-08109),[22](https://www.mdpi.com/1422-0067/22/15/#B22-ijms-22-08109)\]. Briefly, 1 × 10 <sup>5</sup> PrEC cells were resuspended in 1:1 Matrigel (Corning/BD Biosciences, Corning, NY, USA)/PrEGM with 1 mL PrEGM and cultured at 37 °C in 5% CO <sub>2</sub>.

Primary human prostate cancer epithelial cells (PrE-Ca) were isolated from radical prostatectomy tissue from a prostate cancer patient at the University of Illinois at Chicago Medical Center, as previously described \[[38](https://www.mdpi.com/1422-0067/22/15/#B38-ijms-22-08109)\]. Briefly, fresh tissue from the peripheral zone was selected and excised with a 5 mm punch by a pathologist according to an Institutional Review Board-approved protocol. Final tissue pathology was determined by H&E of a thin slice of the punch, and the area was confirmed to be 100% cancer. PrE-Ca cells were maintained in PrEGM medium.

#### 4.2. Prostasphere-Based Label Retention Assays

Prostasphere-based label retention assays were performed based on the protocol recently established in our lab \[[23](https://www.mdpi.com/1422-0067/22/15/#B23-ijms-22-08109),[24](https://www.mdpi.com/1422-0067/22/15/#B24-ijms-22-08109)\]. For prostasphere-based BrdU retention assays, parental PrECs or primary prostate cancer cells were cultured in 2D in the presence of 1 µM BrdU for 10 days. Then, pre-labeled prostate cells were transferred to 3D Matrigel prostasphere culture for 5 days (≈6 cell cycles with a doubling time of 20 h) in the absence of BrdU to wash-out. The prostaspheres were harvested from Matrigel by dispase digestion on day 5 and plated on chamber slides for overnight culture, allowing spheroid cells to attach and grow outwards on the chamber slides. Then, cells were fixed in ice-cold acetone/methanol (1:1), and BrdU-retaining cells were identified by immunostaining using a mouse anti-BrdU antibody. Images were taken using a Zeiss Axioskop 20 fluorescent microscope, and the numbers of prostasphere clusters and BrdU-positive labeled cells were counted.

For the CFSE label-retention assay (100% overlap with BrdU) in live cells \[[23](https://www.mdpi.com/1422-0067/22/15/#B23-ijms-22-08109),[24](https://www.mdpi.com/1422-0067/22/15/#B24-ijms-22-08109)\], 2D-cultured parental PrECs or primary prostate cancer cells were labeled by 5 µM CFSE (Life Technologies, Grand Island, NY, USA) for 30 min. Labeled PrECs were plated in 3D Matrigel in the absence of CFSE and cultured for 5 days to form prostaspheres. Prostaspheres were harvested as above and dispersed into single cells followed by FACS sorting to separate CFSE <sup>High</sup> and CFSE <sup>Low</sup> cells.

#### 4.3. Gene Knockdown by SiRNA

For siRNA knockdown of the KRT13 gene, PrE-Ca cancer cells (≈70% confluence) in 2D culture were incubated with 40 nM of siControl (scramble) or si KRT13 (IDT, Coralville, IA, USA) for 6 h, respectively. Then, 1 × 10 <sup>4</sup> /well of siRNA treated PrE-Ca cells with or without CFSE labeling were resuspended in 1:1 Matrigel/PrEGM plated in a 24-well plate for 7 days to form tumor sphere, and the medium was replenished with fresh 40 nM siRNA on day 3 of 3D culture. Tumor sphere numbers and the CFSE <sup>High</sup> % cells were evaluated as described previously \[[23](https://www.mdpi.com/1422-0067/22/15/#B23-ijms-22-08109),[24](https://www.mdpi.com/1422-0067/22/15/#B24-ijms-22-08109)\].

#### 4.4. Next-Generation Bulk RNA-Sequencing Analysis

CFSE-labeled PrEC cells from three healthy donors and PCa cells from three patient specimens were used to grow prostaspheres or tumor spheres in 3D cultures. CFSE <sup>High</sup> and CFSE <sup>Low</sup> cells were separated by FACS described in [Section 4.5](https://www.mdpi.com/1422-0067/22/15/#sec4dot5-ijms-22-08109). Transcriptome profiles of CFSE <sup>High</sup> and CFSE <sup>Low</sup> cells were determined using next-generation RNA-seq performed by Dr. Shioda at the Center for Cancer Research at Harvard Medical School \[[48](https://www.mdpi.com/1422-0067/22/15/#B48-ijms-22-08109)\]. Briefly, following total RNAs isolation from CFSE <sup>High</sup> and CFSE <sup>Low</sup> cells, cDNA synthesis, and fragmentation, indexed deep sequencing libraries for Illumina NextSeq 500 sequencing were generated, and library quantitation and size distributions were determined using Agilent TapeStation. Paired-end sequencing (75 nt + 75 nt) yielded > 50 million raw reads, which were aligned to the human GRCh38/hg38 reference genome sequence. See Supplemental Methods for further details and bioinformatics analysis.

#### 4.5. Flow Cytometry

Analysis of trypsin-dispersed CFSE-labeled prostasphere cells was performed by single-channel FACS (CyAn <sup>TM</sup> ADP Analyzer, Beckman Coulter Inc., Brea, CA, USA) using the Summit Software polygon tool. Prostasphere cells freshly labeled with 5 µM CFSE before FACS sorting were used as positive control, while unlabeled prostasphere cells were used as negative control. Subpopulations of fractionated CFSE <sup>High</sup> and CFSE <sup>Low</sup> cells were gated based on negative and positive controls using the FACS-DiVa software polygon tool and collected by CellSorter (MoFlo XDP, Beckman Coulter) \[[23](https://www.mdpi.com/1422-0067/22/15/#B23-ijms-22-08109),[24](https://www.mdpi.com/1422-0067/22/15/#B24-ijms-22-08109)\].

#### 4.6. Single-Cell Capture Using the Fluidigm C1-IFC System and Single-Cell RNA-Seq by NextSeq500 Deep Sequencing

PrECs were grown as prostaspheres to enrich and separate the primary stem and daughter progenitor populations. The prostasphere-based CFSE retaining assay \[[23](https://www.mdpi.com/1422-0067/22/15/#B23-ijms-22-08109),[24](https://www.mdpi.com/1422-0067/22/15/#B24-ijms-22-08109)\] was used to fluorescently label the relatively quiescent prostate stem cells. CFSE <sup>High</sup> stem cells and CFSE <sup>Low</sup> progenitor cell fractions were separated by FACS, and the individual populations were separately captured as single cells using the 96-well C1-IFC microfluidic system (Fluidigm, South San Francisco, CA, USA) at the University of Illinois at Chicago (UIC) Research Resource Center. Captured single cells were lysed and reverse transcribed into bar-coded cDNA in each cell of IFC microfluidic chambers. cDNA was ligated to adaptor sequences to construct deep sequencing libraries. The 5′-end and 3′-end of each single-stranded RNA transcript were ligated to different bar-coded adaptors to conserve strand information. Pooled, bar-coded, single-cell cDNA samples were transferred for RNA-seq using the Illumina NextSeq500 deep sequencer at the UIC Genomics Core. Each sequencer run generated 400 million reads, 0.5–2 million reads (paired-end, 75 nt + 75 nt) for each single-cell mRNA sample. We validated ≈150 single-cell cDNA libraries from each sample and ≈10,000 genes per cell, which covers enough coding mRNA for data analysis.

#### 4.7. Single-Cell Capture by the 10XGenomics Platform and Single-Cell RNA-Seq Using NovaSeq 6000

Tumor spheroids derived from primary human PCa epithelial cells were cultured for 7 days and dispersed into a single-cell suspension. Single cells were separated and captured at a 10X Genomics station using Chromium Next GEM Single Cell 3ʹ v3 Reagent Kits (10XGenomics, CG000183). After GEM generation and barcoding, mRNA was reverse transcribed into cDNA, which was further amplified for library construction followed by RNA-seq using NovaSeq 6000 SP at the University of Illinois Keck Center. Each lane of the sequencer generated 1.6 billion reads, 0.4 billion reads per sample, 0.1 million reads (paired-end, 28 nt + 91 nt) for each single-cell. We validated ≈3500 single-cell cDNA libraries from each sample and ≈7500 genes per cell for data analysis.

#### 4.8. Single-Cell Data Analysis

For the Fluidigm C1 system, RNA-seq data generated from single cells were analyzed by bioinformatics specialists at the UIC Center of Research Informatics. Generated nucleotide base sequences were aligned on the human hg38 reference genome sequence using TopHat2. QC-passed data underwent annotation analysis by cufflinks to calculate FPKM (fragments per kilobase exon/million mapped fragments), reflecting expressed transcript amounts. The normalized FPKM of all single cells was analyzed using the Monocle algorithm \[[34](https://www.mdpi.com/1422-0067/22/15/#B34-ijms-22-08109)\], which is an unsupervised method with increased temporal resolution of transcriptome dynamics designed for single-cell RNA-seq data analysis. TSCAN was used to analyze temporal transcript dynamics of a pseudotime stem→progenitor cell process to detect genes contributing to cell identity \[[35](https://www.mdpi.com/1422-0067/22/15/#B35-ijms-22-08109)\]. Hierarchical clustering was used to differentiate meaningful cell signatures. Monocle analyzes the temporal transcript dynamics of the pseudo-time prostate stem → progenitor cells process to detect genes contributing to cell identity. Heatmaps and dendrograms of hierarchical clustering of gene signatures distinguish diverse cell types and identify cell type-specific genes. Enrichment of differentially expressed genes was examined using GSEA. MetaCore Pathway Analysis was used to identify the top enriched pathways across cSC populations.

GSEA was performed on normalized gene expression data from CFSE <sup>High</sup> and CFSE <sup>Low</sup> samples generated by RNA-seq to extract biological knowledge using the Broad Institute’s Molecular Signatures Database \[[65](https://www.mdpi.com/1422-0067/22/15/#B65-ijms-22-08109)\]. Analyses were run using the gene expression data against two gene sets: C2: curated gene sets and CP: BIOCARTA: BioCarta gene sets. The false discovery rate (FDR) was calculated by comparing the actual data with 1000 Monte Carlo simulations. The familywise error rate is a conservative correction that seeks to ensure that reported results do not include any false-positive gene sets.

For the 10XGenomic platform, single-cell RNA-seq data generated by NovaSeq 6000 was aligned against hg38 reference genome sequence using 10x Cellranger pipeline. The aligned dataset was imported into Seurat package (Satijalab). Pre-processing of raw data included filtering out cell debris, unhealthy singlets, and potential doublets by setting cutoff based on the distribution of the feature RNA abundance (cutoff range was from 2000 to 7500; each data point indicates a singlet or single cell) and the percentage cutoff of mitochondrial genes within each data point was 10%. Post-filtered single cell feature expression values were normalized by the total expression through a global-scaling normalization method “LogNormalize”. The scale factor was set at the default value of 10,000. Subsequent feature selection was performed with default setting (nfeature = 2000) to gain a subset of features that reflect high cell-to-cell variation of the dataset. The expression value of each gene was scaled so that data points were centralized around the origin, and the variance across these data points equals to 1. Linear dimensional reduction was performed on the subset of feature genes selected by PCA. Next, a resampling test inspired by the Jackstraw procedure was utilized to determine the PC number to include. The data points or single cells were finally clustered based on PCs determined previously using a graph-based clustering approach based on Macosko et al. \[[66](https://www.mdpi.com/1422-0067/22/15/#B66-ijms-22-08109)\]. Pre-defined PCs were introduced to the clustering analysis to plot UMAP for the better visualization of clusters. Subsequent identification of each cluster was performed mainly by keratin profiling and according to canonical biomarkers \[[67](https://www.mdpi.com/1422-0067/22/15/#B67-ijms-22-08109)\].

#### 4.9. Immunocytochemistry (ICC)

ICC was performed on prostaspheres attached overnight on chamber slides (Millipore, Billerica, MA, USA) to permit limited cell outgrowth and improve reagent penetration \[[21](https://www.mdpi.com/1422-0067/22/15/#B21-ijms-22-08109),[22](https://www.mdpi.com/1422-0067/22/15/#B22-ijms-22-08109),[23](https://www.mdpi.com/1422-0067/22/15/#B23-ijms-22-08109),[24](https://www.mdpi.com/1422-0067/22/15/#B24-ijms-22-08109)\]. Spheres were fixed in ice-cold acetone/methanol (1:1) and blocked in 5% normal goat serum in PBST (PBS with 0.25% Triton X-100) for 30 min at room temperature. After acid wash with 2N HCl for 30 min at room temperature, spheres were incubated with primary antibodies diluted in 2% normal goat sera in PBST overnight at 4 °C, rinsed 3X for 5 min in PBS, and incubated with secondary goat antibody (diluted 1:1000 in 2% normal goat serum in PBST) at room temperature for 2 h. Primary anti-human antibodies used for double staining are anti-BrdU antibody (1:200, #ab152095, Abcam, San Francisco, CA, USA), anti- KRT13 antibody (1:200, #ab97327, Abcam, San Francisco, CA, USA), anti- KRT14 antibody (1:200, #prb-155p, Covance Inc., Princeton, NJ, USA), anti- KRT80 antibody (1:2000, #16835-1-ap, Proteintech, Rosemont, IL, USA), anti- KRT4 antibody (1:400, Abcam, #ab51599, San Francisco, CA, USA), anti- KRT78 antibody (1:2000, #PA5-58902, Invitrogen, Waltham, WA, USA), anti- KRT23 antibody (1:400, #sc365892, Santa Cruz Biotechnology, Dallas, TX, USA), anti- KRT19 antibody (1:400, #NBP2-22116SS, Novus, Centennial, CO, USA), anti-E-cadherin antibody (1:100, #sc8426, Santa Cruz Biotechnology, Dallas, TX, USA), anti-LC3 antibody (1:200, #12741p, Cell Signaling Technology, Danver, MA, USA), anti-CD36 antibody (1:150, #mab19553-sp, Novus, Centennial, CO, USA), anti-SPINK1 antibody (1:200, #mab7496, R&D Systems, Irvine, CA, USA), and anti-PSCA antibody (1:50, #abx236848, Abbexa, Cambridge, UK). Secondary goat anti-rabbit Alexa Fluor 568 (1:2000, #ab175471, Abcam, San Francisco, CA, USA) was used. Following rinses in PBS, slides were cover slipped with fluorescence mounting medium with DAPI (Vectashield, Vector Laboratories, Burlingame, CA, USA). Mouse and rabbit IgG (eBioscience, San Diego, CA, USA) antibodies were used as negative controls. Images of stained spheres/cells were obtained on a Zeiss Axioskop 20 fluorescent microscope with an Axiocamera or a Zeiss LSM 510 META Confocal Microscope.

#### 4.10. Statistical Analysis

Significant differentially expressed transcripts between groups of single cells were identified with Benjamini–Hochberg corrected p -value (FDR) for multiple comparisons. Experiments for in vitro studies were biologically replicated 3 times. Data were analyzed using Student’s t- test. Values are expressed as mean ± SEM, and p < 0.05 was considered significant.

## 5\. Conclusions

Our novel sphere-based label retention assay followed by single-cell RNA-seq analysis and keratin profiling provides a unique tool to study prostate epithelial cell lineage hierarchy and cancer stem-like cells. Its broad application further enables discovery of potential biomarkers for prostate cSCs and creates new strategies for managing therapy-resistant prostate cancer.

## Author Contributions

Conceptualization, W.-Y.H. and G.S.P.; methodology, D.-P.H., W.-Y.H., and L.X.; software, D.-P.H. and R.L.; validation, W.-Y.H. and G.S.P.; formal analysis, W.-Y.H.; investigation, W.-Y.H.; resources, G.S.P., W.-Y.H.; T.S., M.A., and L.N.; data curation, W.-Y.H.; writing—original draft preparation, W.-Y.H.; writing—review and editing, W.-Y.H. and G.S.P.; visualization, W.-Y.H.; supervision, W.-Y.H. and G.S.P. All authors have read and agreed to the published version of the manuscript.

## Funding

This research was funded by NIH/NIEHS (RO1ES022071, G.S.P. and W.-Y.H.); NCI (RO1CA172220-01, G.S.P. and W.-Y.H.); DoD (PC180408, G.S.P. and W.-Y.H.), Michael Reese Research and Education Foundation Translational Research Grant Award (G.S.P. and W.-Y.H.).

## Institutional Review Board Statement

Not applicable.

## Informed Consent Statement

Not applicable.

## Data Availability Statement

Data are available in GEO data portal (GSE95542).

## Acknowledgments

We thank Zarema Arbieva, Donald Vander Griend, and Dan Moline for their help with single cell capture. Hong Hu and Mark Maienschein-Cline for their help with single-cell data analysis. Lynn A Birch and Fresh Eyes Editing for manuscript editing.

## Conflicts of Interest

The authors declare no conflict of interest.

## Abbreviations

| PrEC | Human prostate epithelial cells |
| --- | --- |
| PrEGM | ProstaLife epithelial cell growth medium |
| PCa | Prostate cancer |
| KRT | Keratin |
| cSC | Cancer stem-like cells |
| PSCA | Prostate stem cell antigen |
| FACS | Fluorescence-activated cell sorting |
| PCA | Principal component analysis |
| TSCAN | Tool of single cell analysis |
| ICC | Immunocytochemistry |
| CFSE | 5(6)-Carboxyfluorescein N-hydroxysuccinimidyl ester |
| BrdU | Bromodeoxyuridine/5-bromo-2′-deoxyuridine |
| GSEA | Gene set enrichment analysis |
| PSA | Prostate-specific antigen |

## References

1. Presnell, S.C.; Petersen, B.; Heidaran, M. Stem cells in adult tissues. Semin. Cell Dev. Biol.**2002**, 13, 369–376. \[\] \[[CrossRef](http://doi.org/10.1016/S1084952102000939)\]
2. Smith, S.; Neaves, W.; Teitelbaum, S. Adult Versus Embryonic Stem Cells: Treatments. Science **2007**, 316, 1422–1423. \[\] \[[CrossRef](http://doi.org/10.1126/science.316.5830.1422b)\] \[[PubMed](http://www.ncbi.nlm.nih.gov/pubmed/17556566)\]
3. De Marzo, A.M.; Nelson, W.G.; Meeker, A.K.; Coffey, D.S. Stem cell features of benign and malignant prostate epithelial cells. J. Urol.**1998**, 160, 2381–2392. \[\] \[[CrossRef](http://doi.org/10.1016/S0022-5347\(01\)62196-7)\]
4. Goldstein, A.S.; Lawson, D.A.; Cheng, D.; Sun, W.; Garraway, I.P.; Witte, O.N. Trop2 identifies a subpopulation of murine and human prostate basal cells with stem cell characteristics. Porc. Natl. Acad. Sci. USA **2008**, 105, 20882–20887. \[\] \[[CrossRef](http://doi.org/10.1073/pnas.0811411106)\] \[[Green Version](http://www.pnas.org/content/105/52/20882.full.pdf)\]
5. Prins, G.S.; Lindgren, M. Accessory sex glands in the male. In Knobil and Neill’s Physiology of Reproduction, 4th ed.; Plant, T.M., Zeleznik, A.J., Eds.; Academic Press: Cambridge, MA, USA, 2015; pp. 773–804. \[\]
6. Xin, L.; Lukacs, R.U.; Lawson, D.A.; Cheng, D.; Witte, O.N. Self-renewal and multilineage differentiation in vitro from murine prostate stem cells. Stem Cells **2007**, 25, 2760–2769. \[\] \[[CrossRef](http://doi.org/10.1634/stemcells.2007-0355)\]
7. Leong, K.G.; Wang, B.E.; Johnson, L.; Gao, W.Q. Generation of a prostate from a single cell. Nature **2008**, 456, 804–808. \[\] \[[CrossRef](http://doi.org/10.1038/nature07427)\]
8. Prins, G.S.; Birch, L. The developmental pattern of androgen receptor expression in rat prostate lobes is altered after neonatal exposure to estrogen. Endocrinology **1995**, 136, 1303–1314. \[\] \[[CrossRef](http://doi.org/10.1210/endo.136.3.7867585)\]
9. Wang, X.; Xu, H.; Cheng, C.; Ji, Z.; Zhao, H.; Sheng, Y.; Li, X.; Wang, J.; Shu, Y.; He, Y.; et al. Identification of a Zeb1 expressing basal stem cell subpopulation in the prostate. Nat. Commun.**2020**, 11, 706. \[\] \[[CrossRef](http://doi.org/10.1038/s41467-020-14296-y)\] \[[Green Version](https://www.nature.com/articles/s41467-020-14296-y.pdf)\]
10. Moad, M.; Hannezo, E.; Buczacki, S.J.; Wilson, L.; El-Sherif, A.; Sims, D.; Pickard, R.; Wright, N.A.; Williamson, S.C.; Turnbull, D.M.; et al. Multipotent Basal Stem Cells, Maintained in Localized Proximal Niches, Support Directed Long-Ranging Epithelial Flows in Human Prostates. Cell Rep.**2017**, 20, 1609–1622. \[\] \[[CrossRef](http://doi.org/10.1016/j.celrep.2017.07.061)\] \[[Green Version](http://www.cell.com/article/S2211124717310409/pdf)\]
11. Blackwood, J.K.; Williamson, S.C.; Greaves, L.C.; Wilson, L.; Rigas, A.C.; Sandher, R.; Pickard, R.S.; Robson, C.N.; Turnbull, D.M.; Taylor, R.W.; et al. In situ lineage tracking of human prostatic epithelial stem cell fate reveals a common clonal origin for basal and luminal cells. J. Pathol.**2011**, 225, 181–188. \[\]
12. Wang, J.; Zhu, H.H.; Chu, M.; Liu, Y.; Zhang, C.; Liu, G.; Yang, X.; Yang, R.; Gao, W.Q. Symmetrical and asymmetrical division analysis provides evidence for a hierarchy of prostate epithelial cell lineages. Nat. Commun.**2014**, 5, 4758. \[\] \[[CrossRef](http://doi.org/10.1038/ncomms5758)\] \[[PubMed](http://www.ncbi.nlm.nih.gov/pubmed/25163637)\] \[[Green Version](https://www.nature.com/articles/ncomms5758.pdf)\]
13. Horton, C.; Liu, Y.; Yu, C.; Xie, Q.; Wang, Z.A. Luminal-contact-inhibition of epithelial basal stem cell multipotency in prostate organogenesis and homeostasis. Biol. Open **2019**, 8, bio045724. \[\] \[[CrossRef](http://doi.org/10.1242/bio.045724)\] \[[Green Version](https://bio.biologists.org/content/biolopen/8/10/bio045724.full.pdf)\]
14. Wang, Z.A.; Mitrofanova, A.; Bergren, S.K.; Abate-Shen, C.; Cardiff, R.D.; Califano, A.; Shen, M.M. Lineage analysis of basal epithelial cells reveals their unexpected plasticity and supports a cell-of-origin model for prostate cancer heterogeneity. Nat. Cell Biol.**2013**, 15, 274–283. \[\] \[[CrossRef](http://doi.org/10.1038/ncb2697)\] \[[PubMed](http://www.ncbi.nlm.nih.gov/pubmed/23434823)\] \[[Green Version](http://europepmc.org/articles/pmc3743266?pdf=render)\]
15. Crowley, L.; Cambuli, F.; Aparicio, L.; Shibata, M.; Robinson, B.D.; Xuan, S.; Li, W.; Hibshoosh, H.; Loda, M.; Rabadan, R.; et al. A single-cell atlas of the mouse and human prostate reveals heterogeneity and conservation of epithelial progenitors. eLife **2020**, 9, e59465. \[\] \[[CrossRef](http://doi.org/10.7554/eLife.59465)\]
16. Toivanen, R.; Mohan, A.; Shen, M.M. Basal Progenitors Contribute to Repair of the Prostate Epithelium Following Induced Luminal Anoikis. Stem Cell Rep.**2016**, 6, 660–667. \[\] \[[CrossRef](http://doi.org/10.1016/j.stemcr.2016.03.007)\] \[[Green Version](http://www.cell.com/article/S2213671116300029/pdf)\]
17. Karthaus, W.R.; Iaquinta, P.J.; Drost, J.; Gracanin, A.; van Boxtel, R.; Wongvipat, J.; Dowling, C.M.; Gao, D.; Begthel, H.; Sachs, N.; et al. Identification of multipotent luminal progenitor cells in human prostate organoid cultures. Cell **2014**, 159, 163–175. \[\] \[[CrossRef](http://doi.org/10.1016/j.cell.2014.08.017)\] \[[Green Version](http://www.cell.com/article/S0092867414010484/pdf)\]
18. Karthaus, W.R.; Hofree, M.; Choi, D.; Linton, E.L.; Turkekul, M.; Bejnood, A.; Carver, B.; Gopalan, A.; Abida, W.; Laudone, V.; et al. Regenerative potential of prostate luminal cells revealed by single-cell analysis. Science **2020**, 368, 497–505. \[\] \[[CrossRef](http://doi.org/10.1126/science.aay0267)\]
19. Hu, W.Y.; Shi, G.B.; Lam, H.M.; Hu, D.P.; Ho, S.M.; Madueke, I.C.; Kajdacsy-Balla, A.; Prins, G.S. Estrogen-initiated transformation of prostate epithelium derived from normal human prostate stem-progenitor cells. Endocrinology **2011**, 152, 2150–2163. \[\] \[[CrossRef](http://doi.org/10.1210/en.2010-1377)\] \[[Green Version](https://academic.oup.com/endo/article-pdf/152/6/2150/9036621/endo2150.pdf)\]
20. Garraway, I.P.; Sun, W.; Tran, C.P.; Perner, S.; Zhang, B.; Goldstein, A.S.; Hahm, S.A.; Haider, M.; Head, C.S.; Reiter, R.E.; et al. Human prostate sphere-forming cells represent a subset of basal epithelial cells capable of glandular regeneration in vivo. Prostate **2010**, 70, 491–501. \[\] \[[CrossRef](http://doi.org/10.1002/pros.21083)\]
21. Hu, W.Y.; Shi, G.B.; Hu, D.P.; Nelles, J.L.; Prins, G.S. Actions of endocrine disrupting chemicals on human prostate stem/progenitor cells and prostate cancer risk. Mol. Cell. Endocrinol.**2012**, 354, 63–73. \[\] \[[CrossRef](http://doi.org/10.1016/j.mce.2011.08.032)\] \[[Green Version](http://europepmc.org/articles/pmc3249013?pdf=render)\]
22. Prins, G.S.; Hu, W.Y.; Shi, G.B.; Hu, D.P.; Majumdar, S.; Li, G.; Huang, K.; Nelles, J.; Ho, S.M.; Walker, C.L.; et al. Bisphenol A Promotes Human Prostate Stem-Progenitor Cell Self-Renewal and Increases In Vivo Carcinogenesis in Human Prostate Epithelium. Endocrinology **2014**, 155, 805–817. \[\] \[[CrossRef](http://doi.org/10.1210/en.2013-1955)\] \[[PubMed](http://www.ncbi.nlm.nih.gov/pubmed/24424067)\]
23. Hu, W.Y.; Hu, D.P.; Xie, L.; Li, Y.; Majumdar, S.; Nonn, L.; Shioda, T.; Prins, G.S. Isolation and functional interrogation of adult human prostate epithelial stem cells at single cell resolution. Stem Cell Res.**2017**, 23, 1–12. \[\] \[[CrossRef](http://doi.org/10.1016/j.scr.2017.06.009)\]
24. Hu, W.Y.; Hu, D.P.; Xie, L.; Birch, L.A.; Prins, G.S. Isolation of Stem-like Cells from 3-Dimensional Spheroid Cultures. J. Vis. Exp.**2019**, 154, e60357. \[\] \[[CrossRef](http://doi.org/10.3791/60357)\]
25. Moll, R.; Divo, M.; Langbein, L. The human keratins: Biology and pathology. Histochem. Cell Biol.**2008**, 129, 705–733. \[\] \[[CrossRef](http://doi.org/10.1007/s00418-008-0435-6)\] \[[Green Version](https://link.springer.com/content/pdf/10.1007/s00418-008-0435-6.pdf)\]
26. Bragulla, H.H.; Homberger, D.G. Structure and functions of keratin proteins in simple, stratified, keratinized and cornified epithelia. J. Anat.**2009**, 214, 516–559. \[\] \[[CrossRef](http://doi.org/10.1111/j.1469-7580.2009.01066.x)\]
27. Windoffer, R.; Beil, M.; Magin, T.M.; Leube, R.E. Cytoskeleton in motion: The dynamics of keratin intermediate filaments in epithelia. J. Cell Biol.**2011**, 194, 669–678. \[\] \[[CrossRef](http://doi.org/10.1083/jcb.201008095)\] \[[Green Version](http://jcb.rupress.org/content/jcb/194/5/669.full.pdf)\]
28. Karantza, V. Keratins in health and cancer: More than mere epithelial cell markers. Oncogene **2011**, 30, 127–138. \[\] \[[CrossRef](http://doi.org/10.1038/onc.2010.456)\] \[[Green Version](https://www.nature.com/articles/onc2010456.pdf)\]
29. Trompetter, M.; Smedts, F.; van der Wijk, J.; Schoots, C.; de Jong, H.J.; Hopman, A.; de la Rosette, J. Keratin profiling in the developing human prostate. A different approach to understanding epithelial lineage. Anticancer. Res.**2008**, 28, 237–243. \[\] \[[PubMed](http://www.ncbi.nlm.nih.gov/pubmed/18383851)\]
30. Okada, H.; Tsubura, A.; Okamura, A.; Senzaki, H.; Naka, Y.; Komatz, Y.; Morii, S. Keratin profiles in normal/hyperplastic prostates and prostate carcinoma. Virchows Arch. A Pathol. Anat. Histopathol.**1992**, 421, 157–161. \[\] \[[CrossRef](http://doi.org/10.1007/BF01607049)\]
31. Malecha, M.J.; Miettinen, M. Expression of keratin 13 in human epithelial neoplasms. Virchows Arch. A Pathol. Anat. Histopathol.**1991**, 418, 249–254. \[\] \[[CrossRef](http://doi.org/10.1007/BF01606063)\]
32. Li, Q.; Yin, L.; Jones, L.W.; Chu, G.C.; Wu, J.B.; Huang, J.M.; Li, Q.; You, S.; Kim, J.; Lu, Y.T.; et al. Keratin 13 expression reprograms bone and brain metastases of human prostate cancer cells. Oncotarget **2016**, 7, 84645–84657. \[\] \[[CrossRef](http://doi.org/10.18632/oncotarget.13175)\]
33. Liu, S.; Cadaneanu, R.M.; Zhang, B.; Huo, L.; Lai, K.; Li, X.; Galet, C.; Grogan, T.R.; Elashoff, D.; Freedland, S.J.; et al. Keratin 13 Is Enriched in Prostate Tubule-Initiating Cells and May Identify Primary Prostate Tumors that Metastasize to the Bone. PLoS ONE **2016**, 11, e0163232. \[\]
34. Trapnell, C.; Cacchiarelli, D.; Grimsby, J.; Pokharel, P.; Li, S.; Morse, M.; Lennon, N.J.; Livak, K.J.; Mikkelsen, T.S.; Rinn, J.L. The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells. Nat. Biotechnol.**2014**, 32, 381–386. \[\] \[[CrossRef](http://doi.org/10.1038/nbt.2859)\] \[[Green Version](http://europepmc.org/articles/pmc4122333?pdf=render)\]
35. Ji, Z.; Ji, H. TSCAN: Pseudo-time reconstruction and evaluation in single-cell RNA-seq analysis. Nucleic Acids Res.**2016**, 44, e117. \[\] \[[CrossRef](http://doi.org/10.1093/nar/gkw430)\] \[[Green Version](https://academic.oup.com/nar/article-pdf/44/13/e117/17437043/gkw430.pdf)\]
36. Lang, S.H.; Anderson, E.; Fordham, R.; Collins, A.T. Modeling the prostate stem cell niche: An evaluation of stem cell survival and expansion in vitro. Stem Cells Dev.**2010**, 19, 537–546. \[\] \[[CrossRef](http://doi.org/10.1089/scd.2009.0291)\] \[[Green Version](http://europepmc.org/articles/pmc3158388?pdf=render)\]
37. Blum, R.; Gupta, R.; Burger, P.E.; Ontiveros, C.S.; Salm, S.N.; Xiong, X.; Kamb, A.; Wesche, H.; Marshall, L.; Cutler, G.; et al. Molecular signatures of the primitive prostate stem cell niche reveal novel mesenchymal-epithelial signaling pathways. PLoS ONE **2010**, 5, e13024. \[\] \[[CrossRef](http://doi.org/10.1371/journal.pone.0013024)\] \[Green Version\]
38. Richards, Z.; McCray, T.; Marsili, J.; Zenner, M.L.; Manlucu, J.T.; Garcia, J.; Kajdacsy-Balla, A.; Murray, M.; Voisine, C.; Murphy, A.B.; et al. Prostate Stroma Increases the Viability and Maintains the Branching Phenotype of Human Prostate Organoids. iScience **2019**, 12, 304–317. \[\] \[[CrossRef](http://doi.org/10.1016/j.isci.2019.01.028)\] \[[Green Version](http://www.cell.com/article/S2589004219300306/pdf)\]
39. Gu, Z.; Thomas, G.; Yamashiro, J.; Shintaku, I.P.; Dorey, F.; Raitano, A.; Witte, O.N.; Said, J.W.; Loda, M.; Reiter, R.E. Prostate stem cell antigen (PSCA) expression increases with high gleason score, advanced stage and bone metastasis in prostate cancer. Oncogene **2000**, 19, 1288–1296. \[\] \[[CrossRef](http://doi.org/10.1038/sj.onc.1203426)\] \[[Green Version](https://www.nature.com/articles/1203426.pdf)\]
40. Tiwari, R.; Manzar, N.; Bhatia, V.; Yadav, A.; Nengroo, M.A.; Datta, D.; Carskadon, S.; Gupta, N.; Sigouros, M.; Khani, F.; et al. Androgen deprivation upregulates SPINK1 expression and potentiates cellular plasticity in prostate cancer. Nat. Commun.**2020**, 11, 384. \[\] \[[CrossRef](http://doi.org/10.1038/s41467-019-14184-0)\] \[[Green Version](https://www.nature.com/articles/s41467-019-14184-0.pdf)\]
41. Hale, J.S.; Otvos, B.; Sinyuk, M.; Alvarado, A.G.; Hitomi, M.; Stoltz, K.; Wu, Q.; Flavahan, W.; Levison, B.; Johansen, M.L.; et al. Cancer stem cell-specific scavenger receptor CD36 drives glioblastoma progression. Stem Cells **2014**, 32, 1746–1758. \[\] \[[CrossRef](http://doi.org/10.1002/stem.1716)\] \[[Green Version](http://europepmc.org/articles/pmc4063873?pdf=render)\]
42. Wu, X.; Fleming, A.; Ricketts, T.; Pavel, M.; Virgin, H.; Menzies, F.M.; Rubinsztein, D.C. Autophagy regulates Notch degradation and modulates stem cell development and neurogenesis. Nat. Commun.**2016**, 7, 10533. \[\] \[[CrossRef](http://doi.org/10.1038/ncomms10533)\] \[[Green Version](https://www.nature.com/articles/ncomms10533.pdf)\]
43. Hu, J.; Shi, B.; Liu, X.; Jiang, M.; Yuan, C.; Jiang, B.; Song, Y.; Zeng, Y.; Wang, G. The activation of Toll-like receptor 4 reverses tumor differentiation in human glioma U251 cells via Notch pathway. Int. Immunopharmacol.**2018**, 64, 33–41. \[\] \[[CrossRef](http://doi.org/10.1016/j.intimp.2018.08.019)\]
44. Kobayashi, T.; Piao, W.; Takamura, T.; Kori, H.; Miyachi, H.; Kitano, S.; Iwamoto, Y.; Yamada, M.; Imayoshi, I.; Shioda, S.; et al. Enhanced lysosomal degradation maintains the quiescent state of neural stem cells. Nat. Commun.**2019**, 10, 5446. \[\] \[[CrossRef](http://doi.org/10.1038/s41467-019-13203-4)\] \[[PubMed](http://www.ncbi.nlm.nih.gov/pubmed/31784517)\]
45. Gaisa, N.T.; Graham, T.A.; McDonald, S.A.; Poulsom, R.; Heidenreich, A.; Jakse, G.; Knuechel, R.; Wright, N.A. Clonal architecture of human prostatic epithelium in benign and malignant conditions. J. Pathol.**2011**, 225, 172–180. \[\] \[[CrossRef](http://doi.org/10.1002/path.2959)\]
46. Manzo, G. Similarities Between Embryo Development and Cancer Process Suggest New Strategies for Research and Therapy of Tumors: A New Point of View. Front. Cell Dev. Biol.**2019**, 7, 20. \[\] \[[CrossRef](http://doi.org/10.3389/fcell.2019.00020)\]
47. Naxerova, K.; Bult, C.J.; Peaston, A.; Fancher, K.; Knowles, B.B.; Kasif, S.; Kohane, I.S. Analysis of gene expression in a developmental context emphasizes distinct biological leitmotifs in human cancers. Genome Biol.**2008**, 9, R108. \[\] \[[CrossRef](http://doi.org/10.1186/gb-2008-9-7-r108)\] \[Green Version\]
48. Miyamoto, D.T.; Zheng, Y.; Wittner, B.S.; Lee, R.J.; Zhu, H.; Broderick, K.T.; Desai, R.; Fox, D.B.; Brannigan, B.W.; Trautwein, J.; et al. RNA-Seq of single prostate CTCs implicates noncanonical Wnt signaling in antiandrogen resistance. Science **2015**, 349, 1351–1356. \[\] \[[CrossRef](http://doi.org/10.1126/science.aab0917)\] \[[Green Version](https://science.sciencemag.org/content/sci/349/6254/1351.full.pdf)\]
49. Xie, L.; Hu, W.Y.; Hu, D.P.; Shi, G.B.; Li, Y.; Yang, J.F.; Prins, G.S. Effects of Inorganic Arsenic on Human Prostate Stem-Progenitor Cell Transformation, Autophagic Flux Blockade, and NRF2 Pathway Activation. Environ. Health Perspect.**2020**, 128, 67008. \[\] \[[CrossRef](http://doi.org/10.1289/EHP6471)\] \[[PubMed](http://www.ncbi.nlm.nih.gov/pubmed/32525701)\]
50. Chua, C.W.; Shibata, M.; Lei, M.; Toivanen, R.; Barlow, L.J.; Bergren, S.K.; Badani, K.K.; McKiernan, J.M.; Benson, M.C.; Hibshoosh, H.; et al. Single luminal epithelial progenitors can generate prostate organoids in culture. Nat. Cell Biol.**2014**, 16, 951–961. \[\] \[[CrossRef](http://doi.org/10.1038/ncb3047)\] \[[PubMed](http://www.ncbi.nlm.nih.gov/pubmed/25241035)\] \[[Green Version](http://europepmc.org/articles/pmc4183706?pdf=render)\]
51. Kasper, S. Exploring the origins of the normal prostate and prostate cancer stem cell. Stem Cell Rev.**2008**, 4, 193–201. \[\] \[[CrossRef](http://doi.org/10.1007/s12015-008-9033-1)\]
52. Yang, Y.; Hao, J.; Liu, X.; Dalkin, B.; Nagle, R.B. Differential expression of cytokeratin mRNA and protein in normal prostate, prostatic intraepithelial neoplasia, and invasive carcinoma. Am. J. Pathol.**1997**, 150, 693–704. \[\] \[[PubMed](http://www.ncbi.nlm.nih.gov/pubmed/9033282)\]
53. Xue, Y.; Verhofstad, A.; Lange, W.; Smedts, F.; Debruyne, F.; de la Rosette, J.; Schalken, J. Prostatic neuroendocrine cells have a unique keratin expression pattern and do not express Bcl-2: Cell kinetic features of neuroendocrine cells in the human prostate. Am. J. Pathol.**1997**, 151, 1759–1765. \[\] \[[PubMed](http://www.ncbi.nlm.nih.gov/pubmed/9403726)\]
54. van Leenders, G.J.; Aalders, T.W.; Hulsbergen-van de Kaa, C.A.; Ruiter, D.J.; Schalken, J.A. Expression of basal cell keratins in human prostate cancer metastases and cell lines. J. Pathol.**2001**, 195, 563–570. \[\] \[[CrossRef](http://doi.org/10.1002/path.993)\]
55. Ferronika, P.; Triningsih, F.X.; Ghozali, A.; Moeljono, A.; Rahmayanti, S.; Shadrina, A.N.; Naim, A.E.; Wudexi, I.; Arnurisa, A.M.; Nanwani, S.T.; et al. p63 cytoplasmic aberrance is associated with high prostate cancer stem cell expression. Asian Pac. J. Cancer Prev.**2012**, 13, 1943–1948. \[\] \[[CrossRef](http://doi.org/10.7314/APJCP.2012.13.5.1943)\]
56. Dhillon, P.K.; Barry, M.; Stampfer, M.J.; Perner, S.; Fiorentino, M.; Fornari, A.; Ma, J.; Fleet, J.; Kurth, T.; Rubin, M.A.; et al. Aberrant cytoplasmic expression of p63 and prostate cancer mortality. Cancer Epidemiol. Biomark. Prev.**2009**, 18, 595–600. \[\] \[[CrossRef](http://doi.org/10.1158/1055-9965.EPI-08-0785)\] \[[Green Version](http://cebp.aacrjournals.org/content/18/2/595.full.pdf)\]
57. Maitland, N.J.; Frame, F.M.; Polson, E.S.; Lewis, J.L.; Collins, A.T. Prostate Cancer Stem Cells: Do They Have a Basal or Luminal Phenotype? Horm. Cancer **2011**, 2, 47–61. \[\] \[[CrossRef](http://doi.org/10.1007/s12672-010-0058-y)\] \[[Green Version](https://link.springer.com/content/pdf/10.1007/s12672-010-0058-y.pdf)\]
58. Gu, G.; Yuan, J.; Wills, M.; Kasper, S. Prostate cancer cells with stem cell characteristics reconstitute the original human tumor in vivo. Cancer Res.**2007**, 67, 4807–4815. \[\] \[[CrossRef](http://doi.org/10.1158/0008-5472.CAN-06-4608)\] \[[Green Version](http://cancerres.aacrjournals.org/content/67/10/4807.full.pdf)\]
59. Collins, A.T.; Berry, P.A.; Hyde, C.; Stower, M.J.; Maitland, N.J. Prospective identification of tumorigenic prostate cancer stem cells. Cancer Res.**2005**, 65, 10946–10951. \[\] \[[CrossRef](http://doi.org/10.1158/0008-5472.CAN-05-2018)\] \[[Green Version](http://cancerres.aacrjournals.org/content/canres/65/23/10946.full.pdf)\]
60. Skvortsov, S.; Skvortsova, I.I.; Tang, D.G.; Dubrovska, A. Concise Review: Prostate Cancer Stem Cells: Current Understanding. Stem Cells.**2018**, 36, 1457–1474. \[\] \[[CrossRef](http://doi.org/10.1002/stem.2859)\] \[[Green Version](https://stemcellsjournals.onlinelibrary.wiley.com/doi/pdfdirect/10.1002/stem.2859)\]
61. Hudson, D.L. Epithelial stem cells in human prostate growth and disease. Prostate Cancer Prostatic Dis.**2004**, 7, 188–194. \[\] \[[CrossRef](http://doi.org/10.1038/sj.pcan.4500745)\]
62. Vander Griend, D.J.; Karthaus, W.L.; Dalrymple, S.; Meeker, A.; DeMarzo, A.M.; Isaacs, J.T. The role of CD133 in normal human prostate stem cells and malignant cancer-initiating cells. Cancer Res.**2008**, 68, 9703–9711. \[\] \[[CrossRef](http://doi.org/10.1158/0008-5472.CAN-08-3084)\] \[[Green Version](http://cancerres.aacrjournals.org/content/68/23/9703.full.pdf)\]
63. Chen, X.; Li, Q.; Liu, X.; Liu, C.; Liu, R.; Rycaj, K.; Zhang, D.; Liu, B.; Jeter, C.; Calhoun-Davis, T.; et al. Defining a Population of Stem-like Human Prostate Cancer Cells That Can Generate and Propagate Castration-Resistant Prostate Cancer. Clin. Cancer Res.**2016**, 22, 4505–4516. \[\] \[[CrossRef](http://doi.org/10.1158/1078-0432.CCR-15-2956)\] \[[Green Version](https://clincancerres.aacrjournals.org/content/clincanres/22/17/4505.full.pdf)\]
64. Smith, B.A.; Sokolov, A.; Uzunangelov, V.; Baertsch, R.; Newton, Y.; Graim, K.; Mathis, C.; Cheng, D.; Stuart, J.M.; Witte, O.N. A basal stem cell signature identifies aggressive prostate cancer phenotypes. Porc. Natl. Acad. Sci. USA **2015**, 112, E6544–E6552. \[\] \[[CrossRef](http://doi.org/10.1073/pnas.1518007112)\] \[[Green Version](https://www.pnas.org/content/pnas/112/47/E6544.full.pdf)\]
65. Subramanian, A.; Tamayo, P.; Mootha, V.K.; Mukherjee, S.; Ebert, B.L.; Gillette, M.A.; Paulovich, A.; Pomeroy, S.L.; Golub, T.R.; Lander, E.S.; et al. Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles. Porc. Natl. Acad. Sci. USA **2005**, 102, 15545–15550. \[\] \[[CrossRef](http://doi.org/10.1073/pnas.0506580102)\] \[[Green Version](http://www.pnas.org/content/102/43/15545.full.pdf)\]
66. Macosko, E.Z.; Basu, A.; Satija, R.; Nemesh, J.; Shekhar, K.; Goldman, M.; Tirosh, I.; Bialas, A.R.; Kamitaki, N.; Martersteck, E.M.; et al. Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets. Cell **2015**, 161, 1202–1214. \[\] \[[CrossRef](http://doi.org/10.1016/j.cell.2015.05.002)\] \[[Green Version](http://www.cell.com/article/S0092867415005498/pdf)\]
67. Stuart, T.; Butler, A.; Hoffman, P.; Hafemeister, C.; Papalexi, E.; Mauck WM 3rd Hao, Y.; Stoeckius, M.; Smibert, P.; Satija, R. Comprehensive Integration of Single-Cell Data. Cell **2019**, 177, 1888–1902. \[\] \[[CrossRef](http://doi.org/10.1016/j.cell.2019.05.031)\]

**Figure 1.** Two groups of differentially expressed keratin genes in CFSE <sup>High</sup> vs. CFSE <sup>Low</sup> day 5 prostasphere cells. (**A**) Normal primary prostate epithelial cells (PrEC) pre-labeled with CFSE were 3D cultured (5 days) to form prostaspheres (PS), followed by FACS to separate CFSE <sup>High</sup> stem-like cells (1.01%) from CFSE <sup>Low</sup> progenitor cells (97.52%). Scale bar = 50 µm. (**B**) Heatmap clustering of Fluidigm C1 captured single-cell RNA-seq of prostasphere cells revealed enrichment of KRT13, 23, 80, 78, 86, and 4 in CFSE <sup>High</sup> prostate stem-like cells (89 cells), while KRT6, 17, 14, 5, 8, and 18 were enriched in CFSE <sup>Low</sup> progenitor cells (170 cells). (**C**) Top, Immunostaining confirmed the colocalization of keratins 13, 80, and 78 in BrdU label retaining prostasphere cells (pink) and keratin 14 localization in non-labeled progenitor cells. Bottom, FACS isolated CFSE <sup>High</sup> prostate stem-like cells immunostained for keratin 23, 19, while CFSE <sup>Low</sup> progenitor cells did not. Scale bar = 50 µm. (**D**) Single-cell RNA-seq of CFSE labeled prostaspheres followed by heatmap clustering using selected genes for stem, basal, luminal, and neuroendocrine cells showed enrichment of stem cell marker gene KRT13 in CFSE <sup>High</sup> prostate stem cells, while basal marker genes KRT5, KRT14, TP63, GSTP1, and KRT19, luminal genes KRT8/18, and neuroendocrine gene SYP were enriched in CFSE <sup>Low</sup> prostate progenitor cells. SOX9 was uniquely enriched in CFSE <sup>High</sup> cells and a subpopulation of CFSE <sup>Low</sup> cells.

[![Ijms 22 08109 g001](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g001.png)](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g001.png)

**Figure 2.** Single-cell RNA-seq of CFSE <sup>High</sup> cells revealed three subpopulations of cells. Data from Fluidigm C1 single-cell RNA-seq of CFSE <sup>High</sup> isolated prostasphere cells was analyzed by UMAP (**A**) and TSCAN (**B**) principal component analyses, which identified three major cell clusters that were further confirmed by unsupervised heatmap clustering (**C**). CFSE <sup>High</sup> cell hierarchical clustering (I (1) → II (2) → III (3)) was ordered by MST-based pseudo-time reconstruction to mimic true biological time, identifying cells in cluster I as originating cells which yield cluster II that then yield cluster III (labeled in B). (**D**) Known stemness genes PSCA, CD36, and SPINK1 are enriched in cluster I (1) as compared to clusters II (2) & III (3), which is demarcated by a circle size (top) that represents gene expression levels that are relatively quantified at the bottom.

[![Ijms 22 08109 g002](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g002.png)](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g002.png)

**Figure 3.** Differentially expressed keratin genes in CFSE <sup>High</sup> prostasphere cells are sufficient to distinguish cells in cluster I, cluster II, and cluster III. (**A**) In CFSE <sup>High</sup> cells, KRT13, 23, 80, 78, and 4 are enriched in cluster I quiescent stem cells and have low expression in clusters II and III. KRT16, 17, and 6 genes are enriched in cluster III, whereas cluster II has low expression levels of this keratin gene set. (**B**) TSCAN analysis documents enrichment of KRT13, 80, 78, and 23 in cluster I (1) quiescent stem cells and of KRT16P4 and 17P6 in cluster III (3) cells demarcated by the circle size (top) which is directionally relatively quantified at bottom. (**C**) GSEA analysis identifies enrichment of stem cell-related Notch, Toll-like receptor, autophagy, and lysosome pathways in cluster I (1) vs. cluster III (3).

[![Ijms 22 08109 g003](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g003.png)](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g003.png)

**Figure 4.** Keratin profiles of differentially expressed keratin genes in non-label retaining prostasphere cells. (**A**) Single-cell RNA-seq of the sorted CFSE <sup>Low</sup> prostasphere cells with subsequent keratin gene heatmap clustering reveals three subpopulations of progenitor cells: cluster IV (right), cluster V (middle), and cluster VI (left). Cluster IV is enriched in KRT14, 5, 16, and 6, indicating a basal progenitor cell lineage. Clusters V and VI are enriched in KRT8, 18, and 10, indicating a luminal cell lineage. (**B**) TSCAN analysis of KRT8 and KRT18 in CFSE <sup>Low</sup> cells suggests a relationship between cluster V (2) and VI (1), but not cluster IV (3). Relative gene expression levels are noted numerically for each cluster.

[![Ijms 22 08109 g004](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g004.png)](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g004.png)

**Figure 5.** RNA-seq reveals differentially expressed genes in cancer stem-like cells and vs. non-stem cells in tumor spheroids of patient PCa specimens. (**A**) Cancer stem-like cells (cSC; (CFSE <sup>High</sup>) and non-cSC (CFSE <sup>low</sup>) from spheroids of three PCa specimens were separated by FACS. (**B**) Following RNA-seq of the sorted cells, unsupervised transcriptome profiling identified 1310 differentially expressed genes (Q value < 0.05); 916 enriched in cSCs and 394 enriched in non-cSCs. Sequencing of the non-cSC fractions from three patients were performed in duplicate, highlighting data consistency. For the cSC fraction, one patient sample was sequenced in duplicate, and the other two were sequenced in singlet. (**C**) Heatmap of CD36, KRT13, SPINK1, and SCGB2A1 expression shows enrichment in cSCs vs. non-cSCs. (**D**) ICC documents that PSCA, CD36, and SPINK1 proteins (red) colocalized with BrdU-labeled (green) cSC cells in the 3D cultured tumor spheroids. Similar to normal prostate stem cells, the cSCs exhibit decreased E-cadherin, increased LC3 and KRT13 (Hu et al., 2017), which together identifies unique features of PCa-derived cancer stem-like cells. (E) siRNA knockdown of KRT13 significantly decreased the percentage of CFSE <sup>High</sup> cells in tumor spheroids (left) and tumor spheroid formation (right). \* p < 0.05 vs. siControl, N = 3.

[![Ijms 22 08109 g005](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g005.png)](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g005.png)

**Figure 6.** Heatmap clustering of keratin genes in PCa stem-like cells (cSCs) and the non-stem cells in tumor spheroids from PCa specimens. RNA-seq followed by heatmap clustering analysis shows that in addition to normal stem cell-enriched keratins (KRT13, 23, 80, 78, 4), KRT10, 19, 6C, 75, 16, 79, 3, and 82 also were enriched in cancer stem-like cells relative to expression levels in the non-cSC population. Q -value < 0.05.

[![Ijms 22 08109 g006](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g006.png)](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g006.png)

**Figure 7.** Keratin profiling in tumor spheroids using single-cell RNA-seq identifies cancer stem-like cells. (**A**,**B**) Uniform manifold approximation and projection (UMAP) clustering defined five clusters in cancer cells, identifying cluster 2 as cancer stem-like cells enriched for stemness markers, including KRT13. (**C**) Keratin gene profiling by heatmap and (**D**) dot-plot analyses identified a group of keratin genes enriched in the cluster 2 cancer stem-like cells.

[![Ijms 22 08109 g007](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g007.png)](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g007.png)

**Figure 8.** Model of prostate epithelial stem cell hierarchy based upon keratin profiles. Quiescent prostate stem cells (cluster I: enriched for KRT13, 23, 80, 78, 4) become active stem cells (cluster II), with increased expression of KRT 16, 17, and 6 that generate bipotent progenitor cells (cluster III) and further give rise to committed unique potent basal progenitors (cluster IV: enriched for KRT5, 14, 6, 16), as well as early (cluster V) and late-stage (cluster VI) luminal progenitors (enriched for KRT8, 18, 10).

[![Ijms 22 08109 g008](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g008.png)](https://www.mdpi.com/ijms/ijms-22-08109/article_deploy/html/images/ijms-22-08109-g008.png)

**Table 1.** Top genes enriched in prostate cancer stem-like cells.

<table><thead><tr><th colspan="2" align="center">Gene</th><th align="center">cSC/non-cSC</th></tr><tr><th align="left">Symbol</th><th align="left">Gene Name</th><th align="right">(Fold Change)</th></tr></thead><tbody><tr><td align="left">SCGB2A1</td><td align="left">Secretoglobulin family 2A1</td><td align="right">67,370</td></tr><tr><td align="left">SPINK1</td><td align="left">Serine peptidase inhibitor kazak-type 1</td><td align="right">2270</td></tr><tr><td align="left">PSCA</td><td align="left">Prostate stem cell antigen</td><td align="right">1621</td></tr><tr><td align="left">CD36</td><td align="left">Scavenger receptor class B member 3</td><td align="right">990</td></tr><tr><td align="left">TMPRSS11E</td><td align="left">Transmembrane serine protease 11E</td><td align="right">593</td></tr><tr><td align="left">HMOX1</td><td align="left">Heme oxygenase 1</td><td align="right">128</td></tr><tr><td align="left">IGF2</td><td align="left">Insulin growth factor 2</td><td align="right">117</td></tr></tbody></table>

**Table 2.** Enriched pathways in cSC cells (**A**) and non-cSC cells (**B**) by MetaCore Enrichment analysis.

<table><tbody><tr><td colspan="6" align="center">(<b>A</b>) <b>Pathways enriched in cSC cells</b> (Prostate_Cancer_916_genes_ <span>Q</span> value_0.05_genelist)</td></tr><tr><td align="center"><b>Maps</b></td><td align="center"><b>Total</b></td><td align="center"><b><span>p</span> -Value</b></td><td align="center"><b>FDR</b></td><td align="center"><b>In Data</b></td><td align="center"><b>Network Objects from Active Data</b></td></tr><tr><td align="left"><b>Cytoskeleton remodeling: Keratin filaments</b></td><td align="center">36</td><td align="center">2.7 × 10 <sup>−8</sup></td><td align="center">1.8 × 10 <sup>−5</sup></td><td align="center">10</td><td align="left">Keratin 16, 14-3-3 gamma, Keratin 4, PPL (periplakin), Keratin 6C, Keratin 6A, Keratin 19, Keratin 13, Keratin 4/13, Plakophilin 1</td></tr><tr><td align="left"><b>Ligand-independent activation of androgen receptor in PCa</b></td><td align="center">67</td><td align="center">1.3 × 10 <sup>−5</sup></td><td align="center">3.1 × 10 <sup>−3</sup></td><td align="center">10</td><td align="left">GAB1, Bcl-XL, PP2A regulatory, c-Myc, ERK1 (MAPK3), S5AR2, PP2A catalytic, FGFR1, Kallikrein 3 (PSA), ErbB3</td></tr><tr><td align="left"><b>Androgen receptor activation and downstream signaling in PCa</b></td><td align="center">110</td><td align="center">8.5 × 10 <sup>−4</sup></td><td align="center">6.2 × 10 <sup>−2</sup></td><td align="center">10</td><td align="left">GAB1, Bcl-XL, TMPRSS2, c-Myc, ERK1 (MAPK3), S5AR2, Kallikrein 2, FGFR1, PSCA, Kallikrein 3 (PSA)</td></tr><tr><td align="left"><b>Transcription: HIF-1 targets</b></td><td align="center">95</td><td align="center">5.4 × 10 <sup>−5</sup></td><td align="center">0.0071</td><td align="center">11</td><td align="left">ROR-alpha, TGM2, Mxi1, CITED2, Heme oxygenase 1, HXK2, c-Myc, Cyclin G2, LOXL2, AK3, CTGF</td></tr><tr><td align="left"><b>Development: IGF-1 receptor signaling</b></td><td align="center">51</td><td align="center">0.00044</td><td align="center">0.0416</td><td align="center">7</td><td align="left">Bcl-XL, MNK2(GPRK7), c-Myc, IGF-2, IBP, FOXO3A, ERK1/2</td></tr><tr><td align="left"><b>Autophagy</b></td><td align="center">35</td><td align="center">0.00246</td><td align="center">0.0993</td><td align="center">5</td><td align="left">GATE-16, Bcl-XL, WIPI2, MAP1LC3A, ULK1</td></tr><tr><td colspan="6" align="center">(<b>B</b>) <b>Pathways enriched in non-cSC cells</b> (Prostate_Cancer_394_genes_ <span>Q</span> value_0.05_genelist)</td></tr><tr><td align="center"><b>Maps</b></td><td align="center"><b>Total</b></td><td align="center"><b><span>p</span> -Value</b></td><td align="center"><b>FDR</b></td><td align="center"><b>In Data</b></td><td align="center"><b>Network Objects from Active Data</b></td></tr><tr><td align="left"><b>Cell cycle: Chromosome condensation in prometaphase</b></td><td align="center">21</td><td align="center">4.4 × 10 <sup>−14</sup></td><td align="center">2 × 10 <sup>−11</sup></td><td align="center">11</td><td align="left">Aurora-B, BRRN1, CAP-H/H2, CAP-G, CAP-E, Cyclin B, CAP-C, TOP2, CAP-G/G2, Histone H1, CDK1 (p34)</td></tr><tr><td align="left"><b>Cell cycle: The metaphase checkpoint</b></td><td align="center">36</td><td align="center">2.3 × 10 <sup>−12</sup></td><td align="center">6 × 10 <sup>−10</sup></td><td align="center">12</td><td align="left">Nek2A, Aurora-B, HEC, CDCA1, CDC20, HZwint-1, Rod, CENP-F, DSN1, SPBC24, CENP-E, PLK1</td></tr><tr><td align="left"><b>Cell cycle: Transition and termination of DNA replication</b></td><td align="center">27</td><td align="center">1.4 × 10 <sup>−9</sup></td><td align="center">2 × 10 <sup>−7</sup></td><td align="center">9</td><td align="left">TOP2 alpha, POLD reg (p68), Brca1/Bard1, Brca1, MCM2, TOP2, FEN1, Bard1, CDK1 (p34)</td></tr><tr><td align="left"><b>Cell cycle: Start of DNA replication in early S phase</b></td><td align="center">32</td><td align="center">2.5 × 10 <sup>−6</sup></td><td align="center">0.0001</td><td align="center">7</td><td align="left">CDC18L (CDC6), MCM10, Geminin, MCM4, MCM2, Histone H1, CDC45L</td></tr><tr><td align="left"><b>Cell cycle: Initiation of mitosis</b></td><td align="center">26</td><td align="center">2.5 × 10 <sup>−8</sup></td><td align="center">3 × 10 <sup>−6</sup></td><td align="center">8</td><td align="left">Lamin B, Cyclin B1, Cyclin B2, FOXM1, Kinase MYT1, Histone H1, CDK1 (p34), PLK1</td></tr><tr><td align="left"><b>Signal transduction: NF-kB activation pathways</b></td><td align="center">51</td><td align="center">5 × 10 <sup>−8</sup></td><td align="center">4 × 10 <sup>−6</sup></td><td align="center">10</td><td align="left">NF-kB2 (p100), IL-1 beta, NF-kB2 (p52), I-kB, NF-kB, c-IAP1, c-IAP2, NF-kB1 (p105), BAFF(TNFSF13B), NF-kB1 (p50)</td></tr></tbody></table>

|  | **Publisher’s Note:** MDPI stays neutral with regard to jurisdictional claims in published maps and institutional affiliations. |
| --- | --- |

## Share and Cite

**MDPI and ACS Style**

Hu, W.-Y.; Hu, D.-P.; Xie, L.; Nonn, L.; Lu, R.; Abern, M.; Shioda, T.; Prins, G.S. Keratin Profiling by Single-Cell RNA-Sequencing Identifies Human Prostate Stem Cell Lineage Hierarchy and Cancer Stem-Like Cells. *Int. J. Mol. Sci.***2021**, *22*, 8109. https://doi.org/10.3390/ijms22158109

**AMA Style**  

Hu W-Y, Hu D-P, Xie L, Nonn L, Lu R, Abern M, Shioda T, Prins GS. Keratin Profiling by Single-Cell RNA-Sequencing Identifies Human Prostate Stem Cell Lineage Hierarchy and Cancer Stem-Like Cells. *International Journal of Molecular Sciences*. 2021; 22(15):8109. https://doi.org/10.3390/ijms22158109

**Chicago/Turabian Style**  

Hu, Wen-Yang, Dan-Ping Hu, Lishi Xie, Larisa Nonn, Ranli Lu, Michael Abern, Toshihiro Shioda, and Gail S. Prins. 2021. "Keratin Profiling by Single-Cell RNA-Sequencing Identifies Human Prostate Stem Cell Lineage Hierarchy and Cancer Stem-Like Cells" *International Journal of Molecular Sciences* 22, no. 15: 8109. https://doi.org/10.3390/ijms22158109

**APA Style**  

Hu, W.-Y., Hu, D.-P., Xie, L., Nonn, L., Lu, R., Abern, M., Shioda, T., & Prins, G. S. (2021). Keratin Profiling by Single-Cell RNA-Sequencing Identifies Human Prostate Stem Cell Lineage Hierarchy and Cancer Stem-Like Cells. *International Journal of Molecular Sciences*, *22* (15), 8109. https://doi.org/10.3390/ijms22158109

Note that from the first issue of 2016, this journal uses article numbers instead of page numbers. See further details [here](https://www.mdpi.com/about/announcements/784).

## Article Metrics

### Citations

Crossref

[10](https://www.mdpi.com/1422-0067/22/15/#)

Scopus

[9](https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85111273365&origin=inward)

Web of Science

[8](https://www.webofscience.com/api/gateway?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000681868200001&DestLinkType=FullRecord&DestApp=ALL_WOS&UsrCustomerID=7e717b028bc90856e7c55c7029afc773)

PubMed

[10](https://www.ncbi.nlm.nih.gov/pubmed?linkname=pubmed_pubmed_citedin&from_uid=34360875)

PMC

[10](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8346986/citedby/)

Google Scholar